<html><head><meta content="text/html; charset=UTF-8" http-equiv="content-type"><style type="text/css">@import url(https://themes.googleusercontent.com/fonts/css?kit=OPeqXG-QxW3ZD8BtmPikfA);.lst-kix_a6b21xtlypbx-8>li{counter-increment:lst-ctn-kix_a6b21xtlypbx-8}ul.lst-kix_dqnkgejf3n11-1{list-style-type:none}ul.lst-kix_dqnkgejf3n11-0{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-6.start{counter-reset:lst-ctn-kix_u5e0ktun9hhb-6 0}.lst-kix_mgss2idoka6m-7>li{counter-increment:lst-ctn-kix_mgss2idoka6m-7}.lst-kix_krq4n4yenarg-6>li{counter-increment:lst-ctn-kix_krq4n4yenarg-6}.lst-kix_5j20aiwerfhd-6>li{counter-increment:lst-ctn-kix_5j20aiwerfhd-6}ul.lst-kix_7ly1sailhnxn-8{list-style-type:none}ul.lst-kix_dqnkgejf3n11-7{list-style-type:none}ul.lst-kix_7ly1sailhnxn-7{list-style-type:none}ul.lst-kix_dqnkgejf3n11-6{list-style-type:none}ul.lst-kix_dqnkgejf3n11-8{list-style-type:none}ul.lst-kix_dqnkgejf3n11-3{list-style-type:none}ul.lst-kix_dqnkgejf3n11-2{list-style-type:none}ul.lst-kix_dqnkgejf3n11-5{list-style-type:none}ul.lst-kix_dqnkgejf3n11-4{list-style-type:none}.lst-kix_u5e0ktun9hhb-4>li{counter-increment:lst-ctn-kix_u5e0ktun9hhb-4}ul.lst-kix_jha6kbfqqd5o-8{list-style-type:none}.lst-kix_hh6mlhac9a3w-7>li:before{content:"\0025cb   "}ul.lst-kix_jha6kbfqqd5o-7{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-1.start{counter-reset:lst-ctn-kix_u5e0ktun9hhb-1 0}.lst-kix_hh6mlhac9a3w-6>li:before{content:"\0025cf   "}.lst-kix_hh6mlhac9a3w-8>li:before{content:"\0025a0   "}.lst-kix_hh6mlhac9a3w-5>li:before{content:"\0025a0   "}ol.lst-kix_a6b21xtlypbx-2.start{counter-reset:lst-ctn-kix_a6b21xtlypbx-2 0}.lst-kix_a6b21xtlypbx-1>li{counter-increment:lst-ctn-kix_a6b21xtlypbx-1}ul.lst-kix_jha6kbfqqd5o-0{list-style-type:none}ul.lst-kix_jha6kbfqqd5o-2{list-style-type:none}ul.lst-kix_jha6kbfqqd5o-1{list-style-type:none}ul.lst-kix_jha6kbfqqd5o-4{list-style-type:none}ul.lst-kix_jha6kbfqqd5o-3{list-style-type:none}ul.lst-kix_jha6kbfqqd5o-6{list-style-type:none}ul.lst-kix_jha6kbfqqd5o-5{list-style-type:none}.lst-kix_hh6mlhac9a3w-0>li:before{content:"\0025cf   "}.lst-kix_hh6mlhac9a3w-1>li:before{content:"\0025cb   "}.lst-kix_mgss2idoka6m-0>li{counter-increment:lst-ctn-kix_mgss2idoka6m-0}.lst-kix_hh6mlhac9a3w-3>li:before{content:"\0025cf   "}.lst-kix_hh6mlhac9a3w-2>li:before{content:"\0025a0   "}.lst-kix_hh6mlhac9a3w-4>li:before{content:"\0025cb   "}.lst-kix_k8se1q4z4tp9-0>li:before{content:"\0025cf   "}.lst-kix_k8se1q4z4tp9-2>li:before{content:"\0025a0   "}.lst-kix_k8se1q4z4tp9-4>li:before{content:"\0025cb   "}ol.lst-kix_xrbxb0icqoxu-1.start{counter-reset:lst-ctn-kix_xrbxb0icqoxu-1 0}.lst-kix_k8se1q4z4tp9-8>li:before{content:"\0025a0   "}.lst-kix_k8se1q4z4tp9-6>li:before{content:"\0025cf   "}.lst-kix_bfrlw18c0l4c-1>li:before{content:"\0025cb   "}.lst-kix_bfrlw18c0l4c-3>li:before{content:"\0025cf   "}.lst-kix_bfrlw18c0l4c-5>li:before{content:"\0025a0   "}.lst-kix_bfrlw18c0l4c-7>li:before{content:"\0025cb   "}ul.lst-kix_qzl115totbz-1{list-style-type:none}ul.lst-kix_qzl115totbz-0{list-style-type:none}.lst-kix_a6b21xtlypbx-0>li{counter-increment:lst-ctn-kix_a6b21xtlypbx-0}ul.lst-kix_qzl115totbz-8{list-style-type:none}ul.lst-kix_qzl115totbz-7{list-style-type:none}ul.lst-kix_qzl115totbz-6{list-style-type:none}ol.lst-kix_5j20aiwerfhd-4.start{counter-reset:lst-ctn-kix_5j20aiwerfhd-4 0}ul.lst-kix_qzl115totbz-5{list-style-type:none}ul.lst-kix_qzl115totbz-4{list-style-type:none}ul.lst-kix_qzl115totbz-3{list-style-type:none}ul.lst-kix_qzl115totbz-2{list-style-type:none}.lst-kix_vw1z3jdppwml-3>li:before{content:"\0025cf   "}.lst-kix_vw1z3jdppwml-1>li:before{content:"\0025cb   "}.lst-kix_vw1z3jdppwml-5>li:before{content:"\0025a0   "}.lst-kix_jb5m9ajaek8x-3>li:before{content:"\0025cf   "}.lst-kix_jb5m9ajaek8x-1>li:before{content:"\0025cb   "}ul.lst-kix_uscd8mjnf5fw-2{list-style-type:none}ul.lst-kix_uscd8mjnf5fw-1{list-style-type:none}ul.lst-kix_uscd8mjnf5fw-4{list-style-type:none}ul.lst-kix_uscd8mjnf5fw-3{list-style-type:none}ul.lst-kix_uscd8mjnf5fw-6{list-style-type:none}ul.lst-kix_uscd8mjnf5fw-5{list-style-type:none}ul.lst-kix_uscd8mjnf5fw-8{list-style-type:none}ul.lst-kix_uscd8mjnf5fw-7{list-style-type:none}.lst-kix_jb5m9ajaek8x-7>li:before{content:"\0025cb   "}ul.lst-kix_uscd8mjnf5fw-0{list-style-type:none}.lst-kix_vw1z3jdppwml-7>li:before{content:"\0025cb   "}.lst-kix_jb5m9ajaek8x-5>li:before{content:"\0025a0   "}.lst-kix_4wwjrzyyefv1-0>li:before{content:"\0025cf   "}ul.lst-kix_c9t4sdk23z1e-1{list-style-type:none}ul.lst-kix_c9t4sdk23z1e-0{list-style-type:none}ul.lst-kix_c9t4sdk23z1e-3{list-style-type:none}.lst-kix_dnezncmpad8v-3>li:before{content:"\0025cf   "}ul.lst-kix_c9t4sdk23z1e-2{list-style-type:none}ul.lst-kix_c9t4sdk23z1e-5{list-style-type:none}ul.lst-kix_c9t4sdk23z1e-4{list-style-type:none}ul.lst-kix_c9t4sdk23z1e-7{list-style-type:none}ul.lst-kix_c9t4sdk23z1e-6{list-style-type:none}.lst-kix_jha6kbfqqd5o-6>li:before{content:"\0025cf   "}.lst-kix_dnezncmpad8v-2>li:before{content:"\0025a0   "}.lst-kix_4wwjrzyyefv1-7>li:before{content:"\0025cb   "}.lst-kix_3rsahq87evpc-8>li:before{content:"\0025a0   "}.lst-kix_jha6kbfqqd5o-1>li:before{content:"\0025cb   "}.lst-kix_jha6kbfqqd5o-2>li:before{content:"\0025a0   "}.lst-kix_4wwjrzyyefv1-4>li:before{content:"\0025cb   "}ul.lst-kix_c9t4sdk23z1e-8{list-style-type:none}.lst-kix_jha6kbfqqd5o-5>li:before{content:"\0025a0   "}.lst-kix_dnezncmpad8v-7>li:before{content:"\0025cb   "}.lst-kix_dnezncmpad8v-6>li:before{content:"\0025cf   "}.lst-kix_4wwjrzyyefv1-3>li:before{content:"\0025cf   "}.lst-kix_tcf7t0doxn6s-5>li:before{content:"\0025a0   "}.lst-kix_tcf7t0doxn6s-4>li:before{content:"\0025cb   "}.lst-kix_tcf7t0doxn6s-8>li:before{content:"\0025a0   "}.lst-kix_u5e0ktun9hhb-5>li{counter-increment:lst-ctn-kix_u5e0ktun9hhb-5}.lst-kix_tcf7t0doxn6s-1>li:before{content:"\0025cb   "}.lst-kix_4wwjrzyyefv1-8>li:before{content:"\0025a0   "}.lst-kix_3amtkkojfr7l-0>li:before{content:"-  "}.lst-kix_3amtkkojfr7l-4>li:before{content:"-  "}.lst-kix_3amtkkojfr7l-3>li:before{content:"-  "}ol.lst-kix_5j20aiwerfhd-8{list-style-type:none}ol.lst-kix_5j20aiwerfhd-7{list-style-type:none}ol.lst-kix_5j20aiwerfhd-6{list-style-type:none}.lst-kix_3amtkkojfr7l-8>li:before{content:"-  "}.lst-kix_2l6dfml4zfm9-0>li:before{content:"\0025cf   "}ol.lst-kix_5j20aiwerfhd-1{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-2.start{counter-reset:lst-ctn-kix_u5e0ktun9hhb-2 0}ol.lst-kix_5j20aiwerfhd-0{list-style-type:none}ol.lst-kix_5j20aiwerfhd-5{list-style-type:none}ol.lst-kix_xrbxb0icqoxu-1{list-style-type:none}.lst-kix_efd89z9jnu5a-3>li:before{content:"-  "}ol.lst-kix_5j20aiwerfhd-4{list-style-type:none}ol.lst-kix_xrbxb0icqoxu-0{list-style-type:none}ol.lst-kix_5j20aiwerfhd-3{list-style-type:none}ol.lst-kix_xrbxb0icqoxu-3{list-style-type:none}ol.lst-kix_5j20aiwerfhd-2{list-style-type:none}ol.lst-kix_xrbxb0icqoxu-2{list-style-type:none}.lst-kix_3amtkkojfr7l-7>li:before{content:"-  "}ol.lst-kix_xrbxb0icqoxu-5{list-style-type:none}ol.lst-kix_5j20aiwerfhd-8.start{counter-reset:lst-ctn-kix_5j20aiwerfhd-8 0}ol.lst-kix_xrbxb0icqoxu-4{list-style-type:none}.lst-kix_2jqw2dfnv53p-1>li:before{content:"\0025cb   "}ol.lst-kix_xrbxb0icqoxu-7{list-style-type:none}ol.lst-kix_xrbxb0icqoxu-6{list-style-type:none}.lst-kix_tcf7t0doxn6s-0>li:before{content:"\0025cf   "}ol.lst-kix_xrbxb0icqoxu-8{list-style-type:none}.lst-kix_5j20aiwerfhd-7>li{counter-increment:lst-ctn-kix_5j20aiwerfhd-7}.lst-kix_efd89z9jnu5a-6>li:before{content:"-  "}.lst-kix_2jqw2dfnv53p-2>li:before{content:"\0025a0   "}.lst-kix_efd89z9jnu5a-7>li:before{content:"-  "}.lst-kix_krq4n4yenarg-7>li{counter-increment:lst-ctn-kix_krq4n4yenarg-7}.lst-kix_5j20aiwerfhd-0>li{counter-increment:lst-ctn-kix_5j20aiwerfhd-0}.lst-kix_xrbxb0icqoxu-5>li{counter-increment:lst-ctn-kix_xrbxb0icqoxu-5}.lst-kix_efd89z9jnu5a-2>li:before{content:"-  "}.lst-kix_a6b21xtlypbx-7>li{counter-increment:lst-ctn-kix_a6b21xtlypbx-7}ul.lst-kix_i0lzlnefeb2-2{list-style-type:none}ul.lst-kix_i0lzlnefeb2-3{list-style-type:none}ul.lst-kix_i0lzlnefeb2-0{list-style-type:none}ul.lst-kix_i0lzlnefeb2-1{list-style-type:none}ul.lst-kix_i0lzlnefeb2-6{list-style-type:none}ul.lst-kix_i0lzlnefeb2-7{list-style-type:none}ul.lst-kix_i0lzlnefeb2-4{list-style-type:none}ul.lst-kix_i0lzlnefeb2-5{list-style-type:none}.lst-kix_k8se1q4z4tp9-5>li:before{content:"\0025a0   "}ul.lst-kix_i0lzlnefeb2-8{list-style-type:none}.lst-kix_r6vkfvz5od5n-1>li:before{content:"\0025cb   "}.lst-kix_a6b21xtlypbx-2>li{counter-increment:lst-ctn-kix_a6b21xtlypbx-2}.lst-kix_bfrlw18c0l4c-0>li:before{content:"\0025cf   "}.lst-kix_r6vkfvz5od5n-5>li:before{content:"\0025a0   "}ul.lst-kix_uy8ee0jk5g05-7{list-style-type:none}ul.lst-kix_uy8ee0jk5g05-8{list-style-type:none}ul.lst-kix_uy8ee0jk5g05-5{list-style-type:none}ul.lst-kix_uy8ee0jk5g05-6{list-style-type:none}ul.lst-kix_uy8ee0jk5g05-0{list-style-type:none}.lst-kix_2jqw2dfnv53p-5>li:before{content:"\0025a0   "}ul.lst-kix_uy8ee0jk5g05-3{list-style-type:none}ul.lst-kix_uy8ee0jk5g05-4{list-style-type:none}ul.lst-kix_uy8ee0jk5g05-1{list-style-type:none}ul.lst-kix_uy8ee0jk5g05-2{list-style-type:none}.lst-kix_bfrlw18c0l4c-4>li:before{content:"\0025cb   "}ul.lst-kix_3amtkkojfr7l-8{list-style-type:none}ul.lst-kix_3amtkkojfr7l-4{list-style-type:none}.lst-kix_bfrlw18c0l4c-8>li:before{content:"\0025a0   "}ul.lst-kix_3amtkkojfr7l-5{list-style-type:none}ul.lst-kix_3amtkkojfr7l-6{list-style-type:none}ul.lst-kix_3amtkkojfr7l-7{list-style-type:none}ul.lst-kix_3amtkkojfr7l-0{list-style-type:none}ul.lst-kix_3amtkkojfr7l-1{list-style-type:none}.lst-kix_k8se1q4z4tp9-1>li:before{content:"\0025cb   "}ul.lst-kix_3amtkkojfr7l-2{list-style-type:none}ul.lst-kix_3amtkkojfr7l-3{list-style-type:none}.lst-kix_2l6dfml4zfm9-3>li:before{content:"\0025cf   "}.lst-kix_2l6dfml4zfm9-7>li:before{content:"\0025cb   "}.lst-kix_vk5294fzi1ce-6>li:before{content:"\0025cf   "}.lst-kix_43e7b9brr3ij-5>li:before{content:"\0025a0   "}.lst-kix_xrbxb0icqoxu-3>li{counter-increment:lst-ctn-kix_xrbxb0icqoxu-3}.lst-kix_3rsahq87evpc-0>li:before{content:"\0025cf   "}.lst-kix_vw1z3jdppwml-2>li:before{content:"\0025a0   "}.lst-kix_jb5m9ajaek8x-2>li:before{content:"\0025a0   "}.lst-kix_43e7b9brr3ij-1>li:before{content:"\0025cb   "}.lst-kix_jb5m9ajaek8x-6>li:before{content:"\0025cf   "}ul.lst-kix_r6vkfvz5od5n-8{list-style-type:none}ul.lst-kix_7pm3iovtnuj2-7{list-style-type:none}ul.lst-kix_r6vkfvz5od5n-7{list-style-type:none}ul.lst-kix_7pm3iovtnuj2-8{list-style-type:none}.lst-kix_3rsahq87evpc-4>li:before{content:"\0025cb   "}ul.lst-kix_r6vkfvz5od5n-4{list-style-type:none}ul.lst-kix_7pm3iovtnuj2-3{list-style-type:none}.lst-kix_vw1z3jdppwml-6>li:before{content:"\0025cf   "}ul.lst-kix_r6vkfvz5od5n-3{list-style-type:none}ul.lst-kix_7pm3iovtnuj2-4{list-style-type:none}ul.lst-kix_r6vkfvz5od5n-6{list-style-type:none}ul.lst-kix_7pm3iovtnuj2-5{list-style-type:none}ul.lst-kix_r6vkfvz5od5n-5{list-style-type:none}ul.lst-kix_7pm3iovtnuj2-6{list-style-type:none}ul.lst-kix_r6vkfvz5od5n-0{list-style-type:none}ul.lst-kix_7pm3iovtnuj2-0{list-style-type:none}ul.lst-kix_r6vkfvz5od5n-2{list-style-type:none}ul.lst-kix_7pm3iovtnuj2-1{list-style-type:none}ul.lst-kix_r6vkfvz5od5n-1{list-style-type:none}ul.lst-kix_7pm3iovtnuj2-2{list-style-type:none}.lst-kix_uscd8mjnf5fw-3>li:before{content:"\0025cf   "}.lst-kix_uscd8mjnf5fw-8>li:before{content:"\0025a0   "}ul.lst-kix_tcf7t0doxn6s-7{list-style-type:none}ul.lst-kix_tcf7t0doxn6s-8{list-style-type:none}.lst-kix_uscd8mjnf5fw-1>li:before{content:"\0025cb   "}ol.lst-kix_a6b21xtlypbx-8.start{counter-reset:lst-ctn-kix_a6b21xtlypbx-8 0}.lst-kix_dqnkgejf3n11-0>li:before{content:"\0025cf   "}.lst-kix_xrbxb0icqoxu-6>li{counter-increment:lst-ctn-kix_xrbxb0icqoxu-6}.lst-kix_uscd8mjnf5fw-6>li:before{content:"\0025cf   "}.lst-kix_dqnkgejf3n11-8>li:before{content:"\0025a0   "}.lst-kix_krq4n4yenarg-4>li{counter-increment:lst-ctn-kix_krq4n4yenarg-4}.lst-kix_dqnkgejf3n11-6>li:before{content:"\0025cf   "}ol.lst-kix_5j20aiwerfhd-0.start{counter-reset:lst-ctn-kix_5j20aiwerfhd-0 0}ul.lst-kix_tcf7t0doxn6s-0{list-style-type:none}ul.lst-kix_tcf7t0doxn6s-1{list-style-type:none}ul.lst-kix_tcf7t0doxn6s-2{list-style-type:none}ul.lst-kix_tcf7t0doxn6s-3{list-style-type:none}.lst-kix_7ly1sailhnxn-1>li:before{content:"\0025cb   "}ul.lst-kix_tcf7t0doxn6s-4{list-style-type:none}ul.lst-kix_tcf7t0doxn6s-5{list-style-type:none}ul.lst-kix_tcf7t0doxn6s-6{list-style-type:none}.lst-kix_dqnkgejf3n11-3>li:before{content:"\0025cf   "}.lst-kix_dqnkgejf3n11-5>li:before{content:"\0025a0   "}.lst-kix_7ly1sailhnxn-7>li:before{content:"\0025cb   "}.lst-kix_vk5294fzi1ce-2>li:before{content:"\0025a0   "}.lst-kix_vk5294fzi1ce-3>li:before{content:"\0025cf   "}.lst-kix_7ly1sailhnxn-4>li:before{content:"\0025cb   "}ol.lst-kix_a6b21xtlypbx-3.start{counter-reset:lst-ctn-kix_a6b21xtlypbx-3 0}.lst-kix_7ly1sailhnxn-2>li:before{content:"\0025a0   "}.lst-kix_vk5294fzi1ce-0>li:before{content:"\0025cf   "}.lst-kix_i0lzlnefeb2-2>li:before{content:"\0025a0   "}.lst-kix_xrbxb0icqoxu-2>li{counter-increment:lst-ctn-kix_xrbxb0icqoxu-2}.lst-kix_i0lzlnefeb2-7>li:before{content:"\0025cb   "}.lst-kix_r6vkfvz5od5n-7>li:before{content:"\0025cb   "}.lst-kix_i0lzlnefeb2-4>li:before{content:"\0025cb   "}.lst-kix_i0lzlnefeb2-5>li:before{content:"\0025a0   "}ol.lst-kix_xrbxb0icqoxu-2.start{counter-reset:lst-ctn-kix_xrbxb0icqoxu-2 0}.lst-kix_uscd8mjnf5fw-0>li:before{content:"\0025cf   "}ol.lst-kix_a6b21xtlypbx-1.start{counter-reset:lst-ctn-kix_a6b21xtlypbx-1 0}ul.lst-kix_vk5294fzi1ce-5{list-style-type:none}ul.lst-kix_vk5294fzi1ce-4{list-style-type:none}ul.lst-kix_vk5294fzi1ce-3{list-style-type:none}ul.lst-kix_vk5294fzi1ce-2{list-style-type:none}ul.lst-kix_vk5294fzi1ce-1{list-style-type:none}ul.lst-kix_vk5294fzi1ce-0{list-style-type:none}.lst-kix_mgss2idoka6m-3>li:before{content:"" counter(lst-ctn-kix_mgss2idoka6m-3,lower-latin) ") "}.lst-kix_un80xc3ep54b-7>li:before{content:"\0025cb   "}.lst-kix_r6vkfvz5od5n-2>li:before{content:"\0025a0   "}.lst-kix_un80xc3ep54b-1>li:before{content:"\0025cb   "}.lst-kix_bg52hg4v2ft5-0>li:before{content:"\0025cf   "}.lst-kix_mgss2idoka6m-1>li:before{content:"" counter(lst-ctn-kix_mgss2idoka6m-1,upper-latin) ". "}.lst-kix_r6vkfvz5od5n-4>li:before{content:"\0025cb   "}ul.lst-kix_vk5294fzi1ce-8{list-style-type:none}ul.lst-kix_vk5294fzi1ce-7{list-style-type:none}ul.lst-kix_vk5294fzi1ce-6{list-style-type:none}ul.lst-kix_hh6mlhac9a3w-5{list-style-type:none}ul.lst-kix_hh6mlhac9a3w-4{list-style-type:none}.lst-kix_bg52hg4v2ft5-6>li:before{content:"\0025cf   "}.lst-kix_bg52hg4v2ft5-8>li:before{content:"\0025a0   "}ul.lst-kix_hh6mlhac9a3w-7{list-style-type:none}ul.lst-kix_hh6mlhac9a3w-6{list-style-type:none}ul.lst-kix_hh6mlhac9a3w-1{list-style-type:none}ul.lst-kix_hh6mlhac9a3w-0{list-style-type:none}ul.lst-kix_hh6mlhac9a3w-3{list-style-type:none}ul.lst-kix_hh6mlhac9a3w-2{list-style-type:none}.lst-kix_2jqw2dfnv53p-4>li:before{content:"\0025cb   "}.lst-kix_u5e0ktun9hhb-2>li:before{content:"" counter(lst-ctn-kix_u5e0ktun9hhb-2,lower-roman) ". "}.lst-kix_u5e0ktun9hhb-4>li:before{content:"" counter(lst-ctn-kix_u5e0ktun9hhb-4,lower-latin) ". "}ul.lst-kix_hh6mlhac9a3w-8{list-style-type:none}ul.lst-kix_wmdnhf474sgk-8{list-style-type:none}ul.lst-kix_wmdnhf474sgk-7{list-style-type:none}ul.lst-kix_wmdnhf474sgk-6{list-style-type:none}ul.lst-kix_wmdnhf474sgk-5{list-style-type:none}ol.lst-kix_5j20aiwerfhd-7.start{counter-reset:lst-ctn-kix_5j20aiwerfhd-7 0}ul.lst-kix_wmdnhf474sgk-0{list-style-type:none}ul.lst-kix_wmdnhf474sgk-4{list-style-type:none}ul.lst-kix_wmdnhf474sgk-3{list-style-type:none}.lst-kix_2l6dfml4zfm9-2>li:before{content:"\0025a0   "}ul.lst-kix_wmdnhf474sgk-2{list-style-type:none}ul.lst-kix_wmdnhf474sgk-1{list-style-type:none}.lst-kix_vk5294fzi1ce-5>li:before{content:"\0025a0   "}.lst-kix_dytpm85i4qaq-1>li:before{content:"\0025cb   "}.lst-kix_dytpm85i4qaq-7>li:before{content:"\0025cb   "}.lst-kix_p2twbyrashqm-1>li:before{content:"\0025cb   "}.lst-kix_2l6dfml4zfm9-8>li:before{content:"\0025a0   "}.lst-kix_43e7b9brr3ij-6>li:before{content:"\0025cf   "}.lst-kix_5w88rs2wtjqd-0>li:before{content:"\0025cf   "}.lst-kix_43e7b9brr3ij-8>li:before{content:"\0025a0   "}.lst-kix_3rsahq87evpc-1>li:before{content:"\0025cb   "}ol.lst-kix_5j20aiwerfhd-5.start{counter-reset:lst-ctn-kix_5j20aiwerfhd-5 0}.lst-kix_krq4n4yenarg-5>li{counter-increment:lst-ctn-kix_krq4n4yenarg-5}.lst-kix_5j20aiwerfhd-1>li:before{content:"" counter(lst-ctn-kix_5j20aiwerfhd-1,lower-latin) ". "}.lst-kix_5j20aiwerfhd-3>li:before{content:"" counter(lst-ctn-kix_5j20aiwerfhd-3,decimal) ". "}.lst-kix_43e7b9brr3ij-0>li:before{content:"\0025cf   "}.lst-kix_3rsahq87evpc-3>li:before{content:"\0025cf   "}.lst-kix_jha6kbfqqd5o-8>li:before{content:"\0025a0   "}.lst-kix_dnezncmpad8v-5>li:before{content:"\0025a0   "}.lst-kix_d9ambift4gw4-5>li:before{content:"\0025a0   "}.lst-kix_4wwjrzyyefv1-1>li:before{content:"\0025cb   "}.lst-kix_d9ambift4gw4-0>li:before{content:"\0025cf   "}.lst-kix_dnezncmpad8v-0>li:before{content:"\0025cf   "}.lst-kix_dnezncmpad8v-8>li:before{content:"\0025a0   "}.lst-kix_c9t4sdk23z1e-1>li:before{content:"\0025cb   "}.lst-kix_4wwjrzyyefv1-6>li:before{content:"\0025cf   "}.lst-kix_mgss2idoka6m-6>li{counter-increment:lst-ctn-kix_mgss2idoka6m-6}.lst-kix_cunig93ipaqk-7>li:before{content:"\0025cb   "}.lst-kix_qvbjpn84g05v-0>li:before{content:"\0025cf   "}ol.lst-kix_a6b21xtlypbx-5.start{counter-reset:lst-ctn-kix_a6b21xtlypbx-5 0}.lst-kix_cunig93ipaqk-2>li:before{content:"\0025a0   "}.lst-kix_qvbjpn84g05v-5>li:before{content:"\0025a0   "}.lst-kix_mgza6d4r1c6u-6>li:before{content:"\0025cf   "}.lst-kix_jha6kbfqqd5o-3>li:before{content:"\0025cf   "}.lst-kix_vw1z3jdppwml-0>li:before{content:"\0025cf   "}.lst-kix_5w88rs2wtjqd-3>li:before{content:"\0025cf   "}.lst-kix_tcf7t0doxn6s-7>li:before{content:"\0025cb   "}ol.lst-kix_xrbxb0icqoxu-0.start{counter-reset:lst-ctn-kix_xrbxb0icqoxu-0 0}.lst-kix_tcf7t0doxn6s-2>li:before{content:"\0025a0   "}ol.lst-kix_5j20aiwerfhd-3.start{counter-reset:lst-ctn-kix_5j20aiwerfhd-3 0}.lst-kix_5w88rs2wtjqd-6>li:before{content:"\0025cf   "}.lst-kix_jha6kbfqqd5o-0>li:before{content:"\0025cf   "}.lst-kix_mgza6d4r1c6u-1>li:before{content:"\0025cb   "}.lst-kix_d9ambift4gw4-8>li:before{content:"\0025a0   "}.lst-kix_3amtkkojfr7l-1>li:before{content:"-  "}.lst-kix_p2twbyrashqm-6>li:before{content:"\0025cf   "}.lst-kix_c9t4sdk23z1e-6>li:before{content:"\0025cf   "}.lst-kix_efd89z9jnu5a-8>li:before{content:"-  "}.lst-kix_3amtkkojfr7l-6>li:before{content:"-  "}.lst-kix_u4aul66k7fry-1>li:before{content:"\0025cb   "}.lst-kix_efd89z9jnu5a-5>li:before{content:"-  "}.lst-kix_a6b21xtlypbx-3>li:before{content:"" counter(lst-ctn-kix_a6b21xtlypbx-3,lower-latin) ") "}.lst-kix_d7gy1vadlia6-5>li:before{content:"\0025a0   "}.lst-kix_a6b21xtlypbx-6>li:before{content:"(" counter(lst-ctn-kix_a6b21xtlypbx-6,lower-roman) ") "}.lst-kix_qvbjpn84g05v-8>li:before{content:"\0025a0   "}.lst-kix_d7gy1vadlia6-2>li:before{content:"\0025a0   "}ol.lst-kix_5j20aiwerfhd-2.start{counter-reset:lst-ctn-kix_5j20aiwerfhd-2 0}.lst-kix_u4aul66k7fry-6>li:before{content:"\0025cf   "}.lst-kix_efd89z9jnu5a-0>li:before{content:"-  "}.lst-kix_5j20aiwerfhd-3>li{counter-increment:lst-ctn-kix_5j20aiwerfhd-3}.lst-kix_mgss2idoka6m-6>li:before{content:"(" counter(lst-ctn-kix_mgss2idoka6m-6,lower-roman) ") "}ol.lst-kix_xrbxb0icqoxu-4.start{counter-reset:lst-ctn-kix_xrbxb0icqoxu-4 0}.lst-kix_k8se1q4z4tp9-3>li:before{content:"\0025cf   "}.lst-kix_u5e0ktun9hhb-1>li{counter-increment:lst-ctn-kix_u5e0ktun9hhb-1}.lst-kix_7pm3iovtnuj2-3>li:before{content:"\0025cf   "}.lst-kix_un80xc3ep54b-4>li:before{content:"\0025cb   "}ol.lst-kix_a6b21xtlypbx-7.start{counter-reset:lst-ctn-kix_a6b21xtlypbx-7 0}.lst-kix_krq4n4yenarg-1>li{counter-increment:lst-ctn-kix_krq4n4yenarg-1}.lst-kix_2jqw2dfnv53p-7>li:before{content:"\0025cb   "}.lst-kix_u5e0ktun9hhb-7>li:before{content:"" counter(lst-ctn-kix_u5e0ktun9hhb-7,lower-latin) ". "}ul.lst-kix_jb5m9ajaek8x-8{list-style-type:none}ul.lst-kix_jb5m9ajaek8x-7{list-style-type:none}ul.lst-kix_jb5m9ajaek8x-6{list-style-type:none}ul.lst-kix_jb5m9ajaek8x-5{list-style-type:none}ul.lst-kix_jb5m9ajaek8x-4{list-style-type:none}.lst-kix_bg52hg4v2ft5-3>li:before{content:"\0025cf   "}ul.lst-kix_jb5m9ajaek8x-3{list-style-type:none}ul.lst-kix_jb5m9ajaek8x-2{list-style-type:none}ul.lst-kix_jb5m9ajaek8x-1{list-style-type:none}ul.lst-kix_jb5m9ajaek8x-0{list-style-type:none}.lst-kix_bfrlw18c0l4c-2>li:before{content:"\0025a0   "}.lst-kix_kh84qhxx6ky4-5>li:before{content:"\0025a0   "}ol.lst-kix_a6b21xtlypbx-4.start{counter-reset:lst-ctn-kix_a6b21xtlypbx-4 0}ol.lst-kix_xrbxb0icqoxu-7.start{counter-reset:lst-ctn-kix_xrbxb0icqoxu-7 0}.lst-kix_dslzgh8s1qzs-3>li:before{content:"\0025cf   "}.lst-kix_krq4n4yenarg-8>li{counter-increment:lst-ctn-kix_krq4n4yenarg-8}ol.lst-kix_xrbxb0icqoxu-6.start{counter-reset:lst-ctn-kix_xrbxb0icqoxu-6 0}.lst-kix_2l6dfml4zfm9-5>li:before{content:"\0025a0   "}.lst-kix_vk5294fzi1ce-8>li:before{content:"\0025a0   "}.lst-kix_5j20aiwerfhd-6>li:before{content:"" counter(lst-ctn-kix_5j20aiwerfhd-6,decimal) ". "}ul.lst-kix_d7gy1vadlia6-8{list-style-type:none}ul.lst-kix_d9ambift4gw4-2{list-style-type:none}ul.lst-kix_kh84qhxx6ky4-1{list-style-type:none}ul.lst-kix_d9ambift4gw4-1{list-style-type:none}ul.lst-kix_kh84qhxx6ky4-2{list-style-type:none}ul.lst-kix_d7gy1vadlia6-6{list-style-type:none}ul.lst-kix_d9ambift4gw4-0{list-style-type:none}ul.lst-kix_kh84qhxx6ky4-3{list-style-type:none}ul.lst-kix_d7gy1vadlia6-7{list-style-type:none}ul.lst-kix_kh84qhxx6ky4-4{list-style-type:none}ul.lst-kix_d7gy1vadlia6-4{list-style-type:none}ul.lst-kix_kh84qhxx6ky4-5{list-style-type:none}ul.lst-kix_d7gy1vadlia6-5{list-style-type:none}ul.lst-kix_kh84qhxx6ky4-6{list-style-type:none}.lst-kix_uy8ee0jk5g05-6>li:before{content:"\0025cf   "}ul.lst-kix_d7gy1vadlia6-2{list-style-type:none}ul.lst-kix_kh84qhxx6ky4-7{list-style-type:none}ul.lst-kix_d7gy1vadlia6-3{list-style-type:none}ul.lst-kix_kh84qhxx6ky4-8{list-style-type:none}ul.lst-kix_d9ambift4gw4-8{list-style-type:none}.lst-kix_ylx0esvd75ha-2>li:before{content:"\0025a0   "}ul.lst-kix_d9ambift4gw4-7{list-style-type:none}ul.lst-kix_d9ambift4gw4-6{list-style-type:none}ul.lst-kix_d9ambift4gw4-5{list-style-type:none}.lst-kix_dytpm85i4qaq-4>li:before{content:"\0025cb   "}ul.lst-kix_d9ambift4gw4-4{list-style-type:none}.lst-kix_krq4n4yenarg-3>li:before{content:"" counter(lst-ctn-kix_krq4n4yenarg-3,decimal) ". "}ul.lst-kix_d9ambift4gw4-3{list-style-type:none}ol.lst-kix_xrbxb0icqoxu-5.start{counter-reset:lst-ctn-kix_xrbxb0icqoxu-5 0}.lst-kix_jb5m9ajaek8x-0>li:before{content:"\0025cf   "}ul.lst-kix_d7gy1vadlia6-0{list-style-type:none}ul.lst-kix_d7gy1vadlia6-1{list-style-type:none}li.li-bullet-0:before{margin-left:-18pt;white-space:nowrap;display:inline-block;min-width:18pt}ul.lst-kix_kh84qhxx6ky4-0{list-style-type:none}.lst-kix_43e7b9brr3ij-3>li:before{content:"\0025cf   "}.lst-kix_jb5m9ajaek8x-8>li:before{content:"\0025a0   "}.lst-kix_3rsahq87evpc-6>li:before{content:"\0025cf   "}ol.lst-kix_a6b21xtlypbx-6.start{counter-reset:lst-ctn-kix_a6b21xtlypbx-6 0}.lst-kix_vw1z3jdppwml-8>li:before{content:"\0025a0   "}ul.lst-kix_mgza6d4r1c6u-1{list-style-type:none}ul.lst-kix_mgza6d4r1c6u-2{list-style-type:none}ul.lst-kix_mgza6d4r1c6u-0{list-style-type:none}ol.lst-kix_mgss2idoka6m-6.start{counter-reset:lst-ctn-kix_mgss2idoka6m-6 0}.lst-kix_xrbxb0icqoxu-0>li:before{content:"" counter(lst-ctn-kix_xrbxb0icqoxu-0,upper-roman) ". "}.lst-kix_xrbxb0icqoxu-2>li:before{content:"" counter(lst-ctn-kix_xrbxb0icqoxu-2,decimal) ". "}ol.lst-kix_xrbxb0icqoxu-8.start{counter-reset:lst-ctn-kix_xrbxb0icqoxu-8 0}.lst-kix_xrbxb0icqoxu-1>li:before{content:"" counter(lst-ctn-kix_xrbxb0icqoxu-1,upper-latin) ". "}.lst-kix_xrbxb0icqoxu-5>li:before{content:"(" counter(lst-ctn-kix_xrbxb0icqoxu-5,lower-latin) ") "}.lst-kix_xrbxb0icqoxu-4>li:before{content:"(" counter(lst-ctn-kix_xrbxb0icqoxu-4,decimal) ") "}.lst-kix_xrbxb0icqoxu-3>li:before{content:"" counter(lst-ctn-kix_xrbxb0icqoxu-3,lower-latin) ") "}ol.lst-kix_5j20aiwerfhd-6.start{counter-reset:lst-ctn-kix_5j20aiwerfhd-6 0}.lst-kix_qzl115totbz-2>li:before{content:"-  "}.lst-kix_qzl115totbz-1>li:before{content:"-  "}.lst-kix_qzl115totbz-3>li:before{content:"-  "}.lst-kix_xrbxb0icqoxu-6>li:before{content:"(" counter(lst-ctn-kix_xrbxb0icqoxu-6,lower-roman) ") "}.lst-kix_qzl115totbz-7>li:before{content:"-  "}.lst-kix_u5e0ktun9hhb-8>li{counter-increment:lst-ctn-kix_u5e0ktun9hhb-8}.lst-kix_qzl115totbz-0>li:before{content:"-  "}.lst-kix_qzl115totbz-8>li:before{content:"-  "}.lst-kix_xrbxb0icqoxu-8>li:before{content:"(" counter(lst-ctn-kix_xrbxb0icqoxu-8,lower-roman) ") "}.lst-kix_xrbxb0icqoxu-7>li:before{content:"(" counter(lst-ctn-kix_xrbxb0icqoxu-7,lower-latin) ") "}.lst-kix_qzl115totbz-6>li:before{content:"-  "}.lst-kix_qzl115totbz-5>li:before{content:"-  "}ul.lst-kix_mgza6d4r1c6u-7{list-style-type:none}ul.lst-kix_mgza6d4r1c6u-8{list-style-type:none}.lst-kix_dslzgh8s1qzs-0>li:before{content:"\0025cf   "}.lst-kix_qzl115totbz-4>li:before{content:"-  "}ul.lst-kix_mgza6d4r1c6u-5{list-style-type:none}ul.lst-kix_mgza6d4r1c6u-6{list-style-type:none}ul.lst-kix_mgza6d4r1c6u-3{list-style-type:none}ul.lst-kix_mgza6d4r1c6u-4{list-style-type:none}.lst-kix_c0f136laxhvb-5>li:before{content:"-  "}.lst-kix_c0f136laxhvb-6>li:before{content:"-  "}.lst-kix_c0f136laxhvb-3>li:before{content:"-  "}.lst-kix_c0f136laxhvb-7>li:before{content:"-  "}.lst-kix_mgss2idoka6m-3>li{counter-increment:lst-ctn-kix_mgss2idoka6m-3}.lst-kix_c0f136laxhvb-1>li:before{content:"-  "}.lst-kix_c0f136laxhvb-2>li:before{content:"-  "}ul.lst-kix_3rsahq87evpc-8{list-style-type:none}.lst-kix_c0f136laxhvb-0>li:before{content:"-  "}.lst-kix_c0f136laxhvb-8>li:before{content:"-  "}ol.lst-kix_mgss2idoka6m-1.start{counter-reset:lst-ctn-kix_mgss2idoka6m-1 0}.lst-kix_wmdnhf474sgk-7>li:before{content:"\0025cb   "}.lst-kix_wmdnhf474sgk-6>li:before{content:"\0025cf   "}.lst-kix_wmdnhf474sgk-8>li:before{content:"\0025a0   "}ol.lst-kix_5j20aiwerfhd-1.start{counter-reset:lst-ctn-kix_5j20aiwerfhd-1 0}ol.lst-kix_xrbxb0icqoxu-3.start{counter-reset:lst-ctn-kix_xrbxb0icqoxu-3 0}.lst-kix_wmdnhf474sgk-4>li:before{content:"\0025cb   "}.lst-kix_wmdnhf474sgk-5>li:before{content:"\0025a0   "}ol.lst-kix_mgss2idoka6m-7{list-style-type:none}ol.lst-kix_mgss2idoka6m-6{list-style-type:none}.lst-kix_wmdnhf474sgk-0>li:before{content:"\0025cf   "}ol.lst-kix_mgss2idoka6m-8{list-style-type:none}.lst-kix_wmdnhf474sgk-3>li:before{content:"\0025cf   "}ul.lst-kix_c0f136laxhvb-4{list-style-type:none}ul.lst-kix_c0f136laxhvb-3{list-style-type:none}ol.lst-kix_mgss2idoka6m-1{list-style-type:none}.lst-kix_wmdnhf474sgk-2>li:before{content:"\0025a0   "}ul.lst-kix_c0f136laxhvb-2{list-style-type:none}ol.lst-kix_mgss2idoka6m-0{list-style-type:none}ul.lst-kix_c0f136laxhvb-1{list-style-type:none}ol.lst-kix_mgss2idoka6m-3{list-style-type:none}.lst-kix_wmdnhf474sgk-1>li:before{content:"\0025cb   "}ul.lst-kix_c0f136laxhvb-0{list-style-type:none}ol.lst-kix_mgss2idoka6m-2{list-style-type:none}ol.lst-kix_mgss2idoka6m-5{list-style-type:none}ol.lst-kix_mgss2idoka6m-4{list-style-type:none}ul.lst-kix_2jqw2dfnv53p-0{list-style-type:none}ul.lst-kix_2jqw2dfnv53p-4{list-style-type:none}ul.lst-kix_c0f136laxhvb-8{list-style-type:none}ul.lst-kix_2jqw2dfnv53p-3{list-style-type:none}ul.lst-kix_c0f136laxhvb-7{list-style-type:none}ul.lst-kix_2jqw2dfnv53p-2{list-style-type:none}ul.lst-kix_c0f136laxhvb-6{list-style-type:none}ul.lst-kix_2jqw2dfnv53p-1{list-style-type:none}ul.lst-kix_c0f136laxhvb-5{list-style-type:none}ul.lst-kix_2jqw2dfnv53p-8{list-style-type:none}ul.lst-kix_2jqw2dfnv53p-7{list-style-type:none}ul.lst-kix_2jqw2dfnv53p-6{list-style-type:none}.lst-kix_a6b21xtlypbx-4>li{counter-increment:lst-ctn-kix_a6b21xtlypbx-4}ul.lst-kix_2jqw2dfnv53p-5{list-style-type:none}.lst-kix_xrbxb0icqoxu-8>li{counter-increment:lst-ctn-kix_xrbxb0icqoxu-8}.lst-kix_c0f136laxhvb-4>li:before{content:"-  "}.lst-kix_7pm3iovtnuj2-0>li:before{content:"\0025cf   "}ol.lst-kix_krq4n4yenarg-0.start{counter-reset:lst-ctn-kix_krq4n4yenarg-0 0}.lst-kix_7pm3iovtnuj2-2>li:before{content:"\0025a0   "}.lst-kix_7pm3iovtnuj2-6>li:before{content:"\0025cf   "}.lst-kix_7pm3iovtnuj2-4>li:before{content:"\0025cb   "}.lst-kix_7pm3iovtnuj2-8>li:before{content:"\0025a0   "}.lst-kix_kh84qhxx6ky4-8>li:before{content:"\0025a0   "}.lst-kix_xrbxb0icqoxu-0>li{counter-increment:lst-ctn-kix_xrbxb0icqoxu-0}ol.lst-kix_u5e0ktun9hhb-4.start{counter-reset:lst-ctn-kix_u5e0ktun9hhb-4 0}.lst-kix_kh84qhxx6ky4-2>li:before{content:"\0025a0   "}.lst-kix_kh84qhxx6ky4-4>li:before{content:"\0025cb   "}.lst-kix_kh84qhxx6ky4-6>li:before{content:"\0025cf   "}.lst-kix_xrbxb0icqoxu-1>li{counter-increment:lst-ctn-kix_xrbxb0icqoxu-1}.lst-kix_kh84qhxx6ky4-0>li:before{content:"\0025cf   "}.lst-kix_dslzgh8s1qzs-2>li:before{content:"\0025a0   "}.lst-kix_krq4n4yenarg-8>li:before{content:"" counter(lst-ctn-kix_krq4n4yenarg-8,lower-roman) ". "}.lst-kix_ylx0esvd75ha-5>li:before{content:"\0025a0   "}.lst-kix_ylx0esvd75ha-1>li:before{content:"\0025cb   "}.lst-kix_uy8ee0jk5g05-1>li:before{content:"\0025cb   "}.lst-kix_krq4n4yenarg-6>li:before{content:"" counter(lst-ctn-kix_krq4n4yenarg-6,decimal) ". "}.lst-kix_dslzgh8s1qzs-4>li:before{content:"\0025cb   "}.lst-kix_ylx0esvd75ha-7>li:before{content:"\0025cb   "}.lst-kix_uy8ee0jk5g05-5>li:before{content:"\0025a0   "}.lst-kix_krq4n4yenarg-2>li:before{content:"" counter(lst-ctn-kix_krq4n4yenarg-2,lower-roman) ". "}.lst-kix_uy8ee0jk5g05-3>li:before{content:"\0025cf   "}.lst-kix_uy8ee0jk5g05-7>li:before{content:"\0025cb   "}.lst-kix_krq4n4yenarg-0>li:before{content:"" counter(lst-ctn-kix_krq4n4yenarg-0,lower-roman) ". "}.lst-kix_krq4n4yenarg-4>li:before{content:"" counter(lst-ctn-kix_krq4n4yenarg-4,lower-latin) ". "}.lst-kix_a6b21xtlypbx-5>li{counter-increment:lst-ctn-kix_a6b21xtlypbx-5}.lst-kix_dslzgh8s1qzs-6>li:before{content:"\0025cf   "}.lst-kix_dslzgh8s1qzs-8>li:before{content:"\0025a0   "}.lst-kix_ylx0esvd75ha-3>li:before{content:"\0025cf   "}ol.lst-kix_a6b21xtlypbx-0.start{counter-reset:lst-ctn-kix_a6b21xtlypbx-0 0}.lst-kix_d9ambift4gw4-3>li:before{content:"\0025cf   "}.lst-kix_c9t4sdk23z1e-4>li:before{content:"\0025cb   "}.lst-kix_mgza6d4r1c6u-8>li:before{content:"\0025a0   "}.lst-kix_mgza6d4r1c6u-7>li:before{content:"\0025cb   "}.lst-kix_cunig93ipaqk-4>li:before{content:"\0025cb   "}.lst-kix_cunig93ipaqk-8>li:before{content:"\0025a0   "}.lst-kix_c9t4sdk23z1e-3>li:before{content:"\0025cf   "}.lst-kix_c9t4sdk23z1e-7>li:before{content:"\0025cb   "}.lst-kix_cunig93ipaqk-5>li:before{content:"\0025a0   "}.lst-kix_c9t4sdk23z1e-0>li:before{content:"\0025cf   "}.lst-kix_c9t4sdk23z1e-8>li:before{content:"\0025a0   "}.lst-kix_d9ambift4gw4-2>li:before{content:"\0025a0   "}ul.lst-kix_dnezncmpad8v-0{list-style-type:none}ul.lst-kix_dytpm85i4qaq-2{list-style-type:none}.lst-kix_mgza6d4r1c6u-3>li:before{content:"\0025cf   "}.lst-kix_cunig93ipaqk-0>li:before{content:"\0025cf   "}ul.lst-kix_dytpm85i4qaq-3{list-style-type:none}.lst-kix_qvbjpn84g05v-2>li:before{content:"\0025a0   "}ul.lst-kix_dytpm85i4qaq-0{list-style-type:none}.lst-kix_cunig93ipaqk-1>li:before{content:"\0025cb   "}ul.lst-kix_dytpm85i4qaq-1{list-style-type:none}ul.lst-kix_dnezncmpad8v-6{list-style-type:none}ul.lst-kix_cunig93ipaqk-0{list-style-type:none}ul.lst-kix_dnezncmpad8v-5{list-style-type:none}.lst-kix_mgza6d4r1c6u-4>li:before{content:"\0025cb   "}ul.lst-kix_dnezncmpad8v-8{list-style-type:none}ul.lst-kix_dnezncmpad8v-7{list-style-type:none}ul.lst-kix_dnezncmpad8v-2{list-style-type:none}.lst-kix_qvbjpn84g05v-3>li:before{content:"\0025cf   "}ul.lst-kix_dnezncmpad8v-1{list-style-type:none}ul.lst-kix_dnezncmpad8v-4{list-style-type:none}.lst-kix_u5e0ktun9hhb-7>li{counter-increment:lst-ctn-kix_u5e0ktun9hhb-7}ul.lst-kix_dnezncmpad8v-3{list-style-type:none}ul.lst-kix_bfrlw18c0l4c-1{list-style-type:none}.lst-kix_p2twbyrashqm-8>li:before{content:"\0025a0   "}ul.lst-kix_cunig93ipaqk-8{list-style-type:none}ul.lst-kix_bfrlw18c0l4c-2{list-style-type:none}ul.lst-kix_cunig93ipaqk-7{list-style-type:none}.lst-kix_5w88rs2wtjqd-4>li:before{content:"\0025cb   "}.lst-kix_p2twbyrashqm-7>li:before{content:"\0025cb   "}ul.lst-kix_cunig93ipaqk-6{list-style-type:none}ul.lst-kix_bfrlw18c0l4c-0{list-style-type:none}ul.lst-kix_cunig93ipaqk-5{list-style-type:none}ul.lst-kix_cunig93ipaqk-4{list-style-type:none}ul.lst-kix_cunig93ipaqk-3{list-style-type:none}ul.lst-kix_cunig93ipaqk-2{list-style-type:none}ul.lst-kix_cunig93ipaqk-1{list-style-type:none}ul.lst-kix_dytpm85i4qaq-6{list-style-type:none}ul.lst-kix_dytpm85i4qaq-7{list-style-type:none}ul.lst-kix_dytpm85i4qaq-4{list-style-type:none}ul.lst-kix_dytpm85i4qaq-5{list-style-type:none}.lst-kix_p2twbyrashqm-3>li:before{content:"\0025cf   "}.lst-kix_5w88rs2wtjqd-5>li:before{content:"\0025a0   "}ul.lst-kix_dytpm85i4qaq-8{list-style-type:none}ul.lst-kix_dslzgh8s1qzs-4{list-style-type:none}ul.lst-kix_dslzgh8s1qzs-3{list-style-type:none}ul.lst-kix_dslzgh8s1qzs-6{list-style-type:none}ul.lst-kix_dslzgh8s1qzs-5{list-style-type:none}ul.lst-kix_dslzgh8s1qzs-0{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-3.start{counter-reset:lst-ctn-kix_u5e0ktun9hhb-3 0}.lst-kix_mgza6d4r1c6u-0>li:before{content:"\0025cf   "}ul.lst-kix_dslzgh8s1qzs-2{list-style-type:none}.lst-kix_mgss2idoka6m-4>li{counter-increment:lst-ctn-kix_mgss2idoka6m-4}ul.lst-kix_dslzgh8s1qzs-1{list-style-type:none}.lst-kix_p2twbyrashqm-4>li:before{content:"\0025cb   "}.lst-kix_d9ambift4gw4-7>li:before{content:"\0025cb   "}ul.lst-kix_bfrlw18c0l4c-7{list-style-type:none}ul.lst-kix_bfrlw18c0l4c-8{list-style-type:none}ul.lst-kix_dslzgh8s1qzs-8{list-style-type:none}ul.lst-kix_bfrlw18c0l4c-5{list-style-type:none}ul.lst-kix_dslzgh8s1qzs-7{list-style-type:none}ul.lst-kix_bfrlw18c0l4c-6{list-style-type:none}.lst-kix_5w88rs2wtjqd-8>li:before{content:"\0025a0   "}ul.lst-kix_bfrlw18c0l4c-3{list-style-type:none}.lst-kix_d9ambift4gw4-6>li:before{content:"\0025cf   "}ul.lst-kix_bfrlw18c0l4c-4{list-style-type:none}.lst-kix_u4aul66k7fry-0>li:before{content:"\0025cf   "}.lst-kix_a6b21xtlypbx-0>li:before{content:"" counter(lst-ctn-kix_a6b21xtlypbx-0,upper-roman) ". "}.lst-kix_a6b21xtlypbx-1>li:before{content:"" counter(lst-ctn-kix_a6b21xtlypbx-1,upper-latin) ". "}.lst-kix_a6b21xtlypbx-5>li:before{content:"(" counter(lst-ctn-kix_a6b21xtlypbx-5,lower-latin) ") "}.lst-kix_u4aul66k7fry-8>li:before{content:"\0025a0   "}.lst-kix_a6b21xtlypbx-8>li:before{content:"(" counter(lst-ctn-kix_a6b21xtlypbx-8,lower-roman) ") "}.lst-kix_a6b21xtlypbx-4>li:before{content:"(" counter(lst-ctn-kix_a6b21xtlypbx-4,decimal) ") "}.lst-kix_d7gy1vadlia6-8>li:before{content:"\0025a0   "}.lst-kix_d7gy1vadlia6-7>li:before{content:"\0025cb   "}ol.lst-kix_a6b21xtlypbx-8{list-style-type:none}.lst-kix_qvbjpn84g05v-6>li:before{content:"\0025cf   "}ol.lst-kix_a6b21xtlypbx-6{list-style-type:none}ol.lst-kix_a6b21xtlypbx-7{list-style-type:none}.lst-kix_d7gy1vadlia6-4>li:before{content:"\0025cb   "}.lst-kix_qvbjpn84g05v-7>li:before{content:"\0025cb   "}.lst-kix_d7gy1vadlia6-3>li:before{content:"\0025cf   "}.lst-kix_u4aul66k7fry-7>li:before{content:"\0025cb   "}ol.lst-kix_a6b21xtlypbx-4{list-style-type:none}ol.lst-kix_a6b21xtlypbx-5{list-style-type:none}ol.lst-kix_a6b21xtlypbx-2{list-style-type:none}ol.lst-kix_a6b21xtlypbx-3{list-style-type:none}.lst-kix_u4aul66k7fry-4>li:before{content:"\0025cb   "}ol.lst-kix_a6b21xtlypbx-0{list-style-type:none}.lst-kix_d7gy1vadlia6-0>li:before{content:"\0025cf   "}ol.lst-kix_a6b21xtlypbx-1{list-style-type:none}.lst-kix_u4aul66k7fry-3>li:before{content:"\0025cf   "}.lst-kix_mgss2idoka6m-8>li:before{content:"(" counter(lst-ctn-kix_mgss2idoka6m-8,lower-roman) ") "}.lst-kix_un80xc3ep54b-6>li:before{content:"\0025cf   "}ul.lst-kix_ylx0esvd75ha-4{list-style-type:none}ul.lst-kix_ylx0esvd75ha-3{list-style-type:none}.lst-kix_mgss2idoka6m-4>li:before{content:"(" counter(lst-ctn-kix_mgss2idoka6m-4,decimal) ") "}ul.lst-kix_ylx0esvd75ha-6{list-style-type:none}.lst-kix_7pm3iovtnuj2-1>li:before{content:"\0025cb   "}ul.lst-kix_ylx0esvd75ha-5{list-style-type:none}ul.lst-kix_ylx0esvd75ha-8{list-style-type:none}ul.lst-kix_ylx0esvd75ha-7{list-style-type:none}.lst-kix_u5e0ktun9hhb-0>li{counter-increment:lst-ctn-kix_u5e0ktun9hhb-0}.lst-kix_7pm3iovtnuj2-5>li:before{content:"\0025a0   "}.lst-kix_mgss2idoka6m-0>li:before{content:"" counter(lst-ctn-kix_mgss2idoka6m-0,upper-roman) ". "}.lst-kix_bg52hg4v2ft5-1>li:before{content:"\0025cb   "}.lst-kix_un80xc3ep54b-2>li:before{content:"\0025a0   "}.lst-kix_kh84qhxx6ky4-7>li:before{content:"\0025cb   "}.lst-kix_u5e0ktun9hhb-1>li:before{content:"" counter(lst-ctn-kix_u5e0ktun9hhb-1,lower-latin) ". "}.lst-kix_u5e0ktun9hhb-5>li:before{content:"" counter(lst-ctn-kix_u5e0ktun9hhb-5,lower-roman) ". "}.lst-kix_bg52hg4v2ft5-5>li:before{content:"\0025a0   "}ul.lst-kix_u4aul66k7fry-0{list-style-type:none}.lst-kix_kh84qhxx6ky4-3>li:before{content:"\0025cf   "}ul.lst-kix_u4aul66k7fry-4{list-style-type:none}ul.lst-kix_u4aul66k7fry-3{list-style-type:none}ul.lst-kix_u4aul66k7fry-2{list-style-type:none}ul.lst-kix_u4aul66k7fry-1{list-style-type:none}ul.lst-kix_u4aul66k7fry-8{list-style-type:none}ul.lst-kix_u4aul66k7fry-7{list-style-type:none}ul.lst-kix_u4aul66k7fry-6{list-style-type:none}ul.lst-kix_u4aul66k7fry-5{list-style-type:none}ul.lst-kix_3rsahq87evpc-5{list-style-type:none}ul.lst-kix_3rsahq87evpc-4{list-style-type:none}ul.lst-kix_3rsahq87evpc-7{list-style-type:none}ul.lst-kix_3rsahq87evpc-6{list-style-type:none}ul.lst-kix_3rsahq87evpc-1{list-style-type:none}ul.lst-kix_ylx0esvd75ha-0{list-style-type:none}ul.lst-kix_3rsahq87evpc-0{list-style-type:none}ul.lst-kix_3rsahq87evpc-3{list-style-type:none}ul.lst-kix_ylx0esvd75ha-2{list-style-type:none}ul.lst-kix_3rsahq87evpc-2{list-style-type:none}ul.lst-kix_ylx0esvd75ha-1{list-style-type:none}.lst-kix_ylx0esvd75ha-4>li:before{content:"\0025cb   "}.lst-kix_dslzgh8s1qzs-1>li:before{content:"\0025cb   "}.lst-kix_dytpm85i4qaq-2>li:before{content:"\0025a0   "}.lst-kix_dytpm85i4qaq-6>li:before{content:"\0025cf   "}.lst-kix_ylx0esvd75ha-8>li:before{content:"\0025a0   "}.lst-kix_dslzgh8s1qzs-5>li:before{content:"\0025a0   "}.lst-kix_ylx0esvd75ha-0>li:before{content:"\0025cf   "}.lst-kix_uy8ee0jk5g05-0>li:before{content:"\0025cf   "}.lst-kix_krq4n4yenarg-1>li:before{content:"" counter(lst-ctn-kix_krq4n4yenarg-1,lower-latin) ". "}.lst-kix_krq4n4yenarg-5>li:before{content:"" counter(lst-ctn-kix_krq4n4yenarg-5,lower-roman) ". "}.lst-kix_5j20aiwerfhd-8>li:before{content:"" counter(lst-ctn-kix_5j20aiwerfhd-8,lower-roman) ". "}.lst-kix_p2twbyrashqm-0>li:before{content:"\0025cf   "}ul.lst-kix_43e7b9brr3ij-7{list-style-type:none}.lst-kix_5w88rs2wtjqd-1>li:before{content:"\0025cb   "}ul.lst-kix_43e7b9brr3ij-8{list-style-type:none}.lst-kix_uy8ee0jk5g05-4>li:before{content:"\0025cb   "}.lst-kix_mgss2idoka6m-2>li{counter-increment:lst-ctn-kix_mgss2idoka6m-2}ul.lst-kix_43e7b9brr3ij-3{list-style-type:none}ul.lst-kix_43e7b9brr3ij-4{list-style-type:none}ul.lst-kix_43e7b9brr3ij-5{list-style-type:none}ul.lst-kix_43e7b9brr3ij-6{list-style-type:none}ul.lst-kix_43e7b9brr3ij-0{list-style-type:none}ul.lst-kix_43e7b9brr3ij-1{list-style-type:none}ul.lst-kix_43e7b9brr3ij-2{list-style-type:none}.lst-kix_5j20aiwerfhd-0>li:before{content:"" counter(lst-ctn-kix_5j20aiwerfhd-0,lower-roman) ". "}.lst-kix_uy8ee0jk5g05-8>li:before{content:"\0025a0   "}.lst-kix_5j20aiwerfhd-4>li:before{content:"" counter(lst-ctn-kix_5j20aiwerfhd-4,lower-latin) ". "}.lst-kix_5j20aiwerfhd-2>li{counter-increment:lst-ctn-kix_5j20aiwerfhd-2}.lst-kix_krq4n4yenarg-2>li{counter-increment:lst-ctn-kix_krq4n4yenarg-2}.lst-kix_uscd8mjnf5fw-5>li:before{content:"\0025a0   "}.lst-kix_uscd8mjnf5fw-4>li:before{content:"\0025cb   "}.lst-kix_uscd8mjnf5fw-2>li:before{content:"\0025a0   "}ul.lst-kix_efd89z9jnu5a-6{list-style-type:none}ul.lst-kix_efd89z9jnu5a-5{list-style-type:none}ul.lst-kix_efd89z9jnu5a-4{list-style-type:none}ul.lst-kix_efd89z9jnu5a-3{list-style-type:none}ul.lst-kix_efd89z9jnu5a-2{list-style-type:none}ul.lst-kix_efd89z9jnu5a-1{list-style-type:none}ul.lst-kix_efd89z9jnu5a-0{list-style-type:none}ul.lst-kix_qvbjpn84g05v-8{list-style-type:none}ul.lst-kix_qvbjpn84g05v-7{list-style-type:none}ul.lst-kix_qvbjpn84g05v-6{list-style-type:none}ul.lst-kix_qvbjpn84g05v-5{list-style-type:none}.lst-kix_uscd8mjnf5fw-7>li:before{content:"\0025cb   "}ul.lst-kix_qvbjpn84g05v-4{list-style-type:none}ul.lst-kix_qvbjpn84g05v-3{list-style-type:none}ul.lst-kix_qvbjpn84g05v-2{list-style-type:none}ul.lst-kix_qvbjpn84g05v-1{list-style-type:none}ul.lst-kix_qvbjpn84g05v-0{list-style-type:none}.lst-kix_dqnkgejf3n11-7>li:before{content:"\0025cb   "}ol.lst-kix_mgss2idoka6m-0.start{counter-reset:lst-ctn-kix_mgss2idoka6m-0 0}ul.lst-kix_efd89z9jnu5a-8{list-style-type:none}ul.lst-kix_efd89z9jnu5a-7{list-style-type:none}.lst-kix_dqnkgejf3n11-1>li:before{content:"\0025cb   "}.lst-kix_dqnkgejf3n11-2>li:before{content:"\0025a0   "}.lst-kix_7ly1sailhnxn-0>li:before{content:"\0025cf   "}.lst-kix_dqnkgejf3n11-4>li:before{content:"\0025cb   "}ol.lst-kix_krq4n4yenarg-4.start{counter-reset:lst-ctn-kix_krq4n4yenarg-4 0}ol.lst-kix_u5e0ktun9hhb-0.start{counter-reset:lst-ctn-kix_u5e0ktun9hhb-0 0}.lst-kix_7ly1sailhnxn-5>li:before{content:"\0025a0   "}.lst-kix_vk5294fzi1ce-1>li:before{content:"\0025cb   "}.lst-kix_7ly1sailhnxn-8>li:before{content:"\0025a0   "}.lst-kix_7ly1sailhnxn-3>li:before{content:"\0025cf   "}.lst-kix_i0lzlnefeb2-0>li:before{content:"\0025cf   "}.lst-kix_i0lzlnefeb2-3>li:before{content:"\0025cf   "}.lst-kix_7ly1sailhnxn-6>li:before{content:"\0025cf   "}.lst-kix_i0lzlnefeb2-1>li:before{content:"\0025cb   "}ul.lst-kix_2l6dfml4zfm9-1{list-style-type:none}ul.lst-kix_2l6dfml4zfm9-0{list-style-type:none}.lst-kix_i0lzlnefeb2-8>li:before{content:"\0025a0   "}.lst-kix_r6vkfvz5od5n-6>li:before{content:"\0025cf   "}.lst-kix_i0lzlnefeb2-6>li:before{content:"\0025cf   "}.lst-kix_r6vkfvz5od5n-8>li:before{content:"\0025a0   "}ul.lst-kix_2l6dfml4zfm9-8{list-style-type:none}ul.lst-kix_2l6dfml4zfm9-7{list-style-type:none}ul.lst-kix_2l6dfml4zfm9-6{list-style-type:none}ul.lst-kix_2l6dfml4zfm9-5{list-style-type:none}ul.lst-kix_2l6dfml4zfm9-4{list-style-type:none}ul.lst-kix_2l6dfml4zfm9-3{list-style-type:none}ul.lst-kix_2l6dfml4zfm9-2{list-style-type:none}.lst-kix_mgss2idoka6m-7>li:before{content:"(" counter(lst-ctn-kix_mgss2idoka6m-7,lower-latin) ") "}ol.lst-kix_u5e0ktun9hhb-7.start{counter-reset:lst-ctn-kix_u5e0ktun9hhb-7 0}.lst-kix_mgss2idoka6m-5>li:before{content:"(" counter(lst-ctn-kix_mgss2idoka6m-5,lower-latin) ") "}.lst-kix_r6vkfvz5od5n-0>li:before{content:"\0025cf   "}.lst-kix_krq4n4yenarg-0>li{counter-increment:lst-ctn-kix_krq4n4yenarg-0}.lst-kix_un80xc3ep54b-5>li:before{content:"\0025a0   "}.lst-kix_u5e0ktun9hhb-0>li:before{content:"" counter(lst-ctn-kix_u5e0ktun9hhb-0,decimal) ". "}.lst-kix_un80xc3ep54b-3>li:before{content:"\0025cf   "}.lst-kix_2jqw2dfnv53p-8>li:before{content:"\0025a0   "}.lst-kix_u5e0ktun9hhb-6>li:before{content:"" counter(lst-ctn-kix_u5e0ktun9hhb-6,decimal) ". "}.lst-kix_bg52hg4v2ft5-2>li:before{content:"\0025a0   "}.lst-kix_bg52hg4v2ft5-4>li:before{content:"\0025cb   "}.lst-kix_2jqw2dfnv53p-6>li:before{content:"\0025cf   "}.lst-kix_5j20aiwerfhd-4>li{counter-increment:lst-ctn-kix_5j20aiwerfhd-4}.lst-kix_u5e0ktun9hhb-2>li{counter-increment:lst-ctn-kix_u5e0ktun9hhb-2}.lst-kix_u5e0ktun9hhb-8>li:before{content:"" counter(lst-ctn-kix_u5e0ktun9hhb-8,lower-roman) ". "}.lst-kix_dytpm85i4qaq-5>li:before{content:"\0025a0   "}.lst-kix_2l6dfml4zfm9-4>li:before{content:"\0025cb   "}.lst-kix_vk5294fzi1ce-7>li:before{content:"\0025cb   "}.lst-kix_a6b21xtlypbx-6>li{counter-increment:lst-ctn-kix_a6b21xtlypbx-6}.lst-kix_43e7b9brr3ij-4>li:before{content:"\0025cb   "}.lst-kix_2l6dfml4zfm9-6>li:before{content:"\0025cf   "}.lst-kix_5j20aiwerfhd-7>li:before{content:"" counter(lst-ctn-kix_5j20aiwerfhd-7,lower-latin) ". "}.lst-kix_dytpm85i4qaq-3>li:before{content:"\0025cf   "}ol.lst-kix_krq4n4yenarg-2{list-style-type:none}ol.lst-kix_krq4n4yenarg-1{list-style-type:none}ol.lst-kix_krq4n4yenarg-0{list-style-type:none}ol.lst-kix_krq4n4yenarg-6{list-style-type:none}ol.lst-kix_krq4n4yenarg-5{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-0{list-style-type:none}ol.lst-kix_krq4n4yenarg-4{list-style-type:none}ol.lst-kix_krq4n4yenarg-3{list-style-type:none}.lst-kix_mgss2idoka6m-5>li{counter-increment:lst-ctn-kix_mgss2idoka6m-5}ol.lst-kix_krq4n4yenarg-8{list-style-type:none}ol.lst-kix_krq4n4yenarg-7{list-style-type:none}.lst-kix_3rsahq87evpc-7>li:before{content:"\0025cb   "}.lst-kix_u5e0ktun9hhb-3>li{counter-increment:lst-ctn-kix_u5e0ktun9hhb-3}.lst-kix_43e7b9brr3ij-2>li:before{content:"\0025a0   "}.lst-kix_5j20aiwerfhd-5>li:before{content:"" counter(lst-ctn-kix_5j20aiwerfhd-5,lower-roman) ". "}.lst-kix_3rsahq87evpc-5>li:before{content:"\0025a0   "}ol.lst-kix_u5e0ktun9hhb-5.start{counter-reset:lst-ctn-kix_u5e0ktun9hhb-5 0}ul.lst-kix_un80xc3ep54b-8{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-6{list-style-type:none}ul.lst-kix_un80xc3ep54b-7{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-5{list-style-type:none}ul.lst-kix_un80xc3ep54b-6{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-8{list-style-type:none}ul.lst-kix_un80xc3ep54b-5{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-7{list-style-type:none}ul.lst-kix_un80xc3ep54b-4{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-2{list-style-type:none}ul.lst-kix_un80xc3ep54b-3{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-1{list-style-type:none}.lst-kix_5j20aiwerfhd-5>li{counter-increment:lst-ctn-kix_5j20aiwerfhd-5}ul.lst-kix_un80xc3ep54b-2{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-4{list-style-type:none}ul.lst-kix_un80xc3ep54b-1{list-style-type:none}ol.lst-kix_u5e0ktun9hhb-3{list-style-type:none}ul.lst-kix_un80xc3ep54b-0{list-style-type:none}ul.lst-kix_4wwjrzyyefv1-0{list-style-type:none}.lst-kix_dnezncmpad8v-4>li:before{content:"\0025cb   "}.lst-kix_cunig93ipaqk-6>li:before{content:"\0025cf   "}.lst-kix_c9t4sdk23z1e-5>li:before{content:"\0025a0   "}ul.lst-kix_4wwjrzyyefv1-2{list-style-type:none}ul.lst-kix_4wwjrzyyefv1-1{list-style-type:none}ul.lst-kix_4wwjrzyyefv1-4{list-style-type:none}.lst-kix_d9ambift4gw4-1>li:before{content:"\0025cb   "}ul.lst-kix_4wwjrzyyefv1-3{list-style-type:none}ul.lst-kix_4wwjrzyyefv1-6{list-style-type:none}.lst-kix_d9ambift4gw4-4>li:before{content:"\0025cb   "}ul.lst-kix_4wwjrzyyefv1-5{list-style-type:none}.lst-kix_dnezncmpad8v-1>li:before{content:"\0025cb   "}.lst-kix_jha6kbfqqd5o-7>li:before{content:"\0025cb   "}.lst-kix_c9t4sdk23z1e-2>li:before{content:"\0025a0   "}.lst-kix_qvbjpn84g05v-1>li:before{content:"\0025cb   "}.lst-kix_mgza6d4r1c6u-2>li:before{content:"\0025a0   "}.lst-kix_xrbxb0icqoxu-7>li{counter-increment:lst-ctn-kix_xrbxb0icqoxu-7}.lst-kix_4wwjrzyyefv1-5>li:before{content:"\0025a0   "}.lst-kix_mgza6d4r1c6u-5>li:before{content:"\0025a0   "}.lst-kix_qvbjpn84g05v-4>li:before{content:"\0025cb   "}.lst-kix_4wwjrzyyefv1-2>li:before{content:"\0025a0   "}.lst-kix_jha6kbfqqd5o-4>li:before{content:"\0025cb   "}.lst-kix_cunig93ipaqk-3>li:before{content:"\0025cf   "}.lst-kix_tcf7t0doxn6s-6>li:before{content:"\0025cf   "}.lst-kix_5w88rs2wtjqd-2>li:before{content:"\0025a0   "}.lst-kix_tcf7t0doxn6s-3>li:before{content:"\0025cf   "}ol.lst-kix_krq4n4yenarg-1.start{counter-reset:lst-ctn-kix_krq4n4yenarg-1 0}.lst-kix_p2twbyrashqm-2>li:before{content:"\0025a0   "}ul.lst-kix_5w88rs2wtjqd-5{list-style-type:none}ul.lst-kix_5w88rs2wtjqd-4{list-style-type:none}ul.lst-kix_5w88rs2wtjqd-7{list-style-type:none}ul.lst-kix_5w88rs2wtjqd-6{list-style-type:none}ul.lst-kix_5w88rs2wtjqd-1{list-style-type:none}ul.lst-kix_5w88rs2wtjqd-0{list-style-type:none}ul.lst-kix_5w88rs2wtjqd-3{list-style-type:none}ul.lst-kix_5w88rs2wtjqd-2{list-style-type:none}.lst-kix_5w88rs2wtjqd-7>li:before{content:"\0025cb   "}.lst-kix_p2twbyrashqm-5>li:before{content:"\0025a0   "}ol.lst-kix_krq4n4yenarg-7.start{counter-reset:lst-ctn-kix_krq4n4yenarg-7 0}.lst-kix_3amtkkojfr7l-2>li:before{content:"-  "}ol.lst-kix_mgss2idoka6m-3.start{counter-reset:lst-ctn-kix_mgss2idoka6m-3 0}ul.lst-kix_5w88rs2wtjqd-8{list-style-type:none}ul.lst-kix_bg52hg4v2ft5-0{list-style-type:none}.lst-kix_3amtkkojfr7l-5>li:before{content:"-  "}ul.lst-kix_bg52hg4v2ft5-2{list-style-type:none}ul.lst-kix_bg52hg4v2ft5-1{list-style-type:none}ul.lst-kix_bg52hg4v2ft5-4{list-style-type:none}.lst-kix_efd89z9jnu5a-4>li:before{content:"-  "}ul.lst-kix_bg52hg4v2ft5-3{list-style-type:none}ul.lst-kix_bg52hg4v2ft5-6{list-style-type:none}.lst-kix_krq4n4yenarg-3>li{counter-increment:lst-ctn-kix_krq4n4yenarg-3}ul.lst-kix_bg52hg4v2ft5-5{list-style-type:none}ul.lst-kix_bg52hg4v2ft5-8{list-style-type:none}.lst-kix_a6b21xtlypbx-2>li:before{content:"" counter(lst-ctn-kix_a6b21xtlypbx-2,decimal) ". "}ul.lst-kix_bg52hg4v2ft5-7{list-style-type:none}.lst-kix_2jqw2dfnv53p-0>li:before{content:"\0025cf   "}.lst-kix_d7gy1vadlia6-6>li:before{content:"\0025cf   "}ol.lst-kix_krq4n4yenarg-8.start{counter-reset:lst-ctn-kix_krq4n4yenarg-8 0}.lst-kix_a6b21xtlypbx-7>li:before{content:"(" counter(lst-ctn-kix_a6b21xtlypbx-7,lower-latin) ") "}.lst-kix_2jqw2dfnv53p-3>li:before{content:"\0025cf   "}ol.lst-kix_mgss2idoka6m-2.start{counter-reset:lst-ctn-kix_mgss2idoka6m-2 0}.lst-kix_d7gy1vadlia6-1>li:before{content:"\0025cb   "}ol.lst-kix_krq4n4yenarg-2.start{counter-reset:lst-ctn-kix_krq4n4yenarg-2 0}.lst-kix_efd89z9jnu5a-1>li:before{content:"-  "}.lst-kix_u4aul66k7fry-5>li:before{content:"\0025a0   "}ol.lst-kix_mgss2idoka6m-8.start{counter-reset:lst-ctn-kix_mgss2idoka6m-8 0}ol.lst-kix_u5e0ktun9hhb-8.start{counter-reset:lst-ctn-kix_u5e0ktun9hhb-8 0}.lst-kix_u4aul66k7fry-2>li:before{content:"\0025a0   "}.lst-kix_un80xc3ep54b-8>li:before{content:"\0025a0   "}ul.lst-kix_p2twbyrashqm-0{list-style-type:none}.lst-kix_mgss2idoka6m-2>li:before{content:"" counter(lst-ctn-kix_mgss2idoka6m-2,decimal) ". "}ul.lst-kix_p2twbyrashqm-2{list-style-type:none}ul.lst-kix_p2twbyrashqm-1{list-style-type:none}ul.lst-kix_p2twbyrashqm-4{list-style-type:none}.lst-kix_a6b21xtlypbx-3>li{counter-increment:lst-ctn-kix_a6b21xtlypbx-3}ul.lst-kix_p2twbyrashqm-3{list-style-type:none}.lst-kix_r6vkfvz5od5n-3>li:before{content:"\0025cf   "}.lst-kix_7pm3iovtnuj2-7>li:before{content:"\0025cb   "}.lst-kix_k8se1q4z4tp9-7>li:before{content:"\0025cb   "}ol.lst-kix_krq4n4yenarg-3.start{counter-reset:lst-ctn-kix_krq4n4yenarg-3 0}ul.lst-kix_k8se1q4z4tp9-3{list-style-type:none}.lst-kix_bg52hg4v2ft5-7>li:before{content:"\0025cb   "}ol.lst-kix_mgss2idoka6m-7.start{counter-reset:lst-ctn-kix_mgss2idoka6m-7 0}ul.lst-kix_k8se1q4z4tp9-4{list-style-type:none}ul.lst-kix_k8se1q4z4tp9-5{list-style-type:none}ul.lst-kix_k8se1q4z4tp9-6{list-style-type:none}ul.lst-kix_k8se1q4z4tp9-7{list-style-type:none}ul.lst-kix_k8se1q4z4tp9-8{list-style-type:none}ul.lst-kix_7ly1sailhnxn-0{list-style-type:none}ul.lst-kix_7ly1sailhnxn-2{list-style-type:none}.lst-kix_u5e0ktun9hhb-3>li:before{content:"" counter(lst-ctn-kix_u5e0ktun9hhb-3,decimal) ". "}ul.lst-kix_7ly1sailhnxn-1{list-style-type:none}ul.lst-kix_7ly1sailhnxn-4{list-style-type:none}ul.lst-kix_7ly1sailhnxn-3{list-style-type:none}.lst-kix_5j20aiwerfhd-1>li{counter-increment:lst-ctn-kix_5j20aiwerfhd-1}ul.lst-kix_7ly1sailhnxn-6{list-style-type:none}.lst-kix_un80xc3ep54b-0>li:before{content:"\0025cf   "}ul.lst-kix_7ly1sailhnxn-5{list-style-type:none}ul.lst-kix_p2twbyrashqm-6{list-style-type:none}ul.lst-kix_p2twbyrashqm-5{list-style-type:none}ul.lst-kix_p2twbyrashqm-8{list-style-type:none}.lst-kix_mgss2idoka6m-1>li{counter-increment:lst-ctn-kix_mgss2idoka6m-1}ul.lst-kix_p2twbyrashqm-7{list-style-type:none}.lst-kix_2l6dfml4zfm9-1>li:before{content:"\0025cb   "}ul.lst-kix_k8se1q4z4tp9-0{list-style-type:none}.lst-kix_bfrlw18c0l4c-6>li:before{content:"\0025cf   "}.lst-kix_vk5294fzi1ce-4>li:before{content:"\0025cb   "}ul.lst-kix_k8se1q4z4tp9-1{list-style-type:none}ul.lst-kix_k8se1q4z4tp9-2{list-style-type:none}.lst-kix_kh84qhxx6ky4-1>li:before{content:"\0025cb   "}ol.lst-kix_krq4n4yenarg-5.start{counter-reset:lst-ctn-kix_krq4n4yenarg-5 0}.lst-kix_ylx0esvd75ha-6>li:before{content:"\0025cf   "}.lst-kix_xrbxb0icqoxu-4>li{counter-increment:lst-ctn-kix_xrbxb0icqoxu-4}.lst-kix_dytpm85i4qaq-0>li:before{content:"\0025cf   "}.lst-kix_dytpm85i4qaq-8>li:before{content:"\0025a0   "}.lst-kix_krq4n4yenarg-7>li:before{content:"" counter(lst-ctn-kix_krq4n4yenarg-7,lower-latin) ". "}ol.lst-kix_mgss2idoka6m-4.start{counter-reset:lst-ctn-kix_mgss2idoka6m-4 0}.lst-kix_uy8ee0jk5g05-2>li:before{content:"\0025a0   "}.lst-kix_dslzgh8s1qzs-7>li:before{content:"\0025cb   "}.lst-kix_43e7b9brr3ij-7>li:before{content:"\0025cb   "}.lst-kix_3rsahq87evpc-2>li:before{content:"\0025a0   "}.lst-kix_mgss2idoka6m-8>li{counter-increment:lst-ctn-kix_mgss2idoka6m-8}ol.lst-kix_mgss2idoka6m-5.start{counter-reset:lst-ctn-kix_mgss2idoka6m-5 0}ol.lst-kix_krq4n4yenarg-6.start{counter-reset:lst-ctn-kix_krq4n4yenarg-6 0}.lst-kix_vw1z3jdppwml-4>li:before{content:"\0025cb   "}ul.lst-kix_vw1z3jdppwml-4{list-style-type:none}ul.lst-kix_vw1z3jdppwml-5{list-style-type:none}ul.lst-kix_vw1z3jdppwml-2{list-style-type:none}.lst-kix_u5e0ktun9hhb-6>li{counter-increment:lst-ctn-kix_u5e0ktun9hhb-6}ul.lst-kix_vw1z3jdppwml-3{list-style-type:none}ul.lst-kix_vw1z3jdppwml-0{list-style-type:none}ul.lst-kix_vw1z3jdppwml-1{list-style-type:none}ul.lst-kix_4wwjrzyyefv1-8{list-style-type:none}ul.lst-kix_4wwjrzyyefv1-7{list-style-type:none}.lst-kix_jb5m9ajaek8x-4>li:before{content:"\0025cb   "}.lst-kix_5j20aiwerfhd-8>li{counter-increment:lst-ctn-kix_5j20aiwerfhd-8}ul.lst-kix_vw1z3jdppwml-8{list-style-type:none}ul.lst-kix_vw1z3jdppwml-6{list-style-type:none}.lst-kix_5j20aiwerfhd-2>li:before{content:"" counter(lst-ctn-kix_5j20aiwerfhd-2,lower-roman) ". "}ul.lst-kix_vw1z3jdppwml-7{list-style-type:none}ol{margin:0;padding:0}table td,table th{padding:0}.c89{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:153pt;border-top-color:#000000;border-bottom-style:solid}.c90{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:57.8pt;border-top-color:#000000;border-bottom-style:solid}.c87{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:50.2pt;border-top-color:#000000;border-bottom-style:solid}.c109{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:92.5pt;border-top-color:#000000;border-bottom-style:solid}.c96{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:53.2pt;border-top-color:#000000;border-bottom-style:solid}.c66{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:54.8pt;border-top-color:#000000;border-bottom-style:solid}.c105{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:210.8pt;border-top-color:#000000;border-bottom-style:solid}.c104{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:40.8pt;border-top-color:#000000;border-bottom-style:solid}.c107{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:56.2pt;border-top-color:#000000;border-bottom-style:solid}.c112{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:468pt;border-top-color:#000000;border-bottom-style:solid}.c114{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:111pt;border-top-color:#000000;border-bottom-style:solid}.c46{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:51pt;border-top-color:#000000;border-bottom-style:solid}.c80{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:234pt;border-top-color:#000000;border-bottom-style:solid}.c39{border-right-style:solid;padding:5pt 5pt 5pt 5pt;border-bottom-color:#000000;border-top-width:1pt;border-right-width:1pt;border-left-color:#000000;vertical-align:top;border-right-color:#000000;border-left-width:1pt;border-top-style:solid;border-left-style:solid;border-bottom-width:1pt;width:51.8pt;border-top-color:#000000;border-bottom-style:solid}.c1{margin-left:72pt;padding-top:0pt;text-indent:36pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left;margin-right:-63pt}.c2{background-color:#ffffff;color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:7pt;font-family:"Courier New";font-style:normal}.c56{-webkit-text-decoration-skip:none;font-weight:400;text-decoration:underline;vertical-align:baseline;text-decoration-skip-ink:none;font-size:9pt;font-style:normal}.c3{background-color:#ffffff;padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c17{padding-top:18pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c67{margin-left:-36pt;padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:center}.c0{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c13{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left;margin-right:-63pt}.c21{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:8pt;font-family:"Arial";font-style:normal}.c41{padding-top:20pt;padding-bottom:6pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c10{color:#000000;font-weight:700;text-decoration:none;vertical-align:baseline;font-size:11pt;font-family:"Arial";font-style:normal}.c103{padding-top:0pt;padding-bottom:3pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c9{color:#000000;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:8pt;font-family:"Courier New";font-style:normal}.c61{padding-top:0pt;text-indent:-36pt;padding-bottom:10pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c60{padding-top:16pt;padding-bottom:4pt;line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.c6{margin-left:36pt;padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c11{color:#080808;font-weight:400;text-decoration:none;vertical-align:baseline;font-size:8pt;font-family:"Courier New";font-style:normal}.c115{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:center}.c120{padding-top:3pt;padding-bottom:3pt;line-height:1.50001;orphans:2;widows:2;text-align:left}.c68{padding-top:24pt;padding-bottom:6pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c111{margin-left:22pt;padding-top:11pt;text-indent:-22pt;padding-bottom:11pt;line-height:1.0;text-align:left}.c95{padding-top:0pt;padding-bottom:10pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c100{margin-left:22pt;padding-top:0pt;text-indent:-22pt;padding-bottom:11pt;line-height:1.0;text-align:left}.c70{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:right}.c32{padding-top:12pt;padding-bottom:12pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c14{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:left}.c72{padding-top:0pt;padding-bottom:0pt;line-height:1.15;orphans:2;widows:2;text-align:justify}.c4{font-size:8pt;font-family:"Courier New";color:#067d17;font-weight:400}.c91{border-spacing:0;border-collapse:collapse;margin-right:auto}.c119{background-color:#f7f7f8;color:#374151;font-weight:400;font-family:"Roboto"}.c30{color:#000000;font-weight:400;font-size:16pt;font-family:"Arial"}.c57{color:#000000;font-weight:400;font-size:7pt;font-family:"Courier New"}.c76{padding-top:0pt;padding-bottom:0pt;line-height:1.15;text-align:left}.c34{padding-top:3pt;padding-bottom:0pt;line-height:1.0;text-align:left}.c25{font-size:11pt;font-style:normal;color:#000000;font-weight:400}.c47{color:#000000;font-weight:700;font-size:12pt;font-family:"Arial"}.c84{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:center}.c93{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:justify}.c7{font-size:8pt;font-family:"Courier New";color:#0033b3;font-weight:400}.c28{background-color:#ffffff;font-size:7pt;font-family:"Courier New";font-weight:400}.c5{padding-top:0pt;padding-bottom:0pt;line-height:1.0;text-align:left}.c12{font-size:8pt;font-family:"Courier New";font-weight:400}.c29{text-decoration:none;vertical-align:baseline;font-style:normal}.c63{text-decoration-skip-ink:none;-webkit-text-decoration-skip:none;text-decoration:underline}.c37{color:#000000;font-weight:400;text-decoration:none}.c53{font-weight:400;font-size:9pt;font-family:"Courier New"}.c31{font-weight:400;font-size:11pt;font-family:"Arial"}.c73{font-size:7pt;font-family:"Courier New";font-weight:400}.c69{font-weight:400;font-size:20pt}.c62{vertical-align:baseline;font-family:"Arial"}.c85{font-weight:400;font-size:9pt}.c43{font-size:14pt;font-weight:700}.c48{color:#000000;font-family:"Arial"}.c8{text-indent:36pt;height:11pt}.c117{max-width:468pt;padding:72pt 72pt 72pt 72pt}.c58{color:#8c8c8c;vertical-align:baseline}.c78{font-weight:400;font-size:8pt}.c27{text-indent:36pt;margin-right:-72pt}.c77{padding:0;margin:0}.c106{font-weight:400;font-size:14pt}.c118{font-weight:400;font-size:26pt}.c18{color:inherit;text-decoration:inherit}.c121{orphans:2;widows:2}.c86{color:#434343;font-family:"Arial"}.c54{font-weight:400;font-size:10pt}.c71{font-size:8pt;font-family:"Courier New"}.c64{color:#000000;vertical-align:baseline}.c19{padding-left:0pt}.c23{color:#174ad4}.c36{font-style:italic}.c38{margin-left:108pt}.c110{page-break-after:avoid}.c33{margin-left:72pt}.c40{margin-right:-72pt}.c55{margin-right:-36pt}.c35{text-indent:36pt}.c98{font-family:"Arial"}.c45{text-decoration:none}.c79{height:21pt}.c75{background-color:#ffffff}.c94{color:#1155cc}.c51{margin-left:18pt}.c49{margin-left:36pt}.c108{color:#ff0000}.c88{font-size:12pt}.c24{background-color:#ffffb3}.c74{background-color:#ffff00}.c65{background-color:#80b1d3}.c26{background-color:#e6f5c9}.c22{background-color:#f4cae4}.c113{height:16pt}.c92{font-size:10pt}.c15{height:11pt}.c116{margin-left:-36pt}.c97{margin-left:144pt}.c83{height:0pt}.c59{color:#8c8c8c}.c82{color:#000000}.c81{height:20pt}.c101{color:#222222}.c102{font-size:9pt}.c50{font-weight:700}.c52{font-size:11pt}.c16{color:#080808}.c99{font-size:8pt}.c42{background-color:#fdcdac}.c44{color:#1d1c1d}.c20{background-color:#cbd5e8}.title{padding-top:0pt;color:#000000;font-size:26pt;padding-bottom:3pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}.subtitle{padding-top:0pt;color:#666666;font-size:15pt;padding-bottom:16pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}li{color:#000000;font-size:11pt;font-family:"Arial"}p{margin:0;color:#000000;font-size:11pt;font-family:"Arial"}h1{padding-top:20pt;color:#000000;font-size:20pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h2{padding-top:18pt;color:#000000;font-size:16pt;padding-bottom:6pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h3{padding-top:16pt;color:#434343;font-size:14pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h4{padding-top:14pt;color:#666666;font-size:12pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h5{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;orphans:2;widows:2;text-align:left}h6{padding-top:12pt;color:#666666;font-size:11pt;padding-bottom:4pt;font-family:"Arial";line-height:1.15;page-break-after:avoid;font-style:italic;orphans:2;widows:2;text-align:left}</style></head><body class="c75 c117 doc-content"><div><p class="c103 title" id="h.797eogw6xzg5"><span class="c92">mzIdentML extension for crosslinking data&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;June 24</span><span class="c92">,</span><span class="c92">&nbsp;</span><span class="c92">202</span><span class="c92">4</span></p></div><p class="c103 title" id="h.x9ule1p5drxb"><span class="c29 c48 c54">PSI draft recommendation&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p><p class="c103 title" id="h.h7ijm745xi5r"><span class="c29 c48 c54">PSI Proteomics Informatics Working Group</span></p><p class="c14"><span class="c0">Status: FINAL</span></p><p class="c70"><span class="c37 c62 c36 c52">Colin W. Combe, University of Edinburgh</span></p><p class="c70"><span class="c37 c62 c36 c52">Lars Kolbowski, Technische Universit&auml;t Berlin</span></p><p class="c70"><span class="c36">Lutz Fische</span><span class="c37 c62 c36 c52">r, Technische Universit&auml;t Berlin</span></p><p class="c70"><span class="c37 c62 c36 c52">Ville Koskinen, Matrix Science</span></p><p class="c70"><span class="c37 c62 c36 c52">Joshua Klein, University of Boston</span></p><p class="c70"><span class="c37 c36 c52 c62">Alexander Leitner, ETH Zurich</span></p><p class="c70"><span class="c37 c62 c36 c52">Juan Antonio Vizca&iacute;no, European Bioinformatics Institute</span></p><p class="c70"><span class="c37 c62 c36 c52">Andrew R. Jones, University of Liverpool</span></p><p class="c70"><span class="c36">Juri Rappsilber, University of </span><span class="c36">Edinburgh</span><span class="c37 c62 c36 c52">&nbsp;&amp; Technische Universit&auml;t Berlin</span></p><p class="c70 c15"><span class="c29 c48 c54"></span></p><p class="c70 c15"><span class="c29 c48 c54"></span></p><p class="c70 c15"><span class="c0"></span></p><p class="c70"><span>June 24</span><span>,</span><span class="c0">&nbsp;2024</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c50">mzIdentML: exchange format for peptides and proteins identified from mass spectra</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c50">Extension for crosslinking approaches</span><span class="c0">&nbsp;</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>(This extension is an </span><span class="c36">addendum</span><span>&nbsp;to mzIdentML </span><span>version</span><span>&nbsp;1.3.</span><span>0</span><span class="c0">)</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c25 c63 c62">Status of This Document</span></p><p class="c14"><span class="c0">&nbsp;</span></p><p class="c14"><span class="c0">This document presents a final specification for the mzIdentML data format developed by the HUPO Proteomics Standards Initiative. Distribution is unlimited.</span></p><p class="c14"><span class="c0">&nbsp;</span></p><p class="c14"><span class="c25 c63 c62">Version of This Document</span></p><p class="c14"><span>The current version of this document is: version </span><span>1.0.0 final June 24,</span><span>&nbsp;</span><span class="c0">2024.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c10">Abstract</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The Human Proteome Organisation (HUPO) Proteomics Standards Initiative (PSI) defines </span><span>community</span><span>&nbsp;standards for data representation in proteomics to facilitate data comparison, exchange and verification. This document defines the extension of the mzIdentML data standard to describe the outputs of proteomics search engines or similar software for the identification of cross</span><span>linked</span><span>&nbsp;peptides.</span></p><hr style="page-break-before:always;display:none;"><p class="c14 c15"><span class="c10"></span></p><p class="c14"><span class="c50">Table of </span><span class="c50">Contents</span></p><p class="c34"><span class="c10"><a class="c18" href="#h.59xy9dd3agji">1. Introduction&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;2</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.xrjh0y40cx7p">1.1 Background&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.3qciuuk6z6a7">1.2. Supported Crosslinking Use Cases&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;3</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.4va9ec38iojw">1.3 Document Structure and Changes from mzIdentML Version 1.2.0 to 1.3.0 to Support Crosslinking Results&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;5</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.paqjt69g1ycc">1.4 Availability of Documentation and Example Files&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;6</a></span></p><p class="c34"><span class="c10"><a class="c18" href="#h.buyb3l8upkzl">2. Controlled Vocabularies for Encoding Crosslinks&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;7</a></span></p><p class="c34"><span class="c10"><a class="c18" href="#h.28v06xr0dl6y">3. Encoding the Results of Crosslinking Searches&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;8</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.qt8ma1g7dinj">3.1. Introduction&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;8</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.w4d0zt5miiwi">3.2 &lt;SpectrumIdentificationProtocol&gt; Elements&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;10</a></span></p><p class="c34 c49"><span class="c0"><a class="c18" href="#h.oap1xkt9k8zx">3.2.1 Additional Search Parameters&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;10</a></span></p><p class="c34 c49"><span class="c0"><a class="c18" href="#h.1wnij58lg2z0">3.2.2 Modification Parameters&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;11</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.wi4u0wz85u5v">3.3 Encoding Crosslinked Peptides in the Element &lt;SequenceCollection&gt;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;15</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.2qumsicj7vey">3.4 Encoding Identified Crosslinks in &lt;SpectrumIdentificationItem&gt; Elements&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;19</a></span></p><p class="c34 c49"><span class="c0"><a class="c18" href="#h.59aaerpg4g8l">3.4.1 Identifications of Crosslinked Peptides&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;19</a></span></p><p class="c34 c49"><span class="c0"><a class="c18" href="#h.byui0pnkdhua">3.4.2 Identifications of Noncovalently Associated Peptides&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;21</a></span></p><p class="c34 c49"><span class="c0"><a class="c18" href="#h.8omxepcmkvte">3.4.3 Identifications of an Internally Linked Peptide&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;23</a></span></p><p class="c34"><span class="c10"><a class="c18" href="#h.eskj4bsv28fu">4. Scores and Thresholds&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;24</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.4y36xqq2ftho">4.1 Introduction&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;24</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.gkox7noip2j8">4.2 Thresholds&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;25</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.k0tz6lupv5ds">4.3 Scores&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;28</a></span></p><p class="c34 c49"><span class="c0"><a class="c18" href="#h.m17dfs7z5hvc">4.3.1 Match Level Scores&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;28</a></span></p><p class="c34 c49"><span class="c0"><a class="c18" href="#h.ly8cjum3t06t">4.3.2 Peptide Level Scores&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;28</a></span></p><p class="c34 c49"><span class="c0"><a class="c18" href="#h.78go5sdh2ohx">4.3.3 Interaction Level Scores (Unique Residue-Pairs and PPI)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;31</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.7if82p50b87k">4.4 FDR Specific Comments&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;38</a></span></p><p class="c34"><span class="c10"><a class="c18" href="#h.s698zhk8zhgu">5. Unsupported Use Cases and Future Directions&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;39</a></span></p><p class="c34"><span class="c10"><a class="c18" href="#h.t2z1zrknogih">6. Appendix I. Comparison of rules for &ldquo;crosslink donor&rdquo; and &ldquo;crosslink acceptor&rdquo; depending on context&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;40</a></span></p><p class="c34"><span class="c10"><a class="c18" href="#h.8qg7o6e2yw4k">7. Appendix II. Example encodings of crosslinker reagents as &lt;SearchModification&gt; elements&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;42</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.z7iqem5xmct9">7.1 Bis(sulfosuccinimidyl) suberate (BS3)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;42</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.hfq5x99v2tmv">7.2 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride (EDC)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;43</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.utvdaw73ics">7.3 &nbsp;(NHS-Diazirine) succinimidyl 4,4&#39;-azipentanoate (SDA)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;44</a></span></p><p class="c34 c51"><span class="c0"><a class="c18" href="#h.uhdz3v19usr5">7.4 Disuccinimidyl sulfoxide (DSSO)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;45</a></span></p><p class="c34"><span class="c10"><a class="c18" href="#h.qlyprh427938">8. Authors Information&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;48</a></span></p><p class="c34"><span class="c10"><a class="c18" href="#h.jk40ojgvmqu0">9. Glossary&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;48</a></span></p><p class="c34"><span class="c10"><a class="c18" href="#h.iqbq77xzfts5">10. References&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;50</a></span></p><p class="c34"><span class="c10"><a class="c18" href="#h.hftkqxpufrbu">11. Intellectual Property Statement&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;50</a></span></p><p class="c34"><span class="c10"><a class="c18" href="#h.64x63egbmg6f">Copyright Notice&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;51</a></span></p><p class="c14 c15"><span class="c10"></span></p><hr style="page-break-before:always;display:none;"><h1 class="c41" id="h.59xy9dd3agji"><span class="c43">1. </span><span class="c43">Introduction</span></h1><h2 class="c17" id="h.xrjh0y40cx7p"><span class="c50 c88">1.1 </span><span class="c88 c50">Background</span></h2><p class="c14"><span>A large number of proteomics search engines are available, each outputting results in a variety of file formats.</span><span>&nbsp;</span><span>mzIdentML </span><span class="c37"><a class="c18" href="https://www.google.com/url?q=https://paperpile.com/c/YrD2gH/vjf16&amp;sa=D&amp;source=editors&amp;ust=1719221372766206&amp;usg=AOvVaw0TYclQs5Rw6xXNnqB3PIHx">[1]</a></span><span>&nbsp;is a HUPO-PSI endorsed community standard that provides a common file format for identification results</span><span>. This extension document has been released in parallel with mzIdentML version 1.3. </span><span>mzIdentML version 1.3</span><span>&nbsp;</span><span class="c0">supports extensions for additional features or use cases which can be described in additional documents, rather than editing the original specification document. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">This mzIdentML 1.3.0 extension document provides further information on the encoding of crosslinking Mass Spectrometry (MS) results in mzIdentML. It has two aims: </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>1- Extend the crosslinking use cases supported by mzIdentML version 1.2.0 (2017) </span><span>to cover what is currently required by the state-of-the-art in the field</span><span class="c0">. Particular attention is paid to workflows using cleavable crosslinkers. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">2- Provide further clarification and documentation on how to encode crosslinking data in an mzIdentML file.</span></p><p class="c14 c15"><span class="c0"></span></p><h2 class="c17" id="h.3qciuuk6z6a7"><span class="c47 c29">1.2. Supported Crosslinking Use Cases</span></h2><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">Already supported use case in mzIdentML 1.2.0:</span></p><p class="c14"><span>- Two </span><span>crosslinked</span><span class="c0">&nbsp;peptides (the crosslinking product that is typically of most interest).</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">New use cases supported in this extension (part of mzIdentML 1.3.0):</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>1- Reporting cleavable crosslinkers. MS-cleavable </span><span>crosslinkers</span><span>&nbsp;can cleave upon activation in the mass spectrometer, releasing the individual peptides and thus enabling their individual analysis. </span><span class="c0">Section 7.11 of the main mzIdentML 1.3.0 specification gives a new mechanism for encoding identifications based on multiple spectra, using this is essential for some cleavable crosslinker workflows. Such a workflow is used as an example in Section 7.11 of the main specification document.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>2- Internally linked peptides (commonly known as &ldquo;looplinks&rdquo;). </span><span>Internally linked peptides</span><span class="c0">&nbsp;are cases where both ends of the crosslinker are within a single peptide, not between two copies of the same peptide. This type of crosslinking product is therefore necessarily intramolecular.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>3- Noncovalently associated peptides. Some spectra may show the fragmentation of two different peptides which were not crosslinked but stayed associated due to noncovalent interactions </span><span class="c37"><a class="c18" href="https://www.google.com/url?q=https://paperpile.com/c/YrD2gH/IanGW&amp;sa=D&amp;source=editors&amp;ust=1719221372767995&amp;usg=AOvVaw3s3PqAbYWEG864dItR_lfE">[2]</a></span><span>. </span><span>Both peptides together appear as a single precursor species in the instrument, as opposed to &lsquo;chimeric&rsquo; spectra where a single peptide is selected as precursor but additional peptide(s) fall within the same selection window.</span><span class="c0">&nbsp;Identifying these noncovalently associated peptides may improve the accuracy of the results as it can prevent them from being misidentified as crosslinked peptides.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>4</span><span>- Additionally, the encoding of scores applicable to crosslinking MS results, and their corresponding thresholds, has been </span><span>clarified</span><span>&nbsp;and improved.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">An overview of the different crosslinking product types and their support in mzIdentML is given in Figure 1. For discussion of the product types that are not supported in this version of mzIdentML (crosslinkers with more than two reactive groups, higher order crosslinked peptides) see &sect;5.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c29 c85 c48"></span></p><a id="t.86b6e40ee49344f36759caeb04d857476fab402c"></a><a id="t.0"></a><table class="c91"><tr class="c79"><td class="c96" colspan="1" rowspan="2"><p class="c5"><span class="c21">no crosslinker reaction</span></p></td><td class="c104" colspan="1" rowspan="2"><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 42.58px; height: 20.83px;"><img alt="" src="images/image3.jpg" style="width: 42.58px; height: 20.83px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c29 c48 c78">linear peptide / free</span></p><p class="c5"><span class="c29 c48 c78">peptide</span></p></td><td class="c105" colspan="4" rowspan="2"><p class="c5 c15"><span class="c29 c48 c78"></span></p></td><td class="c39" colspan="1" rowspan="2"><p class="c93"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 43.00px; height: 15.10px;"><img alt="" src="images/image3.jpg" style="width: 43.00px; height: 21.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c93"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 43.00px; height: 14.16px;"><img alt="" src="images/image3.jpg" style="width: 43.00px; height: 21.00px; margin-left: 0.00px; margin-top: -6.84px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c29 c48 c78">non-</span></p><p class="c5"><span class="c29 c48 c78">covalently associated peptides</span></p></td><td class="c114" colspan="2" rowspan="2"><p class="c5 c15"><span class="c29 c48 c78"></span></p></td></tr><tr class="c79"></tr><tr class="c79"><td class="c96" colspan="1" rowspan="2"><p class="c5"><span class="c21">crosslinker</span></p><p class="c5"><span class="c21">reaction</span></p></td><td class="c104" colspan="1" rowspan="2"><p class="c5 c15"><span class="c29 c48 c78"></span></p></td><td class="c39" colspan="1" rowspan="2"><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 56.50px; height: 33.00px;"><img alt="" src="images/image1.jpg" style="width: 56.50px; height: 33.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c29 c48 c78">crosslinker modified peptide</span></p><p class="c5"><span class="c29 c48 c78">(monolink</span></p><p class="c5"><span class="c29 c48 c78">or dead-end link)</span></p></td><td class="c90" colspan="1" rowspan="2"><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 63.00px; height: 41.33px;"><img alt="" src="images/image6.jpg" style="width: 63.00px; height: 41.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c29 c48 c78">crosslinked peptides</span></p></td><td class="c87" colspan="1" rowspan="2"><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 46.71px; height: 34.80px;"><img alt="" src="images/image5.png" style="width: 46.71px; height: 34.80px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5 c15"><span class="c29 c48 c78"></span></p><p class="c5"><span class="c29 c48 c78">cleavable</span></p><p class="c5"><span class="c29 c48 c78">crosslinker</span></p></td><td class="c46" colspan="1" rowspan="2"><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 56.50px; height: 26.01px;"><img alt="" src="images/image8.jpg" style="width: 56.50px; height: 26.01px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c29 c48 c78">internally linked peptide (looplink)</span></p></td><td class="c39" colspan="1" rowspan="2"><p class="c5 c15"><span class="c29 c48 c78"></span></p></td><td class="c66" colspan="1" rowspan="2"><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 63.00px; height: 45.33px;"><img alt="" src="images/image7.jpg" style="width: 63.00px; height: 45.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c29 c48 c78">crosslinked peptides from crosslinkers with more than two reactive groups </span></p></td><td class="c107" colspan="1" rowspan="2"><p class="c5"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 63.00px; height: 41.33px;"><img alt="" src="images/image4.jpg" style="width: 63.00px; height: 41.33px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c5"><span class="c29 c48 c78">higher order</span></p><p class="c5"><span class="c29 c48 c78">crosslinked peptides</span></p></td></tr><tr class="c79"></tr><tr class="c79"><td class="c96" colspan="1" rowspan="1"><p class="c5"><span class="c21">mzIdentML</span></p><p class="c5"><span class="c21">version supporting</span></p></td><td class="c109" colspan="2" rowspan="1"><p class="c84 c15"><span class="c21"></span></p><p class="c84"><span class="c50 c99">1.1.</span><span class="c50 c99">0</span></p></td><td class="c90" colspan="1" rowspan="1"><p class="c84 c15"><span class="c21"></span></p><p class="c84"><span class="c21">1.2.0</span></p></td><td class="c89" colspan="3" rowspan="1"><p class="c84 c15"><span class="c21"></span></p><p class="c84"><span class="c21">1.3.0</span></p></td><td class="c114" colspan="2" rowspan="1"><p class="c84 c15"><span class="c21"></span></p><p class="c84"><span class="c21">Unsupported</span></p></td></tr></table><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c50">Figure 1. Summary of mzIdentML support for crosslinking product types.</span><span class="c0">&nbsp;</span></p><p class="c14 c15"><span class="c0"></span></p><hr style="page-break-before:always;display:none;"><h2 class="c17 c113" id="h.lnmfm43ouxko"><span class="c29 c30"></span></h2><h2 class="c17" id="h.4va9ec38iojw"><span class="c88 c50">1.3</span><span class="c88 c50">&nbsp;Document Structure</span><span class="c88 c50">&nbsp;</span><span class="c88 c50">and</span><span class="c88 c50">&nbsp;</span><span class="c88 c50">Changes</span><span class="c47 c29">&nbsp;from mzIdentML Version 1.2.0 to 1.3.0 to Support Crosslinking Results</span></h2><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">mzIdentML version 1.3.0 makes two significant changes: (i) a new mechanism for encoding identifications based on multiple spectra, including the retiral of the old method for doing this; (ii) the ability to supplement the specification with extension documents covering specific use cases. The general guidance on the mzIdentML file format given in the main specification document all applies here, with this extension document giving guidance on the use of the controlled vocabulary (CV) terms specific to crosslinking.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The previously supported crosslinking use case was described in the main mzIdentML 1.2.0 specification document. In mzIdentML 1.3.0 this information has been moved to this extension document but it remains unchanged. The only change to the previous version 1.2.0 support for crosslinking is regarding identifications based on multiple spectra, this change is covered in Section 7.11 of the main mzIdentML 1.3.0 specification document. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">The new use cases supported in this extension (version 1.0.0, compatible with mzIdentML version 1.3.0) are explained in detail in the following Sections of this document (denoted with an &sect;). All of them have new controlled vocabulary terms associated with them. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">Section 3 of this extension document is organised on the basis of elements in the mzIdentML schema, see Figure 2. For each of the new use cases, the list below states the relevant sections of this document and the new CV terms.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>1- Reporting cleavable crosslinkers. </span><span>See &sect;3.</span><span class="c0">2.2. Three new CV terms have been created related to encoding the derivatives of cleavable crosslinkers:</span></p><ul class="c77 lst-kix_dnezncmpad8v-1 start"><li class="c14 c19 c33 li-bullet-0"><span>&ldquo;</span><span>cleavable </span><span class="c0">crosslinker stub&rdquo; (MS:1003346),</span></li><li class="c14 c19 c33 li-bullet-0"><span class="c0">&ldquo;Unimod derivative code&rdquo; (MS:1003347),</span></li><li class="c14 c19 c33 li-bullet-0"><span class="c0">&ldquo;crosslinker cleavage characteristics&rdquo; (MS:1003390).</span></li></ul><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>2- Internally linked peptides (a.k.a. &ldquo;looplinks&rdquo;). See Sections &sect;3.3 &amp; &sect;3.4</span><span>.3</span><span class="c0">. One new CV term has been created to allow the encoding: </span></p><ul class="c77 lst-kix_dnezncmpad8v-1"><li class="c14 c19 c33 li-bullet-0"><span class="c0">&ldquo;looplink spectrum identification item&rdquo; (MS:1003329).</span></li></ul><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>3- Noncovalently associated peptides. See &sect;3.2.1 </span><span>and &sect;3.4.2</span><span class="c0">. Two new CV terms have been created related to noncovalently associated peptides:</span></p><ul class="c77 lst-kix_dnezncmpad8v-1"><li class="c14 c19 c33 li-bullet-0"><span class="c0">&ldquo;noncovalently associated peptides search&rdquo; (MS:1003330),</span></li><li class="c14 c19 c33 li-bullet-0"><span class="c0">&ldquo;noncovalently associated peptides spectrum identification item&rdquo; (MS:1003331).</span></li></ul><p class="c14 c33 c15"><span class="c0"></span></p><p class="c14"><span>4</span><span>-</span><span>&nbsp;Improvements in the encoding of scores and thresholds related to crosslinking results. See Section &sect;4. Seven CV terms have been created:</span></p><ul class="c77 lst-kix_u4aul66k7fry-1 start"><li class="c14 c19 c33 li-bullet-0"><span class="c0">&quot;crosslinked PSM-level global FDR&quot; (MS:1003337),</span></li><li class="c32 c19 c33 li-bullet-0"><span class="c0">&ldquo;peptide-pair sequence-level global FDR&rdquo; (MS:1003338),</span></li><li class="c32 c19 c33 li-bullet-0"><span class="c0">&ldquo;peptide-pair passes threshold&rdquo; (MS:1003339),</span></li><li class="c32 c19 c33 li-bullet-0"><span class="c0">&ldquo;residue-pair passes threshold&rdquo; (MS:1003340),</span></li><li class="c32 c19 c33 li-bullet-0"><span>&ldquo;protein-protein interaction passes threshold&rdquo; (MS:1003341)</span><span class="c0">,</span></li><li class="c32 c19 c33 li-bullet-0"><span>&ldquo;regular expression for whether interaction score derived from crosslinking passes threshold&rdquo; </span><span>(MS:1003342)</span><span class="c0">,</span></li><li class="c76 c19 c33 li-bullet-0"><span>&ldquo;FDR applied separately to self crosslinks and protein heteromeric crosslinks&rdquo; </span><span>(MS:1003343)</span><span class="c0">,</span></li><li class="c76 c19 c33 li-bullet-0"><span>&ldquo;</span><span>residue pair ref</span><span class="c0">&rdquo; (MS:1003344).</span></li><li class="c76 c19 c33 li-bullet-0"><span class="c0">&ldquo;regular expression for residue-pair ref&rdquo; (MS:1003345)</span></li></ul><p class="c76 c15"><span class="c0"></span></p><p class="c15 c76"><span class="c0"></span></p><h2 class="c17" id="h.paqjt69g1ycc"><span class="c88 c50">1.</span><span class="c88 c50">4</span><span class="c88 c50">&nbsp;</span><span class="c47 c29">Availability of Documentation and Example Files</span></h2><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>All documents in their most recent form are available at the mzIdentML GitHub project (</span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=https://github.com/HUPO-PSI/mzIdentML/tree/master/specification_document&amp;sa=D&amp;source=editors&amp;ust=1719221372784915&amp;usg=AOvVaw3plMdWELXDq8Pdz2TUA1ts">https://github.com/HUPO-PSI/mzIdentML/tree/master/specification_document</a></span><span class="c0">). </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The example files supporting this extension document are available at </span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=https://github.com/HUPO-PSI/mzIdentML/blob/master/examples/1_3examples/crosslinking/&amp;sa=D&amp;source=editors&amp;ust=1719221372785472&amp;usg=AOvVaw196O1CvF8_8eDafwQunmDg">https://github.com/HUPO-PSI/mzIdentML/blob/master/examples/1_3examples/crosslinking/</a></span><span class="c0">.</span></p><p class="c14"><span class="c0">The example files are:</span></p><ul class="c77 lst-kix_uy8ee0jk5g05-0 start"><li class="c6 c19 li-bullet-0"><span class="c0">Xlink_EDC_mzIdentML_1_3_0_draft.mzid (internally linked peptides),</span></li><li class="c6 c19 li-bullet-0"><span class="c0">multiple_spectra_per_id_1_3_0_draft.mzid (identification based on multiple spectra),</span></li><li class="c6 c19 li-bullet-0"><span class="c0">noncovalently_assoc_1_3_0_draft.mzid (noncovalently associated peptides),</span></li><li class="c6 c19 li-bullet-0"><span class="c0">scores_and_thresholds_1_3_0_draft.mzid (scores and thresholds).</span></li></ul><p class="c14 c15"><span class="c0"></span></p><hr style="page-break-before:always;display:none;"><h1 class="c41 c81" id="h.b3xsdb26q82p"><span class="c29 c43 c48"></span></h1><h1 class="c41" id="h.buyb3l8upkzl"><span class="c43">2. </span><span class="c43">Controlled </span><span class="c43">Vocabularies</span><span class="c29 c43 c48">&nbsp;for Encoding Crosslinks</span></h1><p class="c14"><span>A collection of terms for describing a certain domain is called a controlled vocabulary (CV) </span><span class="c37"><a class="c18" href="https://www.google.com/url?q=https://paperpile.com/c/YrD2gH/9dZZN&amp;sa=D&amp;source=editors&amp;ust=1719221372786773&amp;usg=AOvVaw3K4M-6zy1rM5K3URmtBp9r">[3]</a></span><span>.</span><span>&nbsp;Section 4.1 of the main mzIdentML 1.3.0 document describes the use of CVs in mzIdentML. T</span><span>he PSI-MS CV (</span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=https://github.com/HUPO-PSI/psi-ms-CV&amp;sa=D&amp;source=editors&amp;ust=1719221372787053&amp;usg=AOvVaw10qUnY1cOhy1xJ5Q_ldAnP">https://github.com/HUPO-PSI/psi-ms-CV</a></span><span>) can be used to encode many</span><span>&nbsp;types</span><span>&nbsp;of technical information in mzIdentML (e.g. statistical scores, mass spectrometers, etc)</span><span>. There are two other CVs that are relevant to encoding crosslinking data in mzIdentML: Unimod and XLMOD. </span><span>XLMOD </span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/master/cv/XLMOD.obo&amp;sa=D&amp;source=editors&amp;ust=1719221372787401&amp;usg=AOvVaw32kfEZW32uQ8eblGgLca1K">(https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/master/cv/XLMOD.obo)</a></span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=https://raw.githubusercontent.com/HUPO-PSI/mzIdentML/master/cv/XLMOD.obo&amp;sa=D&amp;source=editors&amp;ust=1719221372787588&amp;usg=AOvVaw1VNpBHIpbhF6PYa4QjIT50">&nbsp;</a></span><span>represents the crosslinker reagents. Unimod </span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=http://www.unimod.org/obo/unimod.obo&amp;sa=D&amp;source=editors&amp;ust=1719221372787791&amp;usg=AOvVaw06mY6urR1Rgdr71XgTrYg2">(http://www.unimod.org/obo/unimod.obo)</a></span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=http://www.unimod.org/obo/unimod.obo&amp;sa=D&amp;source=editors&amp;ust=1719221372787933&amp;usg=AOvVaw2XcnoMK08x0R4mxonTCMZN">&nbsp;</a></span><span>represents the resulting modifications in the crosslinked peptides/proteins.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>At the time of writing </span><span>(Unimod v2.1, XLMOD v1.1.12) both CVs have advantages and disadvantages when used for encoding</span><span>&nbsp;crosslinking results in mzIdentML</span><span>. For example, the </span><span>representation of heterobifunctional crosslinkers</span><span>&nbsp;(crosslinkers with different reactive groups) is better in XLMOD. However, the </span><span>representation of the derivatives from a cleavable crosslinker</span><span>&nbsp;is more complete in Unimod. Which CV </span><span>(XLMOD or Unimod)</span><span class="c0">&nbsp;to use for encoding crosslinker modifications is left as the implementers&rsquo; choice. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>There is also some overlap between the information stored in these CVs and the contents of the &lt;SearchModification&gt; elements in mzIdentML. The &lt;SearchModification&gt; elements can encode: the derivatives of cleavable crosslinkers, namely the crosslinker stub as a peptide modification on the MS3 level and crosslinker cleavability as stub fragments on the MS2 level; and crosslinker specificity (including heterobifunctional crosslinkers). </span><span>Implementers</span><span class="c0">&nbsp;SHOULD describe the crosslinker modifications searched for as &lt;SearchModification&gt; elements; this provides a consistent way of retrieving crosslinker modification information regardless of which CV has been used, see &sect;3.2.2.</span></p><hr style="page-break-before:always;display:none;"><p class="c14 c15"><span class="c0"></span></p><h1 class="c41" id="h.28v06xr0dl6y"><span class="c43">3. Encoding the Results of Crosslinking </span><span class="c43">Searches</span></h1><h2 class="c17" id="h.qt8ma1g7dinj"><span class="c47 c29">3.1. Introduction </span></h2><p class="c14"><span>mzIdentML documents MUST indicate that they are implementing the guidance in this extension document by including the following </span><span>CV term </span><span class="c0">inside the top-level &lt;MzIdentML&gt; element, immediately after the &lt;cvList&gt; element:</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003385&quot; name=&quot;mzIdentML crosslinking extension document version&quot;</span></p><p class="c14"><span class="c12">&nbsp; &nbsp; value=&quot;1.0.0&quot;/&gt;</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">Crosslinked peptides presented a challenge for mzIdentML 1.2.0, since more than one peptide can be identified from the same spectrum. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">mzIdentML 1.2.0 solved this by:</span></p><ul class="c77 lst-kix_7pm3iovtnuj2-0 start"><li class="c6 c19 li-bullet-0"><span>introducing the &ldquo;crosslink donor&rdquo; (MS:1002509) and &ldquo;crosslink acceptor&rdquo; (MS:1002510) CV terms &ndash; the values of these terms </span><span>associate</span><span>&nbsp;</span><span class="c50">either </span><span>&lt;SearchModification&gt; elements (see &sect;3.</span><span>2</span><span class="c0">.2) or &lt;Modification&gt; elements (see &sect;3.3);</span></li><li class="c6 c19 li-bullet-0"><span class="c0">introducing the &ldquo;crosslink spectrum identification item&rdquo; (MS:1002511) CV term &ndash; the values of these terms group &nbsp;&lt;SpectrumIdentificationItem&gt; elements within a &lt;SpectrumIdentificationResult&gt; (see &sect;3.4).</span></li></ul><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">Note that &ldquo;crosslink donor&rdquo; (MS:1002509) and &ldquo;crosslink acceptor&rdquo; (MS:1002510) are used in two different contexts:</span></p><ul class="c77 lst-kix_dqnkgejf3n11-0 start"><li class="c6 c19 li-bullet-0"><span class="c0">/MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/ ModificationParams/SearchModification &ndash; encoding the modifications searched for (including the specificity, see &sect;3.2.2); </span></li><li class="c6 c19 li-bullet-0"><span class="c0">/MzIdentML/SequenceCollection/Peptide/Modification - encoding the actual modifications present in the crosslinked peptides (&sect;3.3).</span></li></ul><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The rules that govern their use differ in each context, the details of these rules are given in </span><span>&sect;3.2.2 &amp; &sect;3.3. </span><span>To emphasise that they differ, Appendix I compares them.</span><span class="c0">&nbsp;Appendix I presents no new information on how to encode crosslinking results in mzIdentML.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">Figure 2 gives an overview of how the subsections here (&sect;3) relate to the elements in an mzIdentML file.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15 c116"><span class="c0"></span></p><p class="c67"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 864.00px; height: 562.67px;"><img alt="" src="images/image2.png" style="width: 864.00px; height: 562.67px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c14"><span class="c50">Figure 2. </span><span class="c50">Overview of the mzIdentML 1.3.0. Format (crosslinking extension).</span><span class="c50">&nbsp;</span><span>Elements are labelled with the section from this document that contains guidance on how to encode </span><span>them</span><span class="c0">. </span></p><h2 class="c17" id="h.w4d0zt5miiwi"><span class="c47 c29">3.2 &lt;SpectrumIdentificationProtocol&gt; Elements </span></h2><p class="c14"><span class="c0">A &lt;SpectrumIdentificationProtocol&gt; element describes the parameters and settings of a spectrum identification analysis. There may be several of these protocols included in one mzIdentML file. In the case of analysis workflows in which an identification is based on multiple spectra (see Section 7.11 of the main mzIdentML 1.3.0 specification document), these spectra identifications may be included in different &lt;SpectrumIdentificationList&gt; elements, each associated with a different &lt;SpectrumIdentificationProtocol&gt;.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>Section 2 of the main mzIdentML 1.3.0 specification document states that &ldquo;all search parameters should be described in sufficient detail to enable a user to run the same or a similar search on the same or another search engine&rdquo;. As far as possible, the information that would be needed to reannotate the mass spectra SHOULD be included. The &lt;FragmentTolerance&gt; and &lt;ParentTolerance&gt; subelements of &lt;SpectrumIdentificationProtocol&gt; SHOULD be completed</span><span>. </span><span class="c0">&nbsp;</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">Two child elements of &lt;SpectrumIdentificationProtocol&gt; are covered in more detail here:</span></p><ul class="c77 lst-kix_qvbjpn84g05v-0 start"><li class="c6 c19 li-bullet-0"><span class="c0">&lt;AdditionalSearchParams&gt; (&sect;3.2.1),</span></li><li class="c6 c19 li-bullet-0"><span class="c0">&nbsp;&lt;ModificationParams&gt; (&sect;3.2.2). </span></li></ul><p class="c14 c15"><span class="c0"></span></p><h3 class="c60" id="h.oap1xkt9k8zx"><span class="c29 c86 c50 c52">3.2.1 Additional Search Parameters</span></h3><p class="c14"><span class="c102 c50">Path:</span><span class="c102">&nbsp;</span><span class="c56 c48">/MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/AdditionalSearchParams</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c50">&nbsp;</span><span>If a crosslinking search has been performed then the CV term &ldquo;crosslinking search&rdquo; (MS:1002494) </span><span>MUST</span><span class="c0">&nbsp;be present within the &lt;AdditionalSearchParams&gt; subelement of every &lt;SpectrumIdentificationProtocol&gt; associated with that search (see Figure 3). </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The ion series that were searched for SHOULD also be included here.</span></p><p class="c14 c15"><span class="c10"></span></p><p class="c14"><span class="c63 c36">New supported use case in this extension - noncovalently associated peptides:</span><span class="c50">&nbsp;</span><span>mzIdentML 1.2.1 introduces a</span><span>&nbsp;new CV term &ndash; &ldquo;noncovalently associated </span><span>peptides search&rdquo;</span><span>&nbsp;(MS:1003330).</span><span>&nbsp;If pairs of </span><span>noncovalently associated peptides</span><span>&nbsp;were also searched for, then the &lt;SpectrumIdentificationProtocol&gt; elements</span><span>&nbsp;MUST </span><span>also contain this new CV term within their &lt;AdditionalSearchParams&gt; subelement, see Figure 3. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The new CV term </span><span>&quot;FDR applied separately to self crosslinks and protein heteromeric crosslinks&quot; (MS:1003343), see &sect;4.4, </span><span>which </span><span>SHOULD</span><span>&nbsp;be present is also shown in Figure 3. </span></p><p class="c14 c15"><span class="c9"></span></p><hr style="page-break-before:always;display:none;"><p class="c14 c15"><span class="c9"></span></p><p class="c14"><span class="c9">&lt;AnalysisProtocolCollection&gt;</span></p><p class="c14 c15"><span class="c9"></span></p><p class="c6 c55"><span class="c9">&lt;SpectrumIdentificationProtocol analysisSoftware_ref=&quot;ID_software&quot; id=&quot;SearchProtocol_1&quot;&gt;</span></p><p class="c6 c55 c15"><span class="c9"></span></p><p class="c14 c33 c55"><span class="c9">&lt;SearchType&gt;</span></p><p class="c14 c33 c55 c35"><span class="c9">&lt;cvParam accession=&quot;MS:1001083&quot; cvRef=&quot;PSI-MS&quot; name=&quot;ms-ms search&quot;/&gt;</span></p><p class="c14 c33 c55"><span class="c9">&lt;/SearchType&gt;</span></p><p class="c14 c33 c55 c15"><span class="c9"></span></p><p class="c14 c33 c55"><span class="c12">&lt;AdditionalSearchParams&gt;</span></p><p class="c14 c38 c55"><span class="c9">&lt;cvParam accession=&quot;MS:1001211&quot; cvRef=&quot;PSI-MS&quot; name=&quot;parent mass type mono&quot;/&gt;</span></p><p class="c14 c38 c55"><span class="c9">&lt;cvParam accession=&quot;MS:1001256&quot; cvRef=&quot;PSI-MS&quot; name=&quot;fragment mass type mono&quot;/&gt;</span></p><p class="c14 c38 c55"><span class="c9 c26">&lt;cvParam accession=&quot;MS:1002494&quot; cvRef=&quot;PSI-MS&quot; name=&quot;crosslinking search&quot;/&gt;</span></p><p class="c14 c38 c55"><span class="c9 c20">&lt;cvParam accession=&quot;MS:1003330&quot; cvRef=&quot;PSI-MS&quot; </span></p><p class="c14 c55 c35 c97"><span class="c12 c20">name=&quot;noncovalently associated </span><span class="c12 c20">peptides search</span><span class="c9 c20">&quot;/&gt;</span></p><p class="c14 c55"><span class="c12">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c9 c42">&lt;cvParam accession=&quot;MS:1003343&quot; cvRef=&quot;PSI-MS&quot; </span></p><p class="c14 c55 c35 c97"><span class="c9 c42">name=&quot;FDR applied separately to self crosslinks and protein heteromeric crosslinks&quot; value=&quot;true&quot;/&gt;</span></p><p class="c14 c38 c55"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1001118&quot; name=&quot;param: b ion&quot;/&gt;</span></p><p class="c14 c33 c55 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1001262&quot; name=&quot;param: y ion&quot;/&gt;</span></p><p class="c14 c38 c55 c15"><span class="c9"></span></p><p class="c6 c55 c35"><span class="c9">&lt;/AdditionalSearchParams&gt;</span></p><p class="c6 c8 c55"><span class="c9"></span></p><p class="c6 c55 c35"><span class="c9">...</span></p><p class="c6 c8 c55"><span class="c9"></span></p><p class="c6 c55"><span class="c9">&lt;/SpectrumIdentificationProtocol&gt;</span></p><p class="c6 c55 c15"><span class="c9"></span></p><p class="c14 c55"><span class="c9">&lt;/AnalysisProtocolCollection&gt;</span></p><p class="c14 c55 c15"><span class="c29 c48 c78"></span></p><p class="c14"><span class="c50">Figure 3. XML snippet showing crosslinking related CV </span><span class="c50">terms</span><span class="c50">. </span><span>If a crosslinking search has been performed, </span><span class="c26">MS:1002494</span><span>&nbsp;</span><span class="c50">MUST</span><span>&nbsp;be present.</span><span class="c50">&nbsp;</span><span>If noncovalently associated peptides have also been searched for, then </span><span class="c20">MS:1003330</span><span>&nbsp;</span><span class="c50">MUST</span><span>&nbsp;also be present. </span><span>The new CV term </span><span class="c42">&quot;FDR applied separately to self crosslinks and protein heteromeric crosslinks&quot; (MS:1003343)</span><span>&nbsp;which </span><span class="c50">SHOULD</span><span>&nbsp;be present </span><span>is also shown, see &sect;4.4. </span></p><p class="c14 c15"><span class="c0"></span></p><h3 class="c60" id="h.1wnij58lg2z0"><span class="c29 c86 c50 c52">3.2.2 Modification Parameters</span></h3><p class="c14"><span class="c102 c50">Path:</span><span class="c102">&nbsp;</span><span class="c56 c48">/MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/ModificationParams/SearchModification</span></p><p class="c14 c15"><span class="c29 c48 c54 c24"></span></p><p class="c14"><span>The &lt;SpectrumIdentificationProtocol&gt; element encodes the modifications that were searched &nbsp;for </span><span>within its &lt;ModificationParams&gt; subelement. These are encoded in &lt;SearchModification&gt; elements within </span><span>&lt;ModificationParams&gt;</span><span class="c0">.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>mzIdentML version 1.3.0 </span><span>introduces two new CV terms to link</span><span class="c0">&nbsp;&lt;SearchModification&gt; elements and &lt;Modification&gt; elements - &ldquo;search modification id&rdquo; (MS:1003392) which goes inside &lt;SearchModification&gt; elements, and &ldquo;search modification id ref&rdquo; (MS:1003393) which goes inside &lt;Modification&gt; elements. Making this link is optional but recommended where possible. In the case of open modification searches, such a link cannot be made. See Section 7.12 of the main mzIdentML specification document.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>Each crosslinker reagent is defined by multiple &lt;SearchModification&gt; elements that contain either the &ldquo;crosslink donor&rdquo; (MS:1002509) or &ldquo;crosslink acceptor&rdquo; (MS:1002510) CV term. An example is given in </span><span>Figure 4(</span><span>i</span><span class="c0">). The residue specificities of the crosslinkers used SHOULD be encoded here, examples are given in Appendix II. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The value slot of the crosslink donor and acceptor CV terms is interpreted as a local identifier for the &lt;SearchModification&gt; elements describing a single reagent. The rules governing the use of the crosslink donor and acceptor CV terms in &lt;SearchModification&gt; elements are given in Figure 4(</span><span>ii</span><span class="c0">). </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">There may be more than two &lt;SearchModification&gt; elements required. For example, if the crosslinker reacts with the sidechains and also with the protein termini, see Appendix II for examples.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">&lt;SearchModification&gt; elements can contain one or more children of the CV term &ldquo;peptide modification details&rdquo; (MS:1001471). These CV terms can encode information on neutral losses, see Figure 4(i).</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c63 c36">New supported use case in this extension - cleavable crosslinkers:</span><span class="c50">&nbsp;</span><span>m</span><span>zIdentML 1.3.0 adds three new CV terms relating to modifications from cleavable crosslinkers &ndash;</span><span class="c50 c75">&nbsp;</span><span class="c75">&ldquo;cleavable crosslinker stub&rdquo; (MS:1003346), &ldquo;</span><span class="c75">Unimod derivative code</span><span class="c75">&rdquo; (MS:1003347) and </span><span class="c75">&ldquo;crosslinker cleavage characteristics&rdquo; (MS:1003390)</span><span class="c75">.</span></p><p class="c14 c15"><span class="c0 c75"></span></p><p class="c14"><span>At the MS3 level, where single peptides and part of the cleaved crosslinker are identified, the crosslinker modifications SHOULD include the CV term &ldquo;</span><span class="c75">cleavable </span><span class="c0">crosslinker stub&rdquo; (MS:1003346). </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The crosslink stub modification MUST also have a suitably sourced CV term for the reagent (see Appendix II).</span><span>&nbsp;</span><span>Additionally, if Unimod is being used as the CV, then the CV term </span><span class="c75">&ldquo;Unimod derivative code&rdquo; (MS:1003347) MAY be used to state which derivative of the cleaved crosslinker is identified. The single-letter derivative codes in Unimod are chosen arbitrarily when a linker definition is added to Unimod. For instance, in </span><span class="c63 c75 c94"><a class="c18" href="https://www.google.com/url?q=https://unimod.org/xlink.html&amp;sa=D&amp;source=editors&amp;ust=1719221372799451&amp;usg=AOvVaw2OUSqt2jWr2Gl6AkH0DZW9">https://unimod.org/xlink.html</a></span><span class="c75">&nbsp;one can find the examples &quot;A for alkene, S for sulfenic acid, and T for thiol&quot;, and e.g. Xlink:DSS uses W for loss of water. There is no formal vocabulary for the single-letter codes. &nbsp;&quot;UNIMOD derivative code&quot; must be equal to one of the derivative codes defined in the corresponding Unimod entry (not a random character unrelated to the definition). An example Unimod entry is at </span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=https://www.unimod.org/modifications_view.php?editid1%3D1842&amp;sa=D&amp;source=editors&amp;ust=1719221372799824&amp;usg=AOvVaw00pxAATW3vpHYDA9RFuLpP">https://www.unimod.org/modifications_view.php?editid1=1842 .</a></span><span>&nbsp; </span><span class="c0 c75">&nbsp;</span></p><p class="c14"><span>&nbsp;</span></p><p class="c14"><span class="c0">At the MS2 level, the new CV term &ldquo;crosslinker cleavage characteristics&rdquo; (MS:1003390) signifies that the crosslinker is cleavable and on cleavage can leave a given stub. This can lead to additional stub fragments in the MS2 spectra that contain the crosslinker stub instead of the whole crosslinker plus the second peptide. Each &ldquo;crosslinker cleavage characteristics&rdquo; CV term represents one possible crosslinker stub. It has a structured value -</span></p><p class="c14 c35"><span class="c36">name</span><span>:</span><span class="c36">mass</span><span>:</span><span class="c37 c62 c36 c52">pairs with</span></p><p class="c14"><span class="c36">Name</span><span>&nbsp;must be a single character to identify this stub. </span><span>The scope of </span><span class="c36">name </span><span>is restricted to that crosslinker definition, i.e. they need only be unique within that crosslinker definition not the whole file or the &lt;SpectrumIdentification&gt; element.</span><span>&nbsp;</span><span class="c36">Mass</span><span>&nbsp;gives the monoisotopic mass delta of the resulting stub in Daltons. </span><span class="c36">Pairs with </span><span>MUST be a sequence of one or more</span><span>&nbsp;characters, giving the </span><span class="c36">name(s)</span><span class="c0">&nbsp;of the partner stub(s). See Appendix II for examples.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">Note that the choice of which &lt;SearchModification&gt; is the donor and which one is the acceptor is arbitrary. </span></p><p class="c14 c15"><span class="c9"></span></p><p class="c14"><span class="c10">(i) </span></p><p class="c14"><span class="c9">&lt;SpectrumIdentificationProtocol&gt;</span></p><p class="c14 c35"><span class="c9">...</span></p><p class="c14 c35"><span class="c9">&lt;ModificationParams&gt;</span></p><p class="c14 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; </span><span class="c12 c26">massDelta=&quot;138.06808&quot;</span><span class="c9">&nbsp;residues=&quot;S T Y K&quot;&gt;</span></p><p class="c6 c35"><span class="c12">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;</span><span class="c12">MS:1003392</span><span class="c9">&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;BS3_donor&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9 c20">&lt;cvParam cvRef=&quot;XLMOD&quot; accession=&quot;XLMOD:02000&quot; name=&quot;BS3&quot;/&gt;</span></p><p class="c6"><span class="c12">&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c12 c42">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002509&quot; name=&quot;crosslink donor&quot;</span><span class="c9 c24">&nbsp;</span></p><p class="c14 c38 c35"><span class="c9 c24">value=&quot;0&quot;/&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; </span><span class="c12 c26">massDelta=&quot;138.06808&quot;</span><span class="c12">&nbsp;</span><span class="c12">residues=&quot;.&quot;</span><span class="c9">&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c97"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002057&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein N-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c12">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;</span><span class="c12">MS:1003392</span><span class="c9">&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;BS3_donor_n_term&quot;/&gt;</span></p><p class="c14 c38"><span class="c12 c20">&lt;cvParam cvRef=&quot;XLMOD&quot; accession=&quot;XLMOD:02000&quot; name=&quot;BS3&quot;/&gt;</span></p><p class="c14 c38"><span class="c9 c42">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink donor&quot;</span></p><p class="c14 c38 c35"><span class="c12 c24">value=&quot;0&quot; /&gt;</span></p><p class="c14 c33"><span class="c12">&lt;/SearchModification&gt;</span></p><p class="c6 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; </span><span class="c12 c22">massDelta=&quot;0.0&quot;</span><span class="c9">&nbsp;residues=&quot;S T Y K&quot;&gt;</span></p><p class="c6 c35"><span class="c9">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;BS3_acceptor&quot;/&gt;</span></p><p class="c14 c38"><span class="c12">&nbsp; </span><span class="c9 c20">&lt;cvParam cvRef=&quot;XLMOD&quot; accession=&quot;XLMOD:02000&quot; name=&quot;BS3&quot;/&gt;</span></p><p class="c14 c38"><span class="c12">&nbsp; </span><span class="c12 c42">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink acceptor&quot;</span><span class="c9 c24">&nbsp;</span></p><p class="c14 c38 c35"><span class="c9 c24">value=&quot;0&quot;/&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c8"><span class="c9"></span></p><p class="c14 c33"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; </span><span class="c12 c22">massDelta=&quot;0.0&quot;</span><span class="c12">&nbsp;</span><span class="c12">residues=&quot;.&quot;</span><span class="c9">&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c38 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002058&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein N-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c12">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;</span><span class="c12">MS:1003392</span><span class="c9">&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;BS3_acceptor_n_term&quot;&gt;</span></p><p class="c14 c38"><span class="c12 c20">&lt;cvParam cvRef=&quot;XLMOD&quot; accession=&quot;XLMOD:02000&quot; name=&quot;BS3&quot;/&gt;</span></p><p class="c14 c38"><span class="c9 c42">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink acceptor&quot;</span></p><p class="c14 c38 c35"><span class="c12 c24">value=&quot;0&quot; /&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c33"><span class="c9">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;15.994919&quot; residues=&quot;M&quot;&gt;</span></p><p class="c14 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c14 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;Mox&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:35&quot; name=&quot;Oxidation&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &lt;cvParam accession=&quot;MS:1001524&quot; name=&quot;fragment neutral loss&quot;</span></p><p class="c14 c97"><span class="c9">cvRef=&quot;PSI-MS&quot; value=&quot;63.998291&quot; unitAccession=&quot;UO:0000221&quot; unitName=&quot;dalton&quot; unitCvRef=&quot;UO&quot;/&gt;</span></p><p class="c14 c33"><span class="c12">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c6 c8"><span class="c9"></span></p><p class="c14 c35"><span class="c9">&lt;/ModificationParams&gt;</span></p><p class="c14 c35"><span class="c9">...</span></p><p class="c14"><span class="c12">&lt;/SpectrumIdentificationProtocol&gt;</span></p><hr style="page-break-before:always;display:none;"><p class="c14 c15"><span class="c10"></span></p><p class="c14"><span class="c50">(ii) </span></p><ul class="c77 lst-kix_uscd8mjnf5fw-0 start"><li class="c6 c19 li-bullet-0"><span class="c50 c42">At least two</span><span class="c42">&nbsp;&lt;SearchModification&gt; elements SHOULD be used to encode each crosslink reagent, to encode the site specificity of both the donor and acceptor termini of the reagent.</span><span class="c0">&nbsp;</span></li><li class="c6 c19 li-bullet-0"><span class="c0 c24">The value slot of the crosslink donor and acceptor CV terms is interpreted as a local identifier for the &lt;SearchModification&gt; elements describing a single reagent.</span></li><li class="c6 c19 li-bullet-0"><span class="c0">The choice of which reactive group is the donor and which is the acceptor is arbitrary.</span></li><li class="c6 c19 li-bullet-0"><span class="c26">The crosslink donor </span><span class="c26">&lt;SearchModification&gt; element</span><span class="c26">&nbsp;</span><span class="c26 c50">MUST</span><span class="c26">&nbsp;have the attribute massDelta = the mass gain from the crosslink reagent</span><span class="c0">. </span></li><li class="c6 c19 li-bullet-0"><span class="c22">The crosslink acceptor peptide&rsquo;s &lt;SearchModification&gt; element </span><span class="c22 c50">MUST</span><span class="c22">&nbsp;have massDelta = 0</span><span>.</span><span class="c0">&nbsp;</span></li><li class="c6 c19 li-bullet-0"><span class="c50 c20">Both</span><span class="c20">&nbsp;acceptor and donor </span><span class="c50 c20">MUST</span><span class="c20">&nbsp;have a suitably sourced &lt;cvParam&gt;.</span></li></ul><p class="c14 c15"><span class="c29 c53 c82"></span></p><p class="c14 c15"><span class="c29 c53 c82"></span></p><p class="c14"><span class="c50">Figure 4.</span><span>&nbsp;</span><span class="c50">The use of the &ldquo;crosslink donor&rdquo; (MS:1002509) and &ldquo;crosslink acceptor&rdquo; (MS:1002510) CV terms in &lt;SearchModification&gt; elements.</span><span class="c0">&nbsp;</span></p><p class="c14"><span class="c50">(i)</span><span>&nbsp;XML snippet showing the &ldquo;crosslink donor&rdquo; (MS:1002509) and &ldquo;crosslink acceptor&rdquo; (MS:1002510) CV terms used in &lt;SearchModification&gt;, s</span><span>hows encoding for the BS3 crosslinking reagent. It also shows a modification with a neutral loss.</span></p><p class="c14"><span class="c50">(ii) </span><span class="c0">The rules applying to the use of the &ldquo;crosslink donor&rdquo; (MS:1002509) and &ldquo;crosslink acceptor&rdquo; (MS:1002510) CV terms within &lt;SearchModification&gt;.</span></p><hr style="page-break-before:always;display:none;"><p class="c14 c15"><span class="c9"></span></p><h2 class="c17" id="h.wi4u0wz85u5v"><span class="c88 c50">3.3 Encoding Crosslinked Peptides</span><span class="c47 c29">&nbsp;in the Element &lt;SequenceCollection&gt;</span></h2><p class="c14"><span class="c63 c102 c50">Path:</span><span class="c48 c56">&nbsp;/MzIdentML/SequenceCollection</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">The peptides that have been identified are encoded in the &lt;SequenceCollection&gt; element. This will include both crosslinked and uncrosslinked peptides.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>A word of warning about redundancy, it is not the intention of mzIdentML that every &lt;SpectrumIdentificationItem&gt; (&sect;3.4) references a new &lt;Peptide&gt; in &lt;SequenceCollection&gt; &ndash; &ldquo;the combination of &lt;</span><span>Peptide&gt; </span><span class="c0">sequence and modifications MUST be unique in the file&rdquo; (main mzIdentML specification document, Section 6.68). However, each distinct combination of crosslinked peptides will require a new pair of &lt;Peptide&gt; elements in &lt;SequenceCollection&gt;. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">To represent the crosslinked peptides, mzIdentML 1.2.0 added a mechanism for linking two different &lt;Peptide&gt; elements together, using the CV terms &ldquo;crosslink donor&rdquo; (MS:1002509) and &ldquo;crosslink acceptor&rdquo; (MS:1002510). An identical value for these terms indicates that they are grouped together, see Figure 5(i).</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">The rules governing the use of the crosslink donor and acceptor CV terms in &lt;Modification&gt; elements are given in Figure 5(ii). </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">As of mzIdentML 1.3.0, &lt;Modification&gt; elements MAY contain the CV term &quot;search modification id ref&quot; (MS:1003393) to link a &lt;Modification&gt; to a &lt;SearchModification&gt; element. The value of this term is the unique id of the &lt;SearchModification&gt; as defined by its &quot;search modification id&quot; (MS:1003392) CV term. It is recommended to use this approach for the encoding of modifications from crosslinkers, see Appendix II.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c63 c36">New supported use case in this extension - internally linked peptide:</span><span>&nbsp;An internally linked peptide has both ends of the crosslinker within it. &nbsp;To encode an internally crosslinked peptide the &lt;Peptide&gt; can contain one &lt;Modification&gt; element with the &ldquo;crosslink donor&rdquo; CV term and one &lt;Modification&gt; element with the &ldquo;crosslink acceptor&rdquo; CV term. The same rules apply to these CV terms when encoding internally linked peptides as when encoding crosslinked peptides (Figure 5 (ii)). For an example of how to encode an internally linked peptide, see Figure 5(iii). </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The accompanying example file </span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=https://github.com/HUPO-PSI/mzIdentML/blob/master/examples/1_3_0examples/crosslinking/multiple_spectra_per_id_1.3.0_draft.mzid&amp;sa=D&amp;source=editors&amp;ust=1719221372808488&amp;usg=AOvVaw2IZYfCF6Rs7IV6WiPDlr23">multiple_spectra_per_id_1.3.0_draft.mzid</a></span><span>&nbsp;illustrates a common cleavable crosslinker </span><span>workflow </span><span class="c37"><a class="c18" href="https://www.google.com/url?q=https://paperpile.com/c/YrD2gH/osIjk&amp;sa=D&amp;source=editors&amp;ust=1719221372808863&amp;usg=AOvVaw0-kGSqgYH41BKot7ho2Ama">[4]</a></span><span>.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">Child CV terms of &ldquo;peptide modification details&rdquo; (MS:1001471) can be included in &lt;Modification&gt; elements to provide additional information about the modification, including the new cleavable crosslinker related CV terms, see &sect;3.2.2. This is not recommended if the &lt;Modification&gt; elements have &quot;search modification id ref&quot; (MS:1003393) CV terms to link them to a &lt;SearchModification&gt; element, as it would add unnecessary duplication to the file.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The encoding for crosslinked peptides MAY be combined with the encoding for modification localisation scoring, using </span><span>the same mechanism</span><span>&nbsp;(main mzIdentML 1.3.0 document, Section 5.2.8). </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c10">(i)</span></p><p class="c13"><span class="c9 c75">&lt;SequenceCollection&gt;</span></p><p class="c13 c15"><span class="c9 c75"></span></p><p class="c6"><span class="c9">&lt;Peptide id=&quot;30491856_30492180_2_4_p1&quot;&gt;</span></p><p class="c14 c33"><span class="c9">&lt;PeptideSequence&gt;AAFTKQAADK&lt;/PeptideSequence&gt;</span></p><p class="c14 c33"><span class="c12">&lt;Modification </span><span class="c12 c26">monoisotopicMassDelta=&quot;138.0680796&quot;</span><span class="c9">&nbsp;location=&quot;5&quot;&gt;</span></p><p class="c14 c33"><span class="c9">&nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003393&quot; </span></p><p class="c14 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; name=&quot;search modification id ref&quot; value=&quot;DSS_donor&quot;/&gt;</span></p><p class="c14 c38"><span class="c9 c20">&lt;cvParam accession=&quot;XL:00002&quot; cvRef=&quot;PSI-MS&quot; name=&quot;Xlink:DSS&quot;/&gt;</span></p><p class="c14 c38"><span class="c12 c42">&lt;cvParam accession=&quot;MS:1002509&quot; cvRef=&quot;PSI-MS&quot; name=&quot;crosslink donor&quot; </span><span class="c12 c24">value=&quot;</span><span class="c71 c24 c50">5448</span><span class="c9 c24">&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/Modification&gt;</span></p><p class="c6"><span class="c9">&lt;/Peptide&gt;</span></p><p class="c6 c15"><span class="c9"></span></p><p class="c6"><span class="c9">&lt;Peptide id=&quot;30491856_30492180_2_4_p2&quot;&gt;</span></p><p class="c14 c33"><span class="c9">&lt;PeptideSequence&gt;AMYPPKEDR&lt;/PeptideSequence&gt;</span></p><p class="c14 c33"><span class="c12">&lt;Modification </span><span class="c12 c22">monoisotopicMassDelta=&quot;0.0&quot;</span><span class="c9">&nbsp;location=&quot;6&quot;&gt;</span></p><p class="c14 c33"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003393&quot; </span></p><p class="c14 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; name=&quot;search modification id ref&quot; value=&quot;DSS_acceptor&quot;/&gt;</span></p><p class="c14 c38"><span class="c12 c42">&lt;cvParam accession=&quot;MS:1002510&quot; cvRef=&quot;PSI-MS&quot; name=&quot;crosslink acceptor&quot; </span><span class="c12 c24">value=&quot;</span><span class="c71 c24 c50">5448</span><span class="c9 c24">&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/Modification&gt;</span></p><p class="c6"><span class="c9">&lt;/Peptide&gt;</span></p><p class="c6 c15"><span class="c9"></span></p><p class="c13 c49"><span class="c9 c75">&nbsp;...</span></p><p class="c13 c49 c15"><span class="c9 c75"></span></p><p class="c13"><span class="c12 c75">&lt;/SequenceCollection&gt;</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c50">(ii) </span><span>If a pair of crosslinked peptides has been identified:</span></p><ul class="c77 lst-kix_bg52hg4v2ft5-0 start"><li class="c6 c19 li-bullet-0"><span class="c42">One peptide&rsquo;s &lt;Modification&gt; element MUST be flagged as &ldquo;crosslink donor&rdquo; and one MUST be flagged as &ldquo;crosslink acceptor&rdquo;</span><span class="c0">. </span></li><li class="c5 c19 c49 li-bullet-0"><span class="c24">A unique identifier linking exactly </span><span class="c24 c50">two</span><span class="c24">&nbsp;&lt;Modification&gt; elements together </span><span class="c24 c50">MUST</span><span class="c24">&nbsp;be in the value slot.</span><span class="c0">&nbsp;(Thereby excluding the representation of trimeric crosslinkers, see &sect;6.)</span></li><li class="c6 c19 li-bullet-0"><span class="c0">If the CV term &ldquo;search modification id ref&rdquo; (MS:1003393) is being used then the crosslink donor MUST be chosen to match the end marked as the donor in the corresponding &lt;SearchModification&gt; elements, see &sect;3.2.2. If that CV term is not used, or if the preceding rule does not unambiguously define which end to mark as donor (e.g. because the crosslinker is symmetrical) then the export software SHOULD use the following rules to choose the crosslink donor as the: longer peptide, then higher peptide neutral mass, then alphabetical order. </span></li><li class="c6 c19 li-bullet-0"><span class="c26">The crosslink donor &lt;Modification&gt; element </span><span class="c26 c50">MUST</span><span class="c26">&nbsp;have the attribute monoisotopicMassDelta = the mass gain from the crosslink reagent</span><span class="c0">. </span></li><li class="c6 c19 li-bullet-0"><span class="c22">The crosslink acceptor peptide&rsquo;s &lt;Modification&gt; element </span><span class="c22 c50">MUST</span><span class="c22">&nbsp;have monoisotopicMassDelta = 0</span><span class="c0">. </span></li><li class="c6 c19 li-bullet-0"><span class="c20">The crosslink donor peptide&rsquo;s &lt;Modification&gt; element </span><span class="c50 c20">MUST</span><span class="c20">&nbsp;have a suitably sourced cvParam for the crosslink. </span><span class="c20">The crosslink acceptor</span><span class="c20">&nbsp;peptide&rsquo;s &lt;Modification&gt; element </span><span class="c50 c20">MUST</span><span class="c20">&nbsp;</span><span class="c50 c20">NOT </span><span class="c0 c20">have a cvParam for the reagent.</span></li></ul><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c50">(iii)</span></p><p class="c13"><span class="c9 c75">&lt;SequenceCollection&gt;</span></p><p class="c13 c15"><span class="c9 c75"></span></p><p class="c6"><span class="c9">&lt;Peptide id=&quot;peptide_7_1&quot;&gt;</span></p><p class="c14 c33"><span class="c9">&lt;PeptideSequence&gt;DVIQSLVDDDLVAK&lt;/PeptideSequence&gt;</span></p><p class="c14 c33"><span class="c12">&lt;Modification location=&quot;10&quot; residues=&quot;D&quot; </span><span class="c12 c26">monoisotopicMassDelta=&quot;-18.010565&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33"><span class="c9">&nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003393&quot; </span></p><p class="c14 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; name=&quot;search modification id ref&quot; value=&quot;EDC_donor&quot;/&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c38"><span class="c9 c20">&lt;cvParam accession=&quot;UNIMOD:2018&quot; name=&quot;Xlink:EDC&quot; cvRef=&quot;UNIMOD&quot;/&gt;</span></p><p class="c14 c38"><span class="c9 c42">&lt;cvParam accession=&quot;MS:1002509&quot; cvRef=&quot;PSI-MS&quot; name=&quot;crosslink donor&quot; </span></p><p class="c14 c38 c35"><span class="c12 c24">value=&quot;</span><span class="c24 c50 c71">100</span><span class="c9 c24">&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/Modification&gt;</span></p><p class="c14 c33"><span class="c12">&lt;Modification location=&quot;14&quot; residues=&quot;K&quot; </span><span class="c12 c22">monoisotopicMassDelta=&quot;0.0&quot;</span><span class="c12">&gt;</span></p><p class="c14 c38"><span class="c9">&nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003393&quot; </span></p><p class="c14 c38"><span class="c9">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;name=&quot;search modification id ref&quot; value=&quot;EDC_acceptor&quot;/&gt;</span></p><p class="c14 c38"><span class="c9 c42">&lt;cvParam accession=&quot;MS:1002510&quot; cvRef=&quot;PSI-MS&quot; name=&quot;crosslink acceptor&quot;</span></p><p class="c14 c38 c35"><span class="c12 c24">value=&quot;</span><span class="c71 c24 c50">100</span><span class="c9 c24">&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/Modification&gt;</span></p><p class="c6"><span class="c9">&lt;/Peptide&gt;</span></p><p class="c6 c15"><span class="c9"></span></p><p class="c6"><span class="c9">...</span></p><p class="c6 c15"><span class="c9"></span></p><p class="c13"><span class="c9 c75">&lt;/SequenceCollection&gt;</span></p><p class="c13 c15"><span class="c9 c75"></span></p><p class="c14 c15"><span class="c9"></span></p><p class="c14"><span class="c10">Figure 5. Encoding Crosslinked Peptides in the Element &lt;SequenceCollection&gt;</span></p><p class="c14"><span class="c0">(i) XML snippet showing the encoding of crosslinked peptides.</span></p><p class="c14"><span class="c0">(ii) The rules applying to the use of the &ldquo;crosslink donor&rdquo; (MS:1002509) and &ldquo;crosslink acceptor&rdquo; (MS:1002510) CV terms within &lt;Modification&gt; elements.</span></p><p class="c14"><span>(iii)</span><span class="c50">&nbsp;</span><span class="c0">XML snippet showing the encoding of an internally linked peptide.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c9"></span></p><hr style="page-break-before:always;display:none;"><p class="c14 c15"><span class="c29 c48 c24 c54"></span></p><p class="c14 c15"><span class="c2"></span></p><p class="c13"><span class="c2">&lt;SequenceCollection&gt;</span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c28">&lt;!-- linear peptides--&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;Peptide id=&quot;p1_linear&quot;&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;PeptideSequence&gt;PEPKR&lt;/PeptideSequence&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;Modification location=&quot;4&quot; monoisotopicMassDelta=&quot;176.01433&quot;&gt;</span></p><p class="c13 c49"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003393&quot; </span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; name=&quot;search modification id ref&quot; value=</span><span class="c73">&quot;DSSO_monolink_W&quot;/</span><span class="c2">&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;cvParam accession=&quot;UNIMOD:1842&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:DSSO&quot;/&gt;</span></p><p class="c13"><span class="c28">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c73 c26">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; value=&quot;W&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;/Modification&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;/Peptide&gt;</span></p><p class="c13 c15"><span class="c2"></span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;!-- crosslinked peptides --&gt;</span></p><p class="c13 c35"><span class="c2">&lt;Peptide id=&quot;p1&quot;&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;PeptideSequence&gt;PEPKR&lt;/PeptideSequence&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;Modification location=&quot;4&quot; monoisotopicMassDelta=&quot;158.003765&quot;&gt;</span></p><p class="c13 c49"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003393&quot; </span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; name=&quot;search modification id ref&quot; value=&quot;DSSO_donor&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;cvParam accession=&quot;UNIMOD:1842&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:DSSO&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002509&quot; name=&quot;crosslink donor&quot; value=&quot;1&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;/Modification&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;/Peptide&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;Peptide id=&quot;p2&quot;&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;PeptideSequence&gt;TIDYK&lt;/PeptideSequence&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;Modification location=&quot;4&quot; monoisotopicMassDelta=&quot;0&quot;&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003393&quot; </span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; name=&quot;search modification id ref&quot; value=&quot;DSSO_acceptor&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink acceptor&quot; value=&quot;1&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;/Modification&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;/Peptide&gt;</span></p><p class="c13 c15"><span class="c2"></span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;!-- MS3 peptides are separately listed, as they are linear stub modified peptides --&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;Peptide id=&quot;p1_a&quot;&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;PeptideSequence&gt;PEPKR&lt;/PeptideSequence&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;Modification location=&quot;4&quot; monoisotopicMassDelta=&quot;54.010565&quot;&gt;</span></p><p class="c13 c49"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003393&quot; </span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; name=&quot;search modification id ref&quot; value=&quot;</span><span class="c73">DSSO_crosslink_stub_a</span><span class="c2">&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;cvParam accession=&quot;UNIMOD:1842&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:DSSO&quot;/&gt;</span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c57 c29 c26">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; value=&quot;A&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c57 c29 c20">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003346&quot; name=&quot;cleavable crosslinker stub&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;/Modification&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;/Peptide&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;Peptide id=&quot;p1_t&quot;&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;PeptideSequence&gt;PEPKR&lt;/PeptideSequence&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;Modification location=&quot;4&quot; monoisotopicMassDelta=&quot;85.982635&quot;&gt;</span></p><p class="c13 c49"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003393&quot; </span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; name=&quot;search modification id ref&quot; value=&quot;</span><span class="c73">DSSO_crosslink_stub_t</span><span class="c2">&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;cvParam accession=&quot;UNIMOD:1842&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:DSSO&quot;/&gt;</span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c57 c29 c26">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; value=&quot;T&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c57 c29 c20">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003346&quot; name=&quot;cleavable crosslinker stub&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;/Modification&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;/Peptide&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;Peptide id=&quot;p2_a&quot;&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;PeptideSequence&gt;TIDYK&lt;/PeptideSequence&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;Modification location=&quot;4&quot; monoisotopicMassDelta=&quot;54.010565&quot;&gt;</span></p><p class="c13 c49"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003393&quot; </span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; name=&quot;search modification id ref&quot; value=&quot;</span><span class="c73">DSSO_crosslink_stub_a</span><span class="c2">&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;cvParam accession=&quot;UNIMOD:1842&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:DSSO&quot;/&gt;</span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c57 c29 c26">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; value=&quot;A&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c57 c29 c20">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003346&quot; name=&quot;cleavable crosslinker stub&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;/Modification&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;/Peptide&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;Peptide id=&quot;p2_t&quot;&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;PeptideSequence&gt;TIDYK&lt;/PeptideSequence&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;Modification location=&quot;4&quot; monoisotopicMassDelta=&quot;85.982635&quot;&gt;</span></p><p class="c13 c49"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003393&quot; </span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; name=&quot;search modification id ref&quot; value=&quot;</span><span class="c73">DSSO_crosslink_stub_t</span><span class="c2">&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;cvParam accession=&quot;UNIMOD:1842&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:DSSO&quot;/&gt;</span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c57 c29 c26">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; value=&quot;T&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c13"><span class="c28">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c57 c29 c20">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003346&quot; name=&quot;cleavable crosslinker stub&quot;/&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &lt;/Modification&gt;</span></p><p class="c13"><span class="c2">&nbsp; &nbsp; &nbsp; &nbsp; &lt;/Peptide&gt;</span></p><p class="c13 c15"><span class="c2"></span></p><p class="c13 c15"><span class="c2"></span></p><p class="c13"><span class="c75 c50">Figure 6. </span><span class="c75 c50">XML </span><span class="c50">snippet</span><span class="c75 c50">&nbsp;showing the encoding of modifications from cleavable crosslinkers.</span><span class="c75 c50">&nbsp;</span><span class="c75">The new CV terms are shown: </span><span class="c20">&ldquo;crosslinker stub&rdquo; (MS:1003346)</span><span class="c75">&nbsp;and</span><span class="c75">&nbsp;</span><span class="c26">&ldquo;Unimod derivative code&rdquo; (MS:1003347)</span><span>. This example also uses the new CV term &quot;search modification id ref&quot; (</span><span>MS:1003393</span><span>) to reference the corresponding &lt;SearchModification&gt; elements. &nbsp;</span></p><p class="c13 c15"><span class="c9"></span></p><h2 class="c17" id="h.2qumsicj7vey"><span class="c47 c29">3.4 Encoding Identified Crosslinks in &lt;SpectrumIdentificationItem&gt; Elements </span></h2><a id="id.z3nqrkeid8o3"></a><h3 class="c60" id="h.59aaerpg4g8l"><span class="c29 c86 c50 c52">3.4.1 Identifications of Crosslinked Peptides</span></h3><p class="c14"><span class="c63 c102 c50">Path:</span><span class="c56 c48">&nbsp;/MzIdentML/DataCollection/AnalysisData/SpectrumIdentificationList/SpectrumIdentificationResult</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">&lt;SpectrumIdentificationResult&gt; elements report the evidence associated with the identification of particular peptides. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>A</span><span>&nbsp;pair of crosslinked peptides within a given &lt;SpectrumIdentificationResult&gt; MUST be reported as two instances of &lt;SpectrumIdentificationItem&gt; having a shared </span><span>local</span><span>&nbsp;unique identifier as the value for the CV term &ldquo;</span><span>crosslink</span><span>&nbsp;spectrum identification item&rdquo; (MS:1002511). Locally unique means unique within the containing &lt;SpectrumIdentificationResult&gt;. See </span><span>Figure 7(i</span><span>). The rules governing the use of the &ldquo;crosslink spectrum identification item&rdquo; CV term are given in Figure 7(ii). </span></p><hr style="page-break-before:always;display:none;"><p class="c14 c40 c15"><span class="c10"></span></p><p class="c14 c40"><span class="c50">(i)</span><span class="c0">&nbsp;</span></p><p class="c14 c40"><span class="c9">&lt;SpectrumIdentificationResult spectraData_ref=&quot;SID_1&quot; spectrumID=&quot;index=2776&quot; id=&quot;SIR_1&quot;&gt;</span></p><p class="c6 c40 c15"><span class="c9"></span></p><p class="c6 c40"><span class="c12 c20">&lt;SpectrumIdentificationItem passThreshold=&quot;true&quot; rank=&quot;</span><span class="c71 c50 c20">1</span><span class="c12 c20">&quot;</span><span class="c9 c65">&nbsp;</span></p><p class="c6 c27"><span class="c12">peptide_ref=&quot;30491856_30492180_2_4_p1&quot; </span><span class="c9 c22">experimentalMassToCharge= &quot;569.7912&quot; </span></p><p class="c6 c27"><span class="c12 c22">calculatedMassToCharge=&quot;569.79054&quot; chargeState=&quot;4&quot;</span><span class="c9">&nbsp;id=&quot;SII_1_1&quot;&gt;</span></p><p class="c6 c40 c15"><span class="c9"></span></p><p class="c14 c33 c40"><span class="c9">&lt;PeptideEvidenceRef peptideEvidence_ref=&quot;pepevid_psm121558473_pep30491845_protP02768-A_target_535&quot;/&gt;</span></p><p class="c14 c33 c40 c15"><span class="c9"></span></p><p class="c6 c27"><span class="c12 c42">&lt;cvParam accession=&quot;MS:1002511&quot; cvRef=&quot;PSI-MS&quot; value=&quot;</span><span class="c71 c50 c42">1</span><span class="c9 c42">&quot; </span></p><p class="c14 c27 c33"><span class="c12 c42">name=</span><span class="c12 c42">&quot;crosslink</span><span class="c9 c42">&nbsp;spectrum identification item&quot;/&gt;</span></p><p class="c14 c8 c33 c40"><span class="c9"></span></p><p class="c6 c27"><span class="c9 c26">&lt;cvParam accession=&quot;MS:1002545&quot; cvRef=&quot;PSI-MS&quot; </span></p><p class="c14 c27 c33"><span class="c9 c26">value=&quot;1.3111826921077734&quot; name=&quot;xi:score&quot;/&gt;</span></p><p class="c14 c8 c33 c40"><span class="c9 c26"></span></p><p class="c6 c27"><span class="c9">&lt;cvParam accession=&quot;MS:1003344&quot; cvRef=&quot;PSI-MS&quot; value=&quot;54321.a&quot; name=&quot;Residue pair ref&quot;/&gt;</span></p><p class="c14 c40 c15"><span class="c9"></span></p><p class="c6 c40"><span class="c9">&lt;/SpectrumIdentificationItem&gt;</span></p><p class="c6 c40 c15"><span class="c9"></span></p><p class="c6 c40"><span class="c12 c20">&lt;SpectrumIdentificationItem passThreshold=&quot;true&quot; rank=&quot;</span><span class="c71 c50 c20">1</span><span class="c12 c20">&quot;</span><span class="c9 c65">&nbsp;</span></p><p class="c6 c27"><span class="c12">peptide_ref=&quot;30491715_30491845_3_7_p0&quot; </span><span class="c9 c22">experimentalMassToCharge= &quot;569.7912&quot; </span></p><p class="c6 c27"><span class="c12 c22">calculatedMassToCharge=&quot;569.79054&quot; chargeState=&quot;4&quot;</span><span class="c9">&nbsp;id=&quot;SII_1_2&quot;&gt;</span></p><p class="c14 c33 c40 c15"><span class="c9"></span></p><p class="c14 c33 c40"><span class="c9">&lt;PeptideEvidenceRef </span></p><p class="c14 c33 c40"><span class="c9">peptideEvidence_ref=&quot;pepevid_psm121558473_pep30491715_protP02768-A_target_411&quot;/&gt;</span></p><p class="c14 c33 c40 c15"><span class="c9"></span></p><p class="c14 c33 c40"><span class="c12 c42">&lt;cvParam accession=&quot;MS:1002511&quot; cvRef=&quot;PSI-MS&quot; value=&quot;</span><span class="c71 c50 c42">1</span><span class="c9 c42">&quot; </span></p><p class="c14 c27 c33"><span class="c9 c42">name=&quot;crosslink spectrum identification item&quot;/&gt;</span></p><p class="c14 c8 c33 c40"><span class="c9"></span></p><p class="c14 c33 c40"><span class="c9 c26">&lt;cvParam accession=&quot;MS:1002545&quot; cvRef=&quot;PSI-MS&quot; </span></p><p class="c14 c27 c33"><span class="c9 c26">value=&quot;1.3111826921077734&quot; name=&quot;xi:score&quot;/&gt;</span></p><p class="c14 c8 c33 c40"><span class="c9 c26"></span></p><p class="c6 c27"><span class="c12">&lt;cvParam accession=&quot;MS:1003344&quot; cvRef=&quot;PSI-MS&quot; value=&quot;54321.b&quot; name=&quot;Residue pair ref&quot;/&gt;</span></p><p class="c14 c8 c33 c40"><span class="c9"></span></p><p class="c6 c40"><span class="c9">&lt;/SpectrumIdentificationItem&gt;</span></p><p class="c6 c40 c15"><span class="c9"></span></p><p class="c14 c40"><span class="c9">&lt;/SpectrumIdentificationResult&gt;</span></p><p class="c14 c40 c15"><span class="c9"></span></p><p class="c14"><span class="c50">(ii)</span><span>&nbsp;If a crosslinked pair of peptides has been identified:</span></p><ul class="c77 lst-kix_dytpm85i4qaq-0 start"><li class="c6 c19 li-bullet-0"><span class="c20">T</span><span class="c20">here MUST be</span><span class="c50 c20">&nbsp;two</span><span class="c20">&nbsp;&lt;SpectrumIdentificationItem&gt; elements with the same rank value.</span><span class="c0">&nbsp;</span></li><li class="c6 c19 li-bullet-0"><span class="c42">Both MUST have the &ldquo;crosslink spectrum identification item&rdquo; cvParam, and the value acts as a </span><span class="c50 c42">local</span><span class="c42">&nbsp;identifier within the &lt;SpectrumIdentificationResult&gt; to group these two elements together.</span><span>&nbsp;</span></li><li class="c6 c19 li-bullet-0"><span class="c22">The experimentalMassToCharge, calculatedMassToCharge and chargeState MUST be identical over both SII elements, indicating the overall values for the pair.</span></li><li class="c6 c19 li-bullet-0"><span class="c26">If the search engine applies a score to the paired identification, both &lt;SpectrumIdentificationItem&gt; elements MUST have the same cvParam capturing the value.</span><span class="c0">&nbsp;</span></li><li class="c6 c19 li-bullet-0"><span class="c0">The two &lt;SpectrumIdentificationItem&gt; elements MAY also have independent scores for the two chains (not shown).</span></li></ul><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c50">Figure 7.</span><span>&nbsp;</span><span class="c50">Encoding the identification of a pair of crosslinked peptides. </span><span class="c0">(i) Example XML snippet. (ii) The rules governing the use of &ldquo;crosslink spectrum identification item&rdquo;.</span></p><hr style="page-break-before:always;display:none;"><p class="c14 c15"><span class="c0"></span></p><h3 class="c60" id="h.byui0pnkdhua"><span class="c50 c52">3.4.2 Identifications of Noncovalently Associated </span><span class="c50 c52">Peptides</span><span class="c29 c86 c50 c52">&nbsp;</span></h3><p class="c14"><span class="c63 c102 c50">Path:</span><span class="c56 c48">&nbsp;/MzIdentML/DataCollection/AnalysisData/SpectrumIdentificationList/SpectrumIdentificationResult</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c63 c36">New supported use case in this extension - noncovalently associated peptides:</span><span>&nbsp;</span><span>m</span><span class="c0">zIdentML 1.2.1 introduces a new CV term &ldquo;noncovalently associated peptides spectrum identification item&rdquo; (MS:1003331) to encode such identifications (see &sect;1.2). It operates in the same way as &ldquo;crosslink spectrum identification item&rdquo;, by using the value of the CV term to group the identifications together, see Figure 8(i).</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">As indicated above, to use the &ldquo;noncovalently associated peptides spectrum identification item&rdquo; (MS:1003331), the element &lt;AdditionalSearchParams&gt; MUST contain the CV term &ldquo;noncovalently associated peptides search&rdquo; (MS:1003330), see Figure 3.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The rules governing the use of the &ldquo;noncovalently associated peptides spectrum identification item&rdquo; CV term are given in Figure 8 (ii) and are analogous to those governing the use of &ldquo;crosslink spectrum identification item&rdquo;. The peptides referred to will be linear, uncrosslinked peptides. </span></p><p class="c14"><span class="c24">&nbsp;</span><hr style="page-break-before:always;display:none;"></p><p class="c14 c40"><span class="c50">(i)</span><span class="c0">&nbsp;</span></p><p class="c14 c40"><span class="c9">&lt;SpectrumIdentificationResult spectraData_ref=&quot;SID_1&quot; spectrumID=&quot;index=2776&quot; id=&quot;SIR_1&quot;&gt;</span></p><p class="c6 c40 c15"><span class="c9"></span></p><p class="c6 c40"><span class="c12 c20">&lt;SpectrumIdentificationItem passThreshold=&quot;true&quot; rank=&quot;</span><span class="c71 c50 c20">1</span><span class="c12 c20">&quot;</span><span class="c9 c65">&nbsp;</span></p><p class="c6 c27"><span class="c12">peptide_ref=p1&quot; </span><span class="c9 c22">experimentalMassToCharge= &quot;569.7912&quot; </span></p><p class="c6 c27"><span class="c12 c22">calculatedMassToCharge=&quot;569.79054&quot; chargeState=&quot;4&quot;</span><span class="c9">&nbsp;id=&quot;SII_1_1&quot;&gt;</span></p><p class="c6 c40 c15"><span class="c9"></span></p><p class="c14 c33 c40"><span class="c9">&lt;PeptideEvidenceRef peptideEvidence_ref=&quot;pepevid_pep_1&quot;/&gt;</span></p><p class="c14 c33 c40 c15"><span class="c9"></span></p><p class="c6 c27"><span class="c12 c42">&lt;cvParam accession=&quot;MS:1003331&quot; cvRef=&quot;PSI-MS&quot; value=&quot;</span><span class="c71 c50 c42">1</span><span class="c9 c42">&quot; </span></p><p class="c14 c27 c33"><span class="c9 c42">name=&quot;noncovalently associated peptides spectrum identification item&quot;/&gt;</span></p><p class="c14 c8 c33 c40"><span class="c9"></span></p><p class="c6 c27"><span class="c9 c26">&lt;cvParam accession=&quot;MS:1002545&quot; cvRef=&quot;PSI-MS&quot; </span></p><p class="c14 c27 c33"><span class="c9 c26">value=&quot;1.3111826921077734&quot; name=&quot;xi:score&quot;/&gt;</span></p><p class="c14 c8 c33 c40"><span class="c9"></span></p><p class="c6 c40"><span class="c9">&lt;/SpectrumIdentificationItem&gt;</span></p><p class="c6 c40 c15"><span class="c9"></span></p><p class="c6 c40"><span class="c12 c20">&lt;SpectrumIdentificationItem passThreshold=&quot;true&quot; rank=&quot;</span><span class="c71 c50 c20">1</span><span class="c12 c20">&quot;</span><span class="c9 c65">&nbsp;</span></p><p class="c6 c27"><span class="c12">peptide_ref=&quot;p2&quot; </span><span class="c9 c22">experimentalMassToCharge= &quot;569.7912&quot; </span></p><p class="c6 c27"><span class="c12 c22">calculatedMassToCharge=&quot;569.79054&quot; chargeState=&quot;4&quot;</span><span class="c9">&nbsp;id=&quot;SII_1_2&quot;&gt;</span></p><p class="c14 c33 c40 c15"><span class="c9"></span></p><p class="c14 c33 c40"><span class="c9">&lt;PeptideEvidenceRef peptideEvidence_ref=&quot;pepevid_pep_2&quot;/&gt;</span></p><p class="c14 c33 c40 c15"><span class="c9"></span></p><p class="c6 c27"><span class="c12 c42">&lt;cvParam accession=&quot;MS:1003331&quot; cvRef=&quot;PSI-MS&quot; value=&quot;</span><span class="c71 c50 c42">1</span><span class="c9 c42">&quot; </span></p><p class="c14 c27 c33"><span class="c9 c42">name=&quot;noncovalently associated peptides spectrum identification item&quot;/&gt;</span></p><p class="c14 c8 c33 c40"><span class="c9"></span></p><p class="c14 c33 c40"><span class="c9 c26">&lt;cvParam accession=&quot;MS:1002545&quot; cvRef=&quot;PSI-MS&quot; </span></p><p class="c14 c27 c33"><span class="c9 c26">value=&quot;1.3111826921077734&quot; name=&quot;xi:score&quot;/&gt;</span></p><p class="c14 c8 c33 c40"><span class="c9"></span></p><p class="c6 c40"><span class="c9">&lt;/SpectrumIdentificationItem&gt;</span></p><p class="c6 c40 c15"><span class="c9"></span></p><p class="c14"><span class="c9">&lt;/SpectrumIdentificationResult&gt;</span></p><p class="c14 c15"><span class="c9"></span></p><p class="c14"><span class="c50">(ii)</span><span>&nbsp;If a pair of </span><span class="c50">noncovalently associated peptides</span><span class="c0">&nbsp;has been identified:</span></p><ul class="c77 lst-kix_i0lzlnefeb2-0 start"><li class="c6 c19 li-bullet-0"><span class="c20">T</span><span class="c20">here MUST be</span><span class="c50 c20">&nbsp;two</span><span class="c20">&nbsp;&lt;SpectrumIdentificationItem&gt; elements with the same rank value.</span><span class="c0">&nbsp;</span></li><li class="c6 c19 li-bullet-0"><span class="c42">Both MUST have the &ldquo;noncovalently associated peptides spectrum identification item&rdquo; cvParam, and the value acts as a </span><span class="c50 c42">local</span><span class="c42">&nbsp;identifier within the &lt;SpectrumIdentificationResult&gt; to group these two elements together.</span><span>&nbsp;</span></li><li class="c6 c19 li-bullet-0"><span class="c0 c22">The experimentalMassToCharge, calculatedMassToCharge and chargeState MUST be identical over both SII elements, indicating the overall values for the pair.</span></li><li class="c6 c19 li-bullet-0"><span class="c26">If the search engine applies a score to the paired identification, both &lt;SpectrumIdentificationItem&gt; elements MUST have the same cvParam capturing the value.</span><span class="c0">&nbsp;</span></li><li class="c6 c19 li-bullet-0"><span class="c0">The two &lt;SpectrumIdentificationItem&gt; elements MAY also have independent scores for the two chains (not shown).</span></li></ul><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c50">Figure 8.</span><span>&nbsp;</span><span class="c50">Encoding the identification of a pair of noncovalently associated peptides. </span><span>(i) Example XML </span><span>snippet</span><span class="c0">. (ii) The rules governing the use of &ldquo;noncovalently associated peptides spectrum identification item&rdquo;.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><hr style="page-break-before:always;display:none;"><p class="c14 c15"><span class="c0"></span></p><h3 class="c60" id="h.8omxepcmkvte"><span class="c50 c52">3.4.3 Identifications of an Internally Linked Peptide</span><span class="c29 c86 c106">&nbsp;</span></h3><p class="c14"><span class="c63 c50 c102">Path:</span><span class="c56 c48">&nbsp;/MzIdentML/DataCollection/AnalysisData/SpectrumIdentificationList/SpectrumIdentificationResult</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c36 c63">New supported use case in this extension - internally linked peptide:</span><span>&nbsp;</span><span>mzIdentML 1.3.0 introduces a new CV term &ndash; &ldquo;</span><span>loop</span><span>link spectrum identification item&rdquo; (MS:1003329) &ndash; to allow the encoding of internally linked peptides (a.k.a. &ldquo;looplinks&rdquo;), see Figure 9. The &lt;SpectrumIdentificationItem&gt; element will refer to a &lt;Peptide&gt; containing both crosslink donor and crosslink acceptor modifications (as shown in Figure 5(iii)). <br></span></p><p class="c14 c40 c15"><span class="c0"></span></p><p class="c14 c40 c15"><span class="c0"></span></p><p class="c14 c40 c15"><span class="c0"></span></p><p class="c14 c40"><span class="c9">&lt;SpectrumIdentificationResult spectraData_ref=&quot;SID_1&quot; spectrumID=&quot;index=2776&quot; id=&quot;SIR_1&quot;&gt;</span></p><p class="c6 c40 c15"><span class="c9"></span></p><p class="c6 c40"><span class="c12">&lt;SpectrumIdentificationItem passThreshold=&quot;true&quot; rank=&quot;</span><span class="c71 c50">1</span><span class="c9">&quot; </span></p><p class="c6 c27"><span class="c12 c20">peptide_ref=&quot;</span><span class="c71 c50 c20">looplink_p1</span><span class="c12 c20">&quot;</span><span class="c9">&nbsp;experimentalMassToCharge= &quot;569.7912&quot; </span></p><p class="c6 c27"><span class="c9">calculatedMassToCharge=&quot;569.79054&quot; chargeState=&quot;4&quot; id=&quot;SII_1_1&quot;&gt;</span></p><p class="c6 c40 c15"><span class="c9"></span></p><p class="c14 c33 c40"><span class="c12 c20">&lt;PeptideEvidenceRef peptideEvidence_ref=&quot;</span><span class="c71 c50 c20">looplink_p1_pep_evid</span><span class="c9 c20">&quot;/&gt;</span></p><p class="c14 c33 c40 c15"><span class="c9"></span></p><p class="c6 c27"><span class="c9 c42">&lt;cvParam accession=&quot;MS:1003329&quot; cvRef=&quot;PSI-MS&quot;</span></p><p class="c14 c27 c33"><span class="c9 c42">name=&quot;looplink spectrum identification item&quot;/&gt;</span></p><p class="c14 c8 c33 c40"><span class="c9"></span></p><p class="c6 c27"><span class="c9">&lt;cvParam accession=&quot;MS:1002545&quot; cvRef=&quot;PSI-MS&quot; </span></p><p class="c14 c27 c33"><span class="c9">value=&quot;1.3111826921077734&quot; name=&quot;xi:score&quot;/&gt;</span></p><p class="c14 c8 c33 c40"><span class="c9"></span></p><p class="c6 c40"><span class="c9">&lt;/SpectrumIdentificationItem&gt;</span></p><p class="c14 c40 c15"><span class="c9"></span></p><p class="c6 c40 c15"><span class="c9"></span></p><p class="c14 c40"><span class="c12">&lt;/SpectrumIdentificationResult&gt;</span></p><p class="c14 c15"><span class="c0 c24"></span></p><p class="c14 c15"><span class="c0 c24"></span></p><p class="c14"><span class="c50">Figure 9.</span><span>&nbsp;</span><span class="c50">XML </span><span class="c50">snippet including the encoding of an i</span><span class="c50">dentification of an internally linked peptide.</span><span>&nbsp;Within a &lt;SpectrumIdentificationResult&gt;, a </span><span class="c42">&lt;SpectrumIdentificationItem&gt; element may be marked as referring to a looplink containing peptide by including the CV term &ldquo;looplink spectrum identification item&rdquo; (MS:1003329) CV term</span><span>. </span><span class="c20">This &lt;SpectrumIdentificationItem&gt; will refer to a &lt;Peptide&gt; containing both crosslink donor and crosslink acceptor modifications (as shown in Figure 5(iii))</span><span class="c0">. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><hr style="page-break-before:always;display:none;"><h2 class="c32 c110 c113" id="h.n1cf6o5dg00g"><span class="c30 c29"></span></h2><h1 class="c32 c110" id="h.eskj4bsv28fu"><span class="c43">4. Scores and Thresholds</span></h1><h2 class="c17" id="h.4y36xqq2ftho"><span class="c88 c50">4.1 Introduction</span><a id="id.nvacze2cepgq"></a></h2><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">This section addresses the encoding of error control procedures. This consists of encoding scores (&sect;4.3) and the corresponding thresholds (&sect;4.2) applied to those scores. The contents of this section are all optional; at the PSM level, providing threshold information and identifications that fall below the given significance threshold is encouraged.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c95"><span class="c36">&ldquo;Depending on the intended purpose of the file, the file producer MAY wish to report a number of identifications that fall below the given significance threshold, for example to allow global statistical analyses to be performed which are not possible if only identifications passing the threshold are reported.&rdquo;</span><span class="c0">&nbsp;(Section 7.4 of the main mzIdentML 1.3.0 specification document)</span></p><p class="c14"><span class="c0">mzIdentML also provides the option not to encode the peptide spectrum matches that fell below the threshold applied. (&sect;4.2) &nbsp;</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The correspondence between scores and the applied thresholds is indicated by using the same CV term for both.</span><span class="c0">&nbsp;That is, the same CV term will be used within the &lt;Threshold&gt; element and within either the related &lt;SpectrumIdentificationItem&gt; element or the related &lt;ProteinDetectionHypothesis&gt; element.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">One specific type of score is an FDR (False Discovery Rate) score. Comments specific to FDR are in &sect;4.4.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>There are different points in the analysis at which thresholds may be applied </span><span class="c37"><a class="c18" href="https://www.google.com/url?q=https://paperpile.com/c/YrD2gH/kAbLF&amp;sa=D&amp;source=editors&amp;ust=1719221372835906&amp;usg=AOvVaw3DhN1CTr2_s0dDlhlcFbNU">[5]</a></span><span>&nbsp;</span><span class="c37"><a class="c18" href="https://www.google.com/url?q=https://paperpile.com/c/YrD2gH/ghaA0&amp;sa=D&amp;source=editors&amp;ust=1719221372836204&amp;usg=AOvVaw3CVG-wvk1__ceQAoEhbdXV">[6]</a></span><span class="c0">. These correspond to different levels of consolidation at which analyses may be performed. Scores and thresholds are encoded differently in mzIdentML depending on the level of consolidation at which they were applied. For crosslinking studies encoded in mzIdentML, the possible levels are:</span></p><p class="c14 c15"><span class="c0"></span></p><ul class="c77 lst-kix_4wwjrzyyefv1-0 start"><li class="c6 c19 li-bullet-0"><span>crosslink containing PSM</span><span class="c0">&nbsp;(also known as Crosslink Spectrum Match, CSM), see &sect;4.3.1,</span></li><li class="c6 c19 li-bullet-0"><span class="c0">unique peptide-pair, see &sect;4.3.2,</span></li><li class="c6 c19 li-bullet-0"><span class="c0">unique residue-pair, see &sect;4.3.3,</span></li><li class="c6 c19 li-bullet-0"><span class="c0">protein-protein interaction (PPI) see &sect;4.3.3.</span></li></ul><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">Unique residue-pair and protein-protein interaction level scores are described in the same section as they are encoded using the same mechanism.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The example file </span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=https://github.com/HUPO-PSI/mzIdentML/blob/master/examples/1_3_0examples/crosslinking/scores_and_thresholds_1_3_0_draft.mzid&amp;sa=D&amp;source=editors&amp;ust=1719221372837172&amp;usg=AOvVaw21tZHIIP5JmTg2K0BU7NMK">scores_and_thresholds_1_3_0_draft.mzid</a></span><span class="c0">&nbsp;gives a simplified example containing two crosslinks and shows scores and thresholds applied at all four levels. Figures 10, 11, 14 and 15 are XML-snippets from that example file.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>mzIdentML allows peptide-level scores to be associated with &ldquo;unique peptides&rdquo; (</span><span>not arbitrary groups of peptides</span><span class="c0">). There are three mutually exclusive definitions of &ldquo;unique peptide&rdquo;:</span></p><ul class="c77 lst-kix_2jqw2dfnv53p-0 start"><li class="c6 c19 li-bullet-0"><span class="c0">&ldquo;group PSMs by sequence&rdquo; (MS:1002496);</span></li><li class="c6 c19 li-bullet-0"><span class="c0">&ldquo;group PSMs by sequence with modifications&rdquo; (MS:1002497);</span></li><li class="c6 c19 li-bullet-0"><span class="c0">&ldquo;group PSMs by sequence with modifications and charge&rdquo; (MS:1002498).</span></li></ul><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>If peptide level (re)scoring is used, exactly one of these CV terms must be placed in the &lt;AdditionalSearchParams&gt; element to state the definition of &ldquo;unique peptide&rdquo; in use (see Section 5.2.7 of the main specification document). </span><span class="c0">As these are mutually exclusive, an error control procedure which uses more than one definition of &ldquo;unique peptide&rdquo; cannot be fully captured by mzIdentML. </span></p><h2 class="c17" id="h.gkox7noip2j8"><span class="c47 c29">4.2 Thresholds</span></h2><p class="c14"><span class="c0">Section 7.4 of the main mzIdentML specification document gives general guidance on the encoding of thresholds and what has passed them. Note that thresholds are encoded in two different places: in the &lt;SpectrumIdentificationProtocol&gt; element and in the &lt;ProteinDetectionProtocol&gt; element. In both cases, they are encoded using CV terms inside a &lt;Threshold&gt; element, see Figure 10. </span></p><p class="c32"><span class="c0">The &lt;Threshold&gt; element inside &lt;SpectrumIdentificationProtocol&gt; gives the thresholds associated with &lt;SpectrumIdentificationItem&gt; elements. These thresholds apply at the crosslinked PSM level and at a unique peptide level. </span></p><p class="c32"><span>Analogously, the &lt;Threshold&gt; element inside &lt;ProteinDetectionProtocol&gt; includes the thresholds associated with &lt;ProteinDetectionHypothesis&gt; elements.</span><span>&nbsp;</span><span>These thresholds apply at the unique residue-pair level and PPI level.</span></p><p class="c95"><span>The elements &lt;SpectrumIdentificationItem&gt; and &lt;ProteinDetectionHypothesis&gt; have a mandatory Boolean attribute </span><span class="c36">passThreshold</span><span>&nbsp;that allows a file producer to indicate that an identification has passed the given thresholds or that it has been manually validated.</span></p><p class="c95"><span>The </span><span class="c36">passThreshold</span><span class="c0">&nbsp;attribute of &lt;SpectrumIdentificationItem&gt; relates only to the passing of PSM-level thresholds (see Section 5.2.7 of the main specification document, final paragraph therein). </span></p><p class="c95"><span>To enable additional thresholding at the peptide-pair level in the context of crosslinking, a new CV term is required for all PSMs (&ldquo;peptide-pair passes threshold&rdquo;, MS:1003339) </span><span>as shown in </span><span>Figure 11</span><span class="c0">. This is similar to the general guidance on peptide level thresholds given in Section 5.2.7 of the main specification document. </span></p><p class="c95"><span>The</span><span class="c36">&nbsp;passThreshold</span><span>&nbsp;attribute of </span><span>&lt;ProteinDetectionHypothesis&gt;</span><span>&nbsp;only relates to the presence or absence of proteins, it is not directly related to the identification of crosslinks. Whether or not residue-pairs or PPIs have passed significance thresholds is encoded by including</span><span>&nbsp;the new CV terms</span><span>&nbsp;&ldquo;residue-pair passes threshold&rdquo; (MS:1003340) or &ldquo;protein-protein interaction passes threshold&rdquo; (MS:1003341) in the &lt;ProteinDetectionHypothesis&gt; element. The values of these CV terms include an identifier that associates them with a specific residue pair or PPI, see </span><span>Figure 14</span><span class="c0">.</span></p><p class="c95"><span class="c0">At each level of consolidation there may be multiple scores. Therefore, for each level there is a mechanism for encoding whether the identification passed when all scores are considered:</span></p><ul class="c77 lst-kix_jb5m9ajaek8x-0 start"><li class="c95 c19 c49 li-bullet-0"><span>for PSM-level identifications this is the </span><span class="c36">passThreshold</span><span class="c0">&nbsp;attribute of &lt;SpectrumIdentificiationItem&gt;;</span></li><li class="c95 c19 c49 li-bullet-0"><span class="c0">at peptide-pair level it is the &ldquo;peptide-pair passes threshold&rdquo; (MS:1003339) CV term;</span></li><li class="c95 c19 c49 li-bullet-0"><span class="c0">at residue-pair level it is the &ldquo;residue-pair passes threshold&rdquo; (MS:1003340) CV term;</span></li><li class="c95 c19 c49 li-bullet-0"><span class="c0">and for PPIs it is the &ldquo;protein-pair passes threshold&rdquo; (MS:1003341) CV term.</span></li></ul><p class="c95"><span>If the file producer does not want to indicate that thresholds have been set, </span><span>all identification elements (&lt;SpectrumIdentificationItem&gt; and &lt;ProteinDetectionHypothesis&gt;) MUST have the attribute passThreshold = &ldquo;true&quot;</span><span>&nbsp;and the &ldquo;no threshold&quot; CV term should be provided within the </span><span>&lt;SpectrumIdentificationProtocol&gt;</span><span>&nbsp;and &lt;ProteinDetectionProtocol&gt;</span><span class="c0">&nbsp;(Section 7.4 of the main mzIdentML 1.3.0 specification document). In this case, the new &ldquo;residue-pair passes threshold&quot; (MS:1003340) and &ldquo;protein-protein interaction passes threshold&quot; (MS:1003341) CV terms can be omitted.</span></p><hr style="page-break-before:always;display:none;"><p class="c3 c15"><span class="c9"></span></p><p class="c3"><span class="c12">&lt;</span><span class="c12">AnalysisProtocolCollection </span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp;&lt;</span><span class="c12">SpectrumIdentificationProtocol </span><span class="c12">analysisSoftware_ref</span><span class="c12">=&quot;xiFDR_id&quot; </span><span class="c12">id</span><span class="c12">=&quot;SearchProtocol_1_17022&quot;</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">SearchType</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">cvParam </span><span class="c12">cvRef</span><span class="c12">=&quot;PSI-MS&quot; </span><span class="c12">accession</span><span class="c12">=&quot;MS:1001083&quot; </span><span class="c12">name</span><span class="c12">=&quot;ms-ms search&quot;</span><span class="c9">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c12">SearchType</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">AdditionalSearchParams</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">cvParam </span><span class="c12">cvRef</span><span class="c12">=&quot;PSI-MS&quot; </span><span class="c12">accession</span><span class="c12">=&quot;MS:1001211&quot; </span><span class="c12">name</span><span class="c12">=&quot;parent mass type mono&quot;</span><span class="c9">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">cvParam </span><span class="c12">cvRef</span><span class="c12">=&quot;PSI-MS&quot; </span><span class="c12">accession</span><span class="c12">=&quot;MS:1002494&quot; </span><span class="c12">name</span><span class="c12">=&quot;crosslinking search&quot;</span><span class="c9">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">cvParam </span><span class="c12">cvRef</span><span class="c12">=&quot;PSI-MS&quot; </span><span class="c12">accession</span><span class="c12">=&quot;MS:1001256&quot; </span><span class="c12">name</span><span class="c12">=&quot;fragment mass type mono&quot;</span><span class="c9">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c12 c24">&nbsp;&lt;</span><span class="c12 c24">cvParam </span><span class="c12 c24">cvRef</span><span class="c12 c24">=&quot;PSI-MS&quot; </span><span class="c12 c24">accession</span><span class="c12 c24">=&quot;MS:1002490&quot; </span><span class="c12 c24">name</span><span class="c12 c24">=&quot;peptide-level scoring&quot;</span><span class="c9 c24">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c12 c24">&nbsp;&lt;</span><span class="c12 c24">cvParam </span><span class="c12 c24">cvRef</span><span class="c12 c24">=&quot;PSI-MS&quot; </span><span class="c12 c24">accession </span><span class="c12 c24">= &quot;MS:1002496&quot; </span><span class="c12 c24">name</span><span class="c12 c24">=&quot;group PSMs by sequence&quot;</span><span class="c9 c24">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">cvParam </span><span class="c12">cvRef</span><span class="c12">=&quot;PSI-MS&quot; </span><span class="c12">accession</span><span class="c9">=&quot;MS:1003343&quot; </span></p><p class="c3 c35 c49"><span class="c12">name</span><span class="c9">=&quot;FDR applied separately to self crosslinks </span></p><p class="c3 c33 c35"><span class="c12">and protein heteromeric crosslinks&quot; </span><span class="c9">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">cvParam </span><span class="c12">accession</span><span class="c12">=&quot;MS:1001118&quot; </span><span class="c12">name</span><span class="c12">=&quot;param: b ion&quot; </span><span class="c12">cvRef</span><span class="c12">=&quot;PSI-MS&quot; </span><span class="c9">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">cvParam </span><span class="c12">accession</span><span class="c12">=&quot;MS:1001262&quot; </span><span class="c12">name</span><span class="c12">=&quot;param: y ion&quot; </span><span class="c12">cvRef</span><span class="c12">=&quot;PSI-MS&quot; </span><span class="c9">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c12">AdditionalSearchParams</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">ModificationParams .../</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">Enzymes </span><span class="c12">.../</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">FragmentTolerance</span><span class="c9">&nbsp;.../&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">ParentTolerance</span><span class="c9">&nbsp;.../&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">Threshold</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c42">&lt;</span><span class="c12 c42">cvParam </span><span class="c12 c42">cvRef</span><span class="c12 c42">=&quot;PSI-MS&quot; </span><span class="c12 c42">accession</span><span class="c9 c42">=&quot;MS:1003337&quot; </span></p><p class="c3 c38"><span class="c12 c42">name</span><span class="c12 c42">=&quot;crosslinked PSM-level global FDR&quot; </span><span class="c12 c42">value</span><span class="c12 c42">=&quot;0.05&quot;</span><span class="c9 c42">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c26">&lt;</span><span class="c12 c26">cvParam </span><span class="c12 c26">cvRef</span><span class="c12 c26">=&quot;PSI-MS&quot; </span><span class="c12 c26">accession</span><span class="c9 c26">=&quot;MS:1003338&quot; </span></p><p class="c3 c33 c35"><span class="c12 c26">name</span><span class="c12 c26">=&quot;peptide-pair sequence-level global FDR&quot; </span><span class="c12 c26">value</span><span class="c12 c26">=&quot;0.05&quot;</span><span class="c9 c26">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c12">Threshold</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &lt;/</span><span class="c12">SpectrumIdentificationProtocol</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &lt;</span><span class="c12">ProteinDetectionProtocol </span><span class="c12">analysisSoftware_ref</span><span class="c12">=&quot;xiFDR_id&quot; </span><span class="c12">id</span><span class="c12">=&quot;pdp1&quot;</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">Threshold</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c12 c20">&lt;</span><span class="c12 c20">cvParam </span><span class="c12 c20">cvRef</span><span class="c12 c20">=&quot;PSI-MS&quot; </span><span class="c12 c20">accession</span><span class="c9 c20">=&quot;MS:1002677&quot; </span></p><p class="c3 c33 c35"><span class="c12 c20">name</span><span class="c12 c20">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c20">value</span><span class="c12 c20">=&quot;0.05&quot;</span><span class="c9 c20">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c12 c22">&lt;</span><span class="c12 c22">cvParam </span><span class="c12 c22">cvRef</span><span class="c12 c22">=&quot;PSI-MS&quot; </span><span class="c12 c22">accession</span><span class="c9 c22">=&quot;MS:1002676&quot; </span></p><p class="c3 c33 c35"><span class="c12 c22">name</span><span class="c12 c22">=&quot;protein-pair-level global FDR&quot; </span><span class="c12 c22">value</span><span class="c12 c22">=&quot;0.05&quot;</span><span class="c9 c22">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp;&lt;/</span><span class="c12">Threshold</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &lt;/</span><span class="c12">ProteinDetectionProtocol</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12 c16">&lt;/</span><span class="c12">AnalysisProtocolCollection</span><span class="c11">&gt;</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c50">Figure 10. XML snippet showing the thresholds applied at all four levels of consolidation. </span><span>These are</span><span>&nbsp;- </span><span class="c42">PSM</span><span>, </span><span class="c26">peptide-pair</span><span>, </span><span class="c20">residue pair</span><span>&nbsp;and </span><span class="c22">PPI</span><span>. The CV terms </span><span class="c24">MS:1002490 and MS:1002496</span><span>&nbsp;are required to enable peptide level rescoring (mzIdentML main specification Section 5.2.7) and to state the definition of &lsquo;unique peptide&rsquo; being used. </span></p><p class="c95 c49 c15"><span class="c0"></span></p><hr style="page-break-before:always;display:none;"><h2 class="c17 c113" id="h.vy7v3ifgxpn8"><span class="c47 c29"></span></h2><h2 class="c17" id="h.k0tz6lupv5ds"><span class="c47 c29">4.3 Scores</span></h2><h3 class="c60" id="h.m17dfs7z5hvc"><span class="c29 c50 c52 c86">4.3.1 Match Level Scores</span></h3><p class="c14"><span class="c0">Match level scores are stored in &lt;SpectrumIdentificationItem&gt; elements. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The CV mapping rules for &lt;SpectrumIdentificationItem&gt; are straightforward &ndash; there is only one, which states &lsquo;MAY supply a child term of </span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri%3Dhttp%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMS_1001405&amp;sa=D&amp;source=editors&amp;ust=1719221372849720&amp;usg=AOvVaw3oiv1gHI9jYG33gzOMNBDw">MS:1001405 (spectrum identification result details)</a></span><span>&nbsp;one or more times&rsquo;.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>CV terms to encode match level scores must therefore be children of </span><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=https://www.ebi.ac.uk/ols/ontologies/ms/terms?iri%3Dhttp%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FMS_1001405&amp;sa=D&amp;source=editors&amp;ust=1719221372850315&amp;usg=AOvVaw280iCINkLr4jZex8JT100f">MS:1001405</a></span><span>&nbsp;in the CV&rsquo;s </span><span>&ldquo;is a&rdquo; hierarchy. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>Those which also meet the CV mapping rules for the &lt;Threshold&gt; element can also be used to encode the </span><span>Threshold</span><span>&nbsp;applied.</span><span class="c0">&nbsp;</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">See Section 7.11 of the main mzIdentML document for guidance specific to PSM-level scores for identifications based on multiple spectra.</span></p><h3 class="c60" id="h.ly8cjum3t06t"><span class="c29 c86 c50 c52">4.3.2 Peptide Level Scores</span></h3><p class="c14"><span class="c0">Peptide level scores are also stored in &lt;SpectrumIdentificationItem&gt; elements and everything in &sect;4.3.1 also applies here. </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>Section 5.2.7 of the main mzIdentML specification document describes the encoding of p</span><span>eptide-level scores and statistical measures. The encoding of crosslinking results </span><span>MAY also be combined with the peptide-level re-scoring mechanism described there</span><span class="c0">, but with specific CV terms for scores associated with crosslinked peptides rather than PSM-level terms (as stated in Section 5.2.7 of main specification document).</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>Where needed, </span><span>new CV terms for search specific scores of crosslinked peptides</span><span>&nbsp;should be added as a child of (i.e. with an &ldquo;is a&rdquo; relationship to) the CV term </span><span>&ldquo;interaction score derived from crosslinking&rdquo; (MS:1002664)</span><span>.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c3 c15"><span class="c11"></span></p><p class="c3"><span class="c12 c16">&lt;</span><span class="c7">SpectrumIdentificationList </span><span class="c12 c23">id</span><span class="c4">=&quot;SII_LIST_1_1&quot; </span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationResult </span><span class="c12 c23">spectrumID</span><span class="c4">=&quot;index=26630&quot; </span><span class="c12 c23">spectraData_ref</span><span class="c4">=&quot;SD_17022_recal_B210619_02_Lumos_ZC_CO_190_D2I_SDA-WT1.mgf&quot; </span><span class="c12 c23">id</span><span class="c4">=&quot;SIR_1&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationItem </span><span class="c12 c23">chargeState</span><span class="c4">=&quot;5&quot; </span><span class="c12 c23">experimentalMassToCharge</span><span class="c4 c29">=&quot;1135.3259479607323&quot; </span></p><p class="c3 c33 c35"><span class="c12 c23">calculatedMassToCharge</span><span class="c4 c29">=&quot;1135.3254335427703&quot; </span></p><p class="c3 c33 c35"><span class="c12 c23">peptide_ref</span><span class="c4">=&quot;16734061838_ISDKRAPSQGGLENEGVFEELLR_16734063165_GAEDEEEEEDVGFEQNFEEMLESVTR_4_9_p1&quot; </span><span class="c12 c23">rank</span><span class="c4 c29">=&quot;1&quot;</span></p><p class="c3 c33 c35"><span class="c12 c23">passThreshold</span><span class="c4">=&quot;false&quot; </span><span class="c12 c23">id</span><span class="c4">=&quot;SII_1_1&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">PeptideEvidenceRef </span><span class="c12 c23">peptideEvidence_ref</span><span class="c4">=&quot;pepevid_pep_16734063165&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002511&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;crosslink spectrum identification item&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;1&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002545&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;xi:score&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;25.929927957127177&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- crosslinked PSM level global FDR --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003337 </span><span class="c12 c23">name</span><span class="c4">=&quot;crosslinked PSM-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;0.06&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c36 c45 c58">&lt;!-- peptide pair global FDR --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002520&quot; </span><span class="c12 c23">value</span><span class="c4 c29">=&quot;GAEDEEEEEDVGFEQNFEEMLESVTR-ISDKRAPSQGGLENEGVFEELLR&quot; </span></p><p class="c3 c33 c35"><span class="c12 c23">name</span><span class="c4">=&quot;peptide group ID&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003338&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peptide-pair sequence-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;0.06&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003339&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peptide-pair passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;false&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- residue pair ref value=&quot;1.b&quot; --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam &nbsp;</span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003344&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;11.b&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;Residue-pair ref&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">SpectrumIdentificationItem</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationItem </span><span class="c12 c23">chargeState</span><span class="c4">=&quot;5&quot; </span><span class="c12 c23">experimentalMassToCharge</span><span class="c4 c29">=&quot;1135.3259479607323&quot;</span></p><p class="c3"><span class="c4">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c12 c23">calculatedMassToCharge</span><span class="c4 c29">=&quot;1135.3254335427703&quot;</span></p><p class="c3 c33 c35"><span class="c12 c23">peptide_ref</span><span class="c4">=&quot;16734061838_ISDKRAPSQGGLENEGVFEELLR_16734063165_GAEDEEEEEDVGFEQNFEEMLESVTR_4_9_p0&quot; </span><span class="c12 c23">rank</span><span class="c4 c29">=&quot;1&quot;</span></p><p class="c3 c33 c35"><span class="c12 c23">passThreshold</span><span class="c4">=&quot;false&quot; </span><span class="c12 c23">id</span><span class="c4">=&quot;SII_1_2&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">PeptideEvidenceRef </span><span class="c12 c23">peptideEvidence_ref</span><span class="c4">=&quot;pepevid_pep_16734061838&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002511&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;crosslink spectrum identification item&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;1&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002545&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;xi:score&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;25.929927957127177&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- crosslinked PSM level global FDR --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003337&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;crosslinked PSM-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;0.06&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- peptide pair global FDR --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002520&quot; </span><span class="c12 c23">value</span><span class="c4 c29">=&quot;GAEDEEEEEDVGFEQNFEEMLESVTR-ISDKRAPSQGGLENEGVFEELLR&quot; </span></p><p class="c3 c35 c49"><span class="c12 c23">name</span><span class="c4">=&quot;peptide group ID&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003338&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peptide-pair sequence-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;0.06&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003339&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peptide-pair passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;false&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- residue pair ref value=&quot;11.a&quot; --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam &nbsp;</span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003344&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;11.a&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;Residue-pair ref&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">SpectrumIdentificationItem</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1000797&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peak list scans&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;40560&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;/</span><span class="c7">SpectrumIdentificationResult</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationResult </span><span class="c12 c23">spectrumID</span><span class="c4">=&quot;index=23414&quot; </span><span class="c12 c23">spectraData_ref</span><span class="c4">=&quot;SD_17022_recal_B210619_04_Lumos_ZC_CO_190_D2I_SDA-WT3.mgf&quot; </span><span class="c12 c23">id</span><span class="c4">=&quot;SIR_2&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationItem </span><span class="c12 c23">chargeState</span><span class="c4">=&quot;6&quot; </span><span class="c12 c23">experimentalMassToCharge</span><span class="c4 c29">=&quot;752.7466713415814&quot; </span></p><p class="c3 c33 c35"><span class="c12 c23">calculatedMassToCharge</span><span class="c4 c29">=&quot;752.41371619677&quot;</span></p><p class="c3 c33 c35"><span class="c12 c23">peptide_ref</span><span class="c4">=&quot;16734068348_TAAPTVCcmLLVLGQADKVLEEVDWLIKR_16734057553_SCcmKDLQILQASK_18_1_p1&quot; </span><span class="c12 c23">rank</span><span class="c4 c29">=&quot;1&quot;</span></p><p class="c3 c33 c35"><span class="c12 c23">passThreshold</span><span class="c4">=&quot;true&quot; </span><span class="c12 c23">id</span><span class="c4">=&quot;SII_2_1&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">PeptideEvidenceRef </span><span class="c12 c23">peptideEvidence_ref</span><span class="c4">=&quot;pepevid_pep_16734057553&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002511&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;crosslink spectrum identification item&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;2&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002545&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;xi:score&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;21.55734182309742&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- crosslinked PSM level global FDR --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003337&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;crosslinked PSM-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;0.03&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- peptide pair global FDR --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002520&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;SCKDLQILQASK-TAAPTVCLLVLGQADKVLEEVDWLIKR&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peptide group ID&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003338&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peptide-pair sequence-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;0.03&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003339&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peptide-pair passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- residue pair ref value=&quot;22.b&quot; --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam &nbsp;</span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003344&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;22.b&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;Residue-pair ref&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">SpectrumIdentificationItem</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationItem </span><span class="c12 c23">chargeState</span><span class="c4">=&quot;6&quot; </span><span class="c12 c23">experimentalMassToCharge</span><span class="c4 c29">=&quot;752.7466713415814&quot; </span></p><p class="c3 c33 c35"><span class="c12 c23">calculatedMassToCharge</span><span class="c4 c29">=&quot;752.41371619677&quot;</span></p><p class="c3 c33 c35"><span class="c12 c23">peptide_ref</span><span class="c4">=&quot;16734068348_TAAPTVCcmLLVLGQADKVLEEVDWLIKR_16734057553_SCcmKDLQILQASK_18_1_p0&quot; </span><span class="c12 c23">rank</span><span class="c4 c29">=&quot;1&quot;</span></p><p class="c3 c33 c35"><span class="c12 c23">passThreshold</span><span class="c4">=&quot;true&quot; </span><span class="c12 c23">id</span><span class="c4">=&quot;SII_2_2&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">PeptideEvidenceRef </span><span class="c12 c23">peptideEvidence_ref</span><span class="c4">=&quot;pepevid_pep_16734068348&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002511&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;crosslink spectrum identification item&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;2&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002545&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;xi:score&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;21.55734182309742&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- crosslinked PSM level global FDR --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003337&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;crosslinked PSM-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;0.03&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- peptide pair global FDR --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002520&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;SCKDLQILQASK-TAAPTVCLLVLGQADKVLEEVDWLIKR&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peptide group ID&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003338&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peptide-pair sequence-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;0.03&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003339&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peptide-pair passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- residue pair ref value=&quot;22.a&quot; --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam &nbsp;</span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003344&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;22.a&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;Residue-pair ref&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">SpectrumIdentificationItem</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1000797&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;peak list scans&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;38065&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;/</span><span class="c7">SpectrumIdentificationResult</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&lt;/</span><span class="c7">SpectrumIdentificationList</span><span class="c11">&gt;</span></p><p class="c3 c15"><span class="c29 c98 c50 c52 c16"></span></p><p class="c3"><span class="c50 c16">Figure 11.</span><span class="c16">&nbsp;</span><span class="c50 c16">XML snippet including</span><span class="c50 c16">&nbsp;the encoding of scores for PSM-level matches and peptide pairs.</span><span class="c16">&nbsp;These are encoded inside &lt;SpectrumIdentificationItem&gt; elements. &ldquo;peptide-pair passes threshold&rdquo; (MS:1003339) would become relevant if there was more than one score for that peptide pair (sharing the same &ldquo;peptide group ID&rdquo;), it states whether the peptide pair passed when all scores and thresholds are considered. This is analogous to the</span><span class="c36 c16">&nbsp;passThreshold</span><span class="c16">&nbsp;attribute of &lt;SpectrumIdentificationItem&gt; elements for PSM-level </span><span class="c16">scores</span><span class="c29 c31 c16">.</span></p><a id="id.fhvkm9qtkfql"></a><h3 class="c32 c110" id="h.78go5sdh2ohx"><span class="c29 c86 c50 c52">4.3.3 Interaction Level Scores (Unique Residue-Pairs and PPI) </span></h3><p class="c14"><span class="c0">mzIdentML uses the same mechanism to encode scores for interactions at both the unique residue-pair level and protein-protein interaction level. This encoding was put forward in mzIdentML 1.2.0 and remains unchanged. Where a residue-pair level score gives the position of the crosslinked residue, a protein-protein interaction (PPI) score will instead have the value &lsquo;null&rsquo;. &nbsp;</span></p><p class="c14"><span class="c0">&nbsp; </span></p><p class="c14"><span class="c0">mzIdentML encodes these with the same mechanism it uses to address the protein inference problem, that is, within &lt;ProteinAmbiguityGroup&gt; elements. More specifically, these scores go inside &lt;ProteinDetectionHypothesis&gt; elements. All such scores must therefore meet the CV mapping rules of &lt;ProteinDetectionHypothesis&gt; elements.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>As the encoding of interaction scores uses &lt;ProteinAmbiguityGroup&gt; elements, the guidance in Section 5.2.1 (Protein grouping encoding) of the main specification also applies </span><span>here</span><span class="c0">&nbsp;and MUST be followed. This means that ambiguity about which protein a crosslinked peptide came from must be reflected in how the &lt;ProteinDetectionHypothesis&gt; elements containing the score are assigned to &lt;ProteinAmbiguityGroup&gt; elements, see Figure 12. &nbsp; &nbsp;</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14 c15"><span class="c0"></span></p><p class="c75 c115"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 396.00px;"><img alt="" src="images/image9.png" style="width: 624.00px; height: 396.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p><p class="c3"><span class="c50 c16">Figure 12. Ambiguity at PPI level. </span><span class="c16">Ambiguity regarding which protein is crosslinked (protein inference problem) MUST be reflected in how the &lt;ProteinDetectionHypothesis&gt; elements containing interaction scores are assigned to &lt;ProteinAmbiguityGroup&gt; elements, see </span><span>Section 5.2.1 (Protein grouping encoding) of the main specification.</span><span class="c16">&nbsp;Shown here with PPI level scores.</span><hr style="page-break-before:always;display:none;"></p><p class="c5 c75 c121"><span class="c12 c16">&lt;</span><span class="c7">ProteinAmbiguityGroup </span><span class="c12 c23">id</span><span class="c4">=&quot;PAG_0&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;</span><span class="c7">ProteinDetectionHypothesis </span><span class="c12 c23">dBSequence_ref</span><span class="c4">=&quot;dbseq_P02771&quot; </span><span class="c12 c23">passThreshold</span><span class="c4">=&quot;true&quot; </span><span class="c12 c23">id</span><span class="c4">=&quot;PAG_0_PDH_0&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">PeptideHypothesis </span><span class="c12 c23">peptideEvidence_ref</span><span class="c4">=&quot;pepevid_psm252637369_pep54601081&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationItemRef </span><span class="c12 c23">spectrumIdentificationItem_ref</span><span class="c4">=&quot;SII_1_1&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">PeptideHypothesis</span><span class="c11">&gt;</span></p><p class="c3"><span class="c11">&nbsp; &nbsp;... </span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002676&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;100.b:null:0.001:true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002677&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;106.b:146:0.0294:true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;/</span><span class="c7">ProteinDetectionHypothesis</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002415&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein group passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&lt;/</span><span class="c7">ProteinAmbiguityGroup</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&lt;</span><span class="c7">ProteinAmbiguityGroup </span><span class="c12 c23">id</span><span class="c4">=&quot;PAG_1&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;</span><span class="c7">ProteinDetectionHypothesis </span><span class="c12 c23">dBSequence_ref</span><span class="c4">=&quot;dbseq_P02768&quot; </span><span class="c12 c23">passThreshold</span><span class="c4">=&quot;true&quot; </span><span class="c12 c23">id</span><span class="c4">=&quot;PAG_1_PDH_0&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">PeptideHypothesis </span><span class="c12 c23">peptideEvidence_ref</span><span class="c4">=&quot;pepevid_psm252637369_pep54600650&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationItemRef </span><span class="c12 c23">spectrumIdentificationItem_ref</span><span class="c4">=&quot;SII_1_2&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">PeptideHypothesis</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">PeptideHypothesis </span><span class="c12 c23">peptideEvidence_ref</span><span class="c4">=&quot;pepevid_psm252633422_pep54604445_protP02768-A_target_52&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationItemRef </span><span class="c12 c23">spectrumIdentificationItem_ref</span><span class="c4">=&quot;SII_2_1&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">PeptideHypothesis</span><span class="c11">&gt;</span></p><p class="c3"><span class="c11">&nbsp; &nbsp;....</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002676&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;100.a:null:0.001:true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002677&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;106.a:436:0.0294:true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp;&lt;/</span><span class="c7">ProteinDetectionHypothesis</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002415&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein group passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&lt;/</span><span class="c7">ProteinAmbiguityGroup</span><span class="c12 c16">&gt;</span></p><p class="c3 c15"><span class="c29 c50 c52 c16 c98"></span></p><p class="c3"><span class="c50 c16">Figure 13. </span><span class="c50 c16">XML snippet showing the CV terms </span><span class="c50 c16">&nbsp;</span><span class="c29 c98 c50 c52 c16">&quot;protein-pair-level global FDR&quot; (MS:1002676) and &quot;residue-pair-level global FDR&quot; (MS:1002677).</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The XML snippet in Figure 13 shows the </span><span class="c16">&quot;protein-pair-level global FDR&quot; (MS:1002676) and &quot;residue-pair-level global FDR&quot; (MS:1002677) CV terms, these CV terms have the parent CV term </span><span>&ldquo;interaction score derived from crosslinking&rdquo; (MS:1002664). Where needed, </span><span>new CV terms for search specific interaction scores </span><span>should be added as children of the CV term &ldquo;interaction score derived from crosslinking&rdquo; (MS:1002664).</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">These CV terms must have a paired structure of int_ID.a|b:POS|null:SCORE_OR_VALUE:PASS_THRESHOLD</span></p><p class="c14"><span class="c10">1 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; 2 &nbsp; &nbsp; 3 &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;4</span></p><p class="c14 c15"><span class="c0"></span></p><ol class="c77 lst-kix_u5e0ktun9hhb-0 start" start="1"><li class="c6 c19 li-bullet-0"><span class="c0">The two partners in the interaction share the same integer value for ID followed by a or b. If there is ambiguity in protein identification, two different ProteinDetectionHypothesis (PDH) elements, within the same ProteinAmbiguityGroup (PAG), MAY share the same ID and suffix (a or b). A given identifier (integer and suffix) value MUST NOT be used in more than one PAG.</span></li><li class="c6 c19 li-bullet-0"><span class="c0">The export software MAY indicate the general position of the interaction (potentially taking on board multiple pairs of crosslinked peptides), with respect to the protein sequence &ndash; using a 1-based counting system. A &ldquo;null&rdquo; MAY be used if the export software does not wish to include a value.</span></li><li class="c6 c19 li-bullet-0"><span class="c0">The score or statistical value for the interaction.</span></li><li class="c6 c19 li-bullet-0"><span>&ldquo;true&rdquo; or &ldquo;false&rdquo; to indicate whether the score or value has passed a reported threshold in the file. If no threshold is defined, then PASS_THRESHOLD is always true.</span></li></ol><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>T</span><span>he first &ldquo;int_ID&rdquo; part of the value MUST be identical/shared between interaction level scores </span><span>if they refer to the same residue pair or PPI.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>The new CV term &ldquo;Residue pair ref&rdquo; (MS:1003344) SHOULD be included within &lt;SpectrumIdentificationItem&gt; elements to indicate that these are the spectra which supported the linking of a specific residue pair. The value of the new &ldquo;Residue pair ref&rdquo; CV term is the &ldquo;</span><span class="c36">int_ID.a|b</span><span>&rdquo; part of the values,</span><span>&nbsp;s</span><span>ee Figure </span><span class="c0">11. More than one &ldquo;Residue pair ref&rdquo; (MS:1003344) CV term (with different values) can be included in a single &lt;SpectrumIdentificationItem&gt; element if it has been taken as evidence for more than one linked residue pair. &nbsp; </span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>It is not a requirement that the &lt;SpectrumIdentificationItem&gt; elements </span><span>containing</span><span>&nbsp;&ldquo;Residue pair ref&rdquo; (MS:1003344) place the linkage sites at the same position in the peptide as the residue-pair they are claiming to support. </span><span>Hence, analyses which utilise link site reassignment can be encoded in mzIdentML. (Some analyses may look at a collection of spectra to reach a conclusion about where the linkage site was, therefore some identifications may end up supporting a residue-pair that places the linkage site at a different position from where they themselves did).</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">References to supporting &lt;SpectrumIdentificationItem&gt; elements for PPIs are given by the &lt;SpectrumIdentificationItemRef&gt; elements inside &lt;PeptideHypothesis&gt; elements in &lt;ProteinDectectionHypothesis&gt;. This performs the equivalent role as the &ldquo;Residue pair ref&rdquo; (MS:1003344) CV term does for residue-pair interactions.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">See Figure 14 for an example of encoding residue-pair and PPI level scores.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>Positional ambiguity of the residues linked can be encoded by repeating the score CV terms, keeping the same identifier (integer and suffix) , for each of the positional alternatives, see Figure 15.</span><span>&nbsp;T</span><span class="c0">his may be due to ambiguity regarding the position of the peptide in the protein sequences (protein inference problem) or ambiguity regarding the linkage site in the peptide.</span></p><p class="c14 c15"><span class="c11"></span></p><p class="c3"><span class="c12 c16">&lt;</span><span class="c7">ProteinDetectionList </span><span class="c12 c23">id</span><span class="c4">=&quot;PDL_1&quot; </span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;</span><span class="c7">ProteinAmbiguityGroup </span><span class="c12 c23">id</span><span class="c4">=&quot;PAG_0&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">ProteinDetectionHypothesis </span><span class="c12 c23">dBSequence_ref</span><span class="c4">=&quot;dbseq_ggFANCI_target&quot; </span><span class="c12 c23">passThreshold</span><span class="c4">=&quot;true&quot; </span><span class="c12 c23">id</span><span class="c4">=&quot;PAG_0_PDH_0&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">PeptideHypothesis </span><span class="c12 c23">peptideEvidence_ref</span><span class="c4">=&quot;pepevid_pep_16734063165&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationItemRef </span><span class="c12 c23">spectrumIdentificationItem_ref</span><span class="c4">=&quot;SII_1_1&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">PeptideHypothesis</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">PeptideHypothesis </span><span class="c12 c23">peptideEvidence_ref</span><span class="c4">=&quot;pepevid_pep_16734057553&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationItemRef </span><span class="c12 c23">spectrumIdentificationItem_ref</span><span class="c4">=&quot;SII_2_1&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">PeptideHypothesis</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">PeptideHypothesis </span><span class="c12 c23">peptideEvidence_ref</span><span class="c4">=&quot;pepevid_pep_16734068348&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationItemRef </span><span class="c12 c23">spectrumIdentificationItem_ref</span><span class="c4">=&quot;SII_2_2&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">PeptideHypothesis</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002403&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;group representative&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1001593&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;group member with undefined relationship OR ortholog protein&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- forms a protein heteromeric PPI with its partner 10.a in PAG_1_PDH_0 --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002676&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;10.b:null:0.059:false&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003341&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein-protein interaction passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;10:false&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- forms a self PPI with its partner 20.b in PAG_0_PDH_0 --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002676&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;20.a:null:0.030:true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002676&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;20.b:null:0.030:true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003341&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein-protein interaction passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;20:true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- forms a protein heteromeric crosslink with its partner 11.a in PAG_1_PDH_0 --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002677&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;11.b:697:0.06:false&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003340&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;residue-pair passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;11:false&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c58 c36 c45">&lt;!-- forms a self crosslink with its partner 22.b in PAG_0_PDH_0 --&gt;</span></p><p class="c3"><span class="c12 c36 c59">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c16">&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002677&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;22.a:1095:0.01:true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002677&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;22.b:339:0.01:true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003340&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;residue-pair passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;22:true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">ProteinDetectionHypothesis</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002415&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein group passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;/</span><span class="c7">ProteinAmbiguityGroup</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;</span><span class="c7">ProteinAmbiguityGroup </span><span class="c12 c23">id</span><span class="c4">=&quot;PAG_1&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">ProteinDetectionHypothesis </span><span class="c12 c23">dBSequence_ref</span><span class="c4">=&quot;dbseq_ggFANCD2_target&quot; </span><span class="c12 c23">passThreshold</span><span class="c4">=&quot;true&quot; </span><span class="c12 c23">id</span><span class="c4">=&quot;PAG_1_PDH_0&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">PeptideHypothesis </span><span class="c12 c23">peptideEvidence_ref</span><span class="c4">=&quot;pepevid_pep_16734061838&quot;</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">SpectrumIdentificationItemRef </span><span class="c12 c23">spectrumIdentificationItem_ref</span><span class="c4">=&quot;SII_1_2&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">PeptideHypothesis</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002403&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;group representative&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1001593&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;group member with undefined relationship OR ortholog protein&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002676&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;10.a:null:059:false&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003341&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein-protein interaction passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;10:false&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002677&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;11.a:36:0.06:false&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1003340&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;residue-pair passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;11:false&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;/</span><span class="c7">ProteinDetectionHypothesis</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002415&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;protein group passes threshold&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;true&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;/</span><span class="c7">ProteinAmbiguityGroup</span><span class="c11">&gt;</span></p><p class="c3"><span class="c12 c16">&nbsp; &nbsp;&lt;</span><span class="c7">cvParam </span><span class="c12 c23">cvRef</span><span class="c4">=&quot;PSI-MS&quot; </span><span class="c12 c23">accession</span><span class="c4">=&quot;MS:1002404&quot; </span><span class="c12 c23">name</span><span class="c4">=&quot;count of identified proteins&quot; </span><span class="c12 c23">value</span><span class="c4">=&quot;2&quot;</span><span class="c11">/&gt;</span></p><p class="c3"><span class="c12 c16">&lt;/</span><span class="c7">ProteinDetectionList</span><span class="c11">&gt;</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c3"><span class="c50 c16">Figure 14. </span><span class="c50 c16">XML snippet including the encoding of </span><span class="c50 c16">scores for residue-pairs and PPIs.</span><span class="c16">&nbsp;These are encoded inside &lt;ProteinDetectionHypothesis&gt; elements. The CV terms &quot;residue-pair passes threshold&quot; (MS:1003340) and &ldquo;protein-pair passes threshold&rdquo; (MS:1003341) would become relevant if there was more than one score for those residue or protein pairs (sharing the same integer id part of their value). These are analogous to the</span><span class="c36 c16">&nbsp;passThreshold </span><span class="c29 c31 c16">attribute of &lt;SpectrumIdentificationItem&gt; elements.</span></p><p class="c3 c15"><span class="c29 c31 c16"></span></p><hr style="page-break-before:always;display:none;"><p class="c3 c15"><span class="c29 c16 c31"></span></p><p class="c3"><span class="c29 c31 c16">(i)</span></p><p class="c3 c15"><span class="c9"></span></p><p class="c3"><span class="c12">&lt;</span><span class="c12">ProteinAmbiguityGroup </span><span class="c12">id</span><span class="c12">=&quot;PAG_0&quot;</span><span class="c9">&gt;</span></p><p class="c3"><span class="c9">&nbsp; &nbsp;&lt;!-- example of both peptide ambiguity (classical protein inference) and site ambiguity with in a peptide --&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp;&lt;</span><span class="c12">ProteinDetectionHypothesis </span><span class="c12">dBSequence_ref</span><span class="c12">=&quot;dbseq_A_target&quot; </span><span class="c12">passThreshold</span><span class="c12">=&quot;true&quot; </span><span class="c12">id</span><span class="c12">=&quot;PAG_0_PDH_0&quot;</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;...</span></p><p class="c3"><span class="c12 c36">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c9 c24">&lt;!-- each possible linksite in the originating peptide is referenced here as a possible residue pair--&gt;</span></p><p class="c3"><span class="c12 c36 c24">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c9 c24">&lt;!-- the first two have the same score as there is no fragmentation distinguishing the two neighbouring residues--&gt;</span></p><p class="c3"><span class="c12 c36 c24">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c24">&lt;</span><span class="c12 c24">cvParam </span><span class="c12 c24">cvRef</span><span class="c12 c24">=&quot;PSI-MS&quot; </span><span class="c12 c24">accession</span><span class="c12 c24">=&quot;MS:1002677&quot; </span><span class="c12 c24">name</span><span class="c12 c24">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c24">value</span><span class="c12 c24">=&quot;22.a:1095:0.01:true&quot;</span><span class="c9 c24">/&gt;</span></p><p class="c3"><span class="c12 c36 c24">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c9 c24">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002677&quot; name=&quot;residue-pair-level global FDR&quot; value=&quot;22.a:1096:0.01:true&quot;/&gt;</span></p><p class="c3"><span class="c12 c36 c24">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c9 c24">&lt;!-- the third residue would be a possible linksite, but there is some fragments speaking in favour of the first two, therefore this one has a lower score and hence a worse FDR--&gt;</span></p><p class="c3"><span class="c12 c24">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12 c24">cvParam </span><span class="c12 c24">cvRef</span><span class="c12 c24">=&quot;PSI-MS&quot; </span><span class="c12 c24">accession</span><span class="c12 c24">=&quot;MS:1002677&quot; </span><span class="c12 c24">name</span><span class="c12 c24">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c24">value</span><span class="c12 c24">=&quot;22.a:1091:0.09:false&quot;</span><span class="c9 c24">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">cvParam </span><span class="c12">cvRef</span><span class="c12">=&quot;PSI-MS&quot; </span><span class="c12">accession</span><span class="c12">=&quot;MS:1003341&quot; </span><span class="c12">name</span><span class="c12">=&quot;residue-pair passes threshold&quot; </span><span class="c12">value</span><span class="c12">=&quot;22:true&quot;</span><span class="c9">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp;&lt;/</span><span class="c12">ProteinDetectionHypothesis</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp;&lt;</span><span class="c12">ProteinDetectionHypothesis </span><span class="c12">dBSequence_ref</span><span class="c12">=&quot;dbseq_B_target&quot; </span><span class="c12">passThreshold</span><span class="c12">=&quot;true&quot; </span><span class="c12">id</span><span class="c12">=&quot;PAG_0_PDH_1&quot;</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;...</span></p><p class="c3"><span class="c12 c36 c24">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c9 c24">&lt;!-- (all) peptide(s) for site a could also come from a different protein--&gt;</span></p><p class="c3"><span class="c12 c36 c24">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c24">&lt;</span><span class="c12 c24">cvParam </span><span class="c12 c24">cvRef</span><span class="c12 c24">=&quot;PSI-MS&quot; </span><span class="c12 c24">accession</span><span class="c12 c24">=&quot;MS:1002677&quot; </span><span class="c12 c24">name</span><span class="c12 c24">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c24">value</span><span class="c12 c24">=&quot;22.a:295:0.01:true&quot;</span><span class="c9 c24">/&gt;</span></p><p class="c3"><span class="c12 c36 c24">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c9 c24">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002677&quot; name=&quot;residue-pair-level global FDR&quot; value=&quot;22.a:296:0.01:true&quot;/&gt;</span></p><p class="c3"><span class="c12 c24">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12 c24">cvParam </span><span class="c12 c24">cvRef</span><span class="c12 c24">=&quot;PSI-MS&quot; </span><span class="c12 c24">accession</span><span class="c12 c24">=&quot;MS:1002677&quot; </span><span class="c12 c24">name</span><span class="c12 c24">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c24">value</span><span class="c12 c24">=&quot;22.a:291:0.09:false&quot;</span><span class="c9 c24">/&gt;</span></p><p class="c3 c15"><span class="c9"></span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">cvParam </span><span class="c12">cvRef</span><span class="c12">=&quot;PSI-MS&quot; </span><span class="c12">accession</span><span class="c12">=&quot;MS:1003341&quot; </span><span class="c12">name</span><span class="c12">=&quot;residue-pair passes threshold&quot; </span><span class="c12">value</span><span class="c12">=&quot;22:true&quot;</span><span class="c9">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp;&lt;/</span><span class="c12">ProteinDetectionHypothesis</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&lt;/</span><span class="c12">ProteinAmbiguityGroup</span><span class="c9">&gt;</span></p><p class="c3"><span class="c9">&lt;ProteinAmbiguityGroup id=&quot;PAG_1&quot;&gt;</span></p><p class="c3"><span class="c9">&nbsp; &nbsp;&lt;ProteinDetectionHypothesis dBSequence_ref=&quot;dbseq_C_target&quot; passThreshold=&quot;true&quot; id=&quot;PAG_1_PDH_0&quot;&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;...</span></p><p class="c3"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002677&quot; name=&quot;residue-pair-level global FDR&quot; value=&quot;22.b:339:0.01:true&quot;/&gt;</span></p><p class="c3"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003341&quot; name=&quot;residue-pair passes threshold&quot; value=&quot;22:true&quot;/&gt;</span></p><p class="c3"><span class="c9">&nbsp; &nbsp;&lt;/ProteinDetectionHypothesis&gt;</span></p><p class="c3"><span class="c9">&lt;/ProteinAmbiguityGroup&gt;</span></p><p class="c3 c15"><span class="c0"></span></p><p class="c3"><span>(ii)</span></p><p class="c3 c15"><span class="c9"></span></p><p class="c3"><span class="c12">&lt;</span><span class="c12">ProteinAmbiguityGroup </span><span class="c12">id</span><span class="c12">=&quot;PAG_0&quot;</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp;&lt;</span><span class="c12">ProteinDetectionHypothesis </span><span class="c12">dBSequence_ref</span><span class="c12">=&quot;dbseq_B_target&quot; </span><span class="c12">passThreshold</span><span class="c12">=&quot;true&quot; </span><span class="c12">id</span><span class="c12">=&quot;PAG_0_PDH_1&quot;</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;...</span></p><p class="c3"><span class="c12 c36 c24">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c9 c24">&lt;!-- peptide has two possible link sites and is present in two places in protein B--&gt;</span></p><p class="c3"><span class="c12 c36 c24">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c12 c24">&lt;</span><span class="c12 c24">cvParam </span><span class="c12 c24">cvRef</span><span class="c12 c24">=&quot;PSI-MS&quot; </span><span class="c12 c24">accession</span><span class="c12 c24">=&quot;MS:1002677&quot; </span><span class="c12 c24">name</span><span class="c12 c24">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c24">value</span><span class="c12 c24">=&quot;23.a:1095:0.01:true&quot;</span><span class="c9 c24">/&gt;</span></p><p class="c3"><span class="c12 c24">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12 c24">cvParam </span><span class="c12 c24">cvRef</span><span class="c12 c24">=&quot;PSI-MS&quot; </span><span class="c12 c24">accession</span><span class="c12 c24">=&quot;MS:1002677&quot; </span><span class="c12 c24">name</span><span class="c12 c24">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c24">value</span><span class="c12 c24">=&quot;23.a:1091:0.09:false&quot;</span><span class="c9 c24">/&gt;</span></p><p class="c3"><span class="c12 c24">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12 c24">cvParam </span><span class="c12 c24">cvRef</span><span class="c12 c24">=&quot;PSI-MS&quot; </span><span class="c12 c24">accession</span><span class="c12 c24">=&quot;MS:1002677&quot; </span><span class="c12 c24">name</span><span class="c12 c24">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c24">value</span><span class="c12 c24">=&quot;23.a:295:0.01:true&quot;</span><span class="c9 c24">/&gt;</span></p><p class="c3"><span class="c12 c24">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12 c24">cvParam </span><span class="c12 c24">cvRef</span><span class="c12 c24">=&quot;PSI-MS&quot; </span><span class="c12 c24">accession</span><span class="c12 c24">=&quot;MS:1002677&quot; </span><span class="c12 c24">name</span><span class="c12 c24">=&quot;residue-pair-level global FDR&quot; </span><span class="c12 c24">value</span><span class="c12 c24">=&quot;23.a:291:0.09:false&quot;</span><span class="c9 c24">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">cvParam </span><span class="c12">cvRef</span><span class="c12">=&quot;PSI-MS&quot; </span><span class="c12">accession</span><span class="c12">=&quot;MS:1002677&quot; </span><span class="c12">name</span><span class="c12">=&quot;residue-pair-level global FDR&quot; </span><span class="c12">value</span><span class="c12">=&quot;23.b:339:0.01:true&quot;</span><span class="c9">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;&lt;</span><span class="c12">cvParam </span><span class="c12">cvRef</span><span class="c12">=&quot;PSI-MS&quot; </span><span class="c12">accession</span><span class="c12">=&quot;MS:1003341&quot; </span><span class="c12">name</span><span class="c12">=&quot;residue-pair passes threshold&quot; </span><span class="c12">value</span><span class="c12">=&quot;23:true&quot;</span><span class="c9">/&gt;</span></p><p class="c3"><span class="c12">&nbsp; &nbsp;&lt;/</span><span class="c12">ProteinDetectionHypothesis</span><span class="c9">&gt;</span></p><p class="c3"><span class="c12">&lt;/</span><span class="c12">ProteinAmbiguityGroup</span><span class="c9">&gt;</span></p><p class="c3 c15"><span class="c11"></span></p><p class="c3 c15"><span class="c29 c98 c50 c52 c16"></span></p><p class="c3"><span class="c50 c16">Figure 15. </span><span class="c50 c16">XML snippet including the e</span><span class="c50 c16">ncoding</span><span class="c50 c16">&nbsp;</span><span class="c50 c16">of </span><span class="c50 c16">positional ambiguity of residue pairs.</span><span class="c16">&nbsp;In panel</span><span class="c16">&nbsp;(i), residue-pair 22 is a protein heteromeric crosslink where the &ldquo;a&rdquo; end of the crosslink is ambiguous between two proteins and there are three possible positions of the crosslink in peptide &ldquo;a&rdquo;. In panel (ii), residue pair 23 is a self link but there is ambiguity about where peptide &ldquo;a&rdquo; came from within that protein (two possible positions) and two possible link sites in peptide &ldquo;a&rdquo;, giving a total of four possible residues. </span></p><h2 class="c17 c113" id="h.bjlkvef0tyec"><span class="c47 c29"></span></h2><h2 class="c17" id="h.7if82p50b87k"><span class="c47 c29">4.4 FDR Specific Comments</span></h2><p class="c14"><span>Section 7.5 of the main mzIdentML 1.3.0 specification document (&lsquo;Using decoy databases to set different thresholds of false discovery rate&rsquo;) states that:</span></p><p class="c14 c15"><span class="c0"></span></p><ul class="c77 lst-kix_vk5294fzi1ce-0 start"><li class="c95 c19 c49 li-bullet-0"><span class="c37 c62 c36 c52">A &lt;SpectrumIdentificationItem&gt; can be marked as matching a decoy peptide using the isDecoy attribute of the referenced &lt;PeptideEvidence&gt; element, thus allowing the false discovery rate to be calculated across an entire file.</span></li><li class="c95 c19 c49 li-bullet-0"><span class="c36">Implementers of the format SHOULD report the peptide identifications</span><span>&nbsp;[including those of decoy peptides]</span><span class="c37 c62 c36 c52">&nbsp;that pass the threshold they wish to communicate to a consumer of the data.</span></li><li class="c95 c19 c49 li-bullet-0"><span class="c37 c62 c36 c52">It is not guaranteed that a consumer of an mzIdentML file will be able to calculate other results, or global false discovery rates, using different thresholds from the reported information, although in some circumstances they may be able to, for example, if a user reports the complete output of a search against a target and decoy search.</span></li></ul><p class="c95"><span class="c0">CV terms exist for FDR scores at each level of consolidation:</span></p><ul class="c77 lst-kix_kh84qhxx6ky4-0 start"><li class="c6 c19 li-bullet-0"><span>&quot;crosslinked PSM-level global FDR&quot; (MS:1003337)</span></li><li class="c6 c19 li-bullet-0"><span class="c0">&ldquo;peptide-pair sequence-level global FDR&rdquo; (MS:1003339) </span></li><li class="c6 c19 li-bullet-0"><span>&ldquo;residue-pair-level global FDR&rdquo; (</span><span class="c101">MS:1002677)</span><span class="c0">&nbsp;</span></li><li class="c6 c19 li-bullet-0"><span>&ldquo;protein-pair-level global FDR&rdquo; (</span><span class="c101">MS:1002676)</span></li></ul><p class="c32"><span>A new CV term &ldquo;FDR applied separately to self crosslinks and protein heteromeric crosslinks&rdquo; </span><span>(</span><span>MS:1003343</span><span>)</span><span>&nbsp;has been introduced to encode whether self crosslinks (</span><span>crosslinks between peptides within one protein sequence</span><span>) and protein heteromeric crosslinks (</span><span>crosslinks between distinct protein sequences</span><span>) were grouped separately for FDR analysis </span><span class="c37"><a class="c18" href="https://www.google.com/url?q=https://paperpile.com/c/YrD2gH/kAbLF&amp;sa=D&amp;source=editors&amp;ust=1719221372905383&amp;usg=AOvVaw10iyu-zIHYOxakKzVn5WFm">[5]</a></span><span>. This CV term goes within the &lt;AdditionalSearchParameters&gt; element (see </span><span>Figure 3</span><span class="c0">). </span></p><p class="c32"><span>The value of &ldquo;</span><span class="c44">FDR applied separately to self crosslinks and protein heteromeric crosslinks</span><span class="c0">&rdquo; (MS:1003343) is a boolean, stating whether or not this happened. This CV term SHOULD be supplied. If it is omitted then it is unspecified whether self and heteromeric links were grouped separately for analysis (there is no default value).</span></p><hr style="page-break-before:always;display:none;"><h1 class="c41 c81" id="h.7f96p1v8jh6n"><span class="c29 c43 c48"></span></h1><h1 class="c41" id="h.s698zhk8zhgu"><span class="c43">5. </span><span class="c43">Unsupported Use Cases and Future </span><span class="c43">Directions</span></h1><p class="c14"><span class="c0">The two unsupported crosslinking product types shown in Figure 1 are: crosslinkers with more than two reactive groups and higher order crosslinks (arbitrarily many peptides identified with many crosslinks between them).</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>Crosslinkers with more than two reactive groups </span><span class="c37"><a class="c18" href="https://www.google.com/url?q=https://paperpile.com/c/YrD2gH/8J3I&amp;sa=D&amp;source=editors&amp;ust=1719221372906667&amp;usg=AOvVaw0hsbIraRU8aApdxkx0dXLx">[7]</a></span><span>&nbsp;</span><span class="c0">cannot be represented using the current model for two reasons. First, the donor/acceptor mechanism for crosslinked Peptides in &lt;SequenceCollection&gt; elements (&sect;3.2) restricts the number of reactive groups to two. Second, there can be at most two crosslinked &lt;SpectrumidentificationItem&gt; elements, each of which references an identified peptide within a &lt;SpectrumIdentificationResult&gt; (&sect;3.3).</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>In the case of higher order crosslinks, the specification already allows the encoding of this in the &lt;Peptide&gt; elements within &lt;SequenceCollection&gt; (or rather nothing forbids it), see &sect;3.2. It is only the restriction of there being </span><span>at most two</span><span class="c0">&nbsp;crosslinked &lt;SpectrumIdentificationItem&gt; elements that share the same value within a &lt;SpectrumIdentificationResult&gt; that prevents the encoding of higher order crosslinks.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>It would be possible to support higher order crosslinks by allowing </span><span class="c36">n</span><span>&nbsp;crosslinked &lt;SpectrumidentificationItem&gt; elements within a &lt;SpectrumIdentificationResult&gt;. This would pose some problems for the validation of the documents. </span><span>These would not be insurmountable</span><span>&nbsp;because the number of peptides that are crosslinked could be derived from &lt;Peptide&gt; elements in &lt;SequenceCollection&gt;. </span><span>However, this would make validation significantly more complex, to alleviate this an additional CV term in peptide that links peptides as part of a &quot;crosslink-group&quot; independent of the crosslinker could be introduced.</span><span><br><br></span><span>In the case of</span><span>&nbsp;</span><span>crosslinkers with more than two reactive groups</span><span>&nbsp;and the identification of higher order crosslinks, there was no demand for supporting these use cases at this point in time and so, for the sake of simplicity and minimal changes, they are still not supported.</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span>This remains an open question for future versions of the specification. </span><span>There are other use cases in which </span><span class="c36">n</span><span>&nbsp;&lt;SpectrumIdentificationItem&gt; elements need to be associated</span><span>. </span><span>Characterisation of antibodies</span><span>&nbsp;or other multi-chain proteins that contain complex patterns of disulfide bonds (representing endogenous crosslinks) by</span><span>&nbsp;top-down mass spectrometry would be</span><span>&nbsp;an example of this. </span><span class="c0">&nbsp; &nbsp;</span></p><p class="c14 c15"><span class="c0"></span></p><hr style="page-break-before:always;display:none;"><p class="c14 c15"><span class="c0"></span></p><h1 class="c41" id="h.t2z1zrknogih"><span class="c43">6. </span><span class="c43">Appendix I</span><span class="c43">. </span><span class="c43">Comparison of rules for </span><span class="c43">&ldquo;crosslink donor&rdquo; and &ldquo;crosslink acceptor&rdquo;</span><span class="c43">&nbsp;depending on </span><span class="c43">context</span></h1><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">The CV terms &ldquo;crosslink donor&rdquo; (MS:1002509) and &ldquo;crosslink acceptor&rdquo; (MS:1002510) are used in two different contexts:</span></p><ul class="c77 lst-kix_dqnkgejf3n11-0"><li class="c6 c19 li-bullet-0"><span class="c0">/MzIdentML/AnalysisProtocolCollection/SpectrumIdentificationProtocol/ ModificationParams/SearchModification &ndash; encoding the modifications searched for (including specificity, see &sect;3.1.2); </span></li><li class="c6 c19 li-bullet-0"><span class="c0">/MzIdentML/SequenceCollection/Peptide/Modification - encoding the modifications of crosslinked peptides (&sect;3.2).</span></li></ul><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">Table 1 summarises the commonalities and differences between the rules governing their use in these two contexts.</span></p><p class="c14 c15"><span class="c0"></span></p><a id="t.39e6f444da719ce7634347a219edf70d44169ce1"></a><a id="t.1"></a><table class="c91"><tr class="c83"><td class="c80" colspan="1" rowspan="1"><p class="c5"><span class="c10">Element &lt;SearchModification&gt; (see &sect;3.1.2)</span></p></td><td class="c80" colspan="1" rowspan="1"><p class="c5"><span class="c50">Element &lt;Peptide&gt;&lt;Modification&gt;</span><span class="c10">&nbsp;(see &sect;3.2)</span></p></td></tr><tr class="c83"><td class="c80" colspan="1" rowspan="1"><p class="c5"><span class="c50">Two or more</span><span>&nbsp;&lt;SearchModification&gt; elements are needed to describe the specificity of a single crosslinker with two reactive groups. All of the donor and acceptor CV terms contained in these </span><span class="c50">MUST</span><span class="c0">&nbsp;share a unique identifier in their value slot.</span></p></td><td class="c80" colspan="1" rowspan="1"><p class="c5"><span>A unique identifier linking these </span><span class="c50">two</span><span>&nbsp;Modification elements together </span><span class="c50">MUST</span><span class="c0">&nbsp;be in the value slot. (Thereby excluding the representation of trimeric crosslinkers.)</span></p></td></tr><tr class="c83"><td class="c80" colspan="1" rowspan="1"><p class="c5"><span class="c0">The choice of which end is the &lsquo;donor&rsquo; and which end is the &lsquo;acceptor&rsquo; is arbitrary.</span></p></td><td class="c80" colspan="1" rowspan="1"><p class="c14"><span>If the CV term &ldquo;search modification id ref&rdquo; (MS:1003393) is being used then the crosslink donor </span><span class="c50">MUST</span><span>&nbsp;be chosen to match the end marked as the donor in the corresponding &lt;SearchModification&gt; elements, see &sect;3.2.2. If that CV term is not used, or if the preceding rule does not unambiguously define which end to mark as donor (e.g. because the crosslinker is symmetrical) then the export software </span><span class="c50">SHOULD</span><span class="c0">&nbsp;use the following rules to choose the crosslink donor as the: longer peptide, then higher peptide neutral mass, then alphabetical order.</span></p></td></tr><tr class="c79"><td class="c112" colspan="2" rowspan="1"><p class="c5"><span>The element(s) containing the crosslink donor CV term </span><span class="c50">MUST</span><span class="c0">&nbsp;have their mass delta attribute = the mass gained from the crosslink reagent.</span></p><p class="c5 c15"><span class="c0"></span></p></td></tr><tr class="c79"><td class="c112" colspan="2" rowspan="1"><p class="c5"><span>The element(s) containing the crosslink acceptor CV term </span><span class="c50">MUST</span><span class="c0">&nbsp;have their mass delta attribute = 0. </span></p></td></tr><tr class="c79"><td class="c80" colspan="1" rowspan="2"><p class="c5"><span class="c50">Both</span><span>&nbsp;crosslink donor and crosslink acceptor </span><span class="c50">MUST</span><span class="c0">&nbsp;have a suitably sourced cvParam for the crosslink.</span></p></td><td class="c80" colspan="1" rowspan="1"><p class="c5"><span>The crosslink donor peptide&rsquo;s Modification element </span><span class="c50">MUST</span><span class="c0">&nbsp;have a suitably sourced cvParam for the crosslink.</span></p></td></tr><tr class="c79"><td class="c80" colspan="1" rowspan="1"><p class="c5"><span>The crosslink acceptor peptide&rsquo;s Modification element </span><span class="c50">MUST NOT</span><span class="c0">&nbsp;have a cvParam for the reagent.</span></p></td></tr></table><p class="c14 c15"><span class="c10"></span></p><p class="c14"><span class="c50">Table 1. </span><span class="c0">The rules governing the use of &ldquo;crosslink donor&rdquo; (MS:1002509) and &ldquo;crosslink acceptor&rdquo; (MS:1002510) differ depending on the context.</span></p><p class="c72 c15"><span class="c47 c29"></span></p><hr style="page-break-before:always;display:none;"><p class="c14 c15"><span class="c0"></span></p><h1 class="c41" id="h.8qg7o6e2yw4k"><span class="c43">7. Appendix II. Example encodings of crosslinker reagents as &lt;SearchModification&gt; elements</span></h1><h2 class="c17" id="h.z7iqem5xmct9"><span class="c47 c29">7.1 Bis(sulfosuccinimidyl) suberate (BS3)</span></h2><p class="c14"><span class="c9">&lt;SpectrumIdentificationProtocol&gt;</span></p><p class="c14 c35"><span class="c9">...</span></p><p class="c14 c35"><span class="c9">&lt;ModificationParams&gt;</span></p><p class="c6 c35"><span class="c9">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;138.06808&quot; residues=&quot;S T Y K&quot;&gt;</span></p><p class="c6 c35"><span class="c9">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;BS3_donor&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;XLMOD&quot; accession=&quot;XLMOD:02000&quot; name=&quot;BS3&quot;/&gt;</span></p><p class="c6"><span class="c9">&nbsp; &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002509&quot; name=&quot;crosslink donor&quot; </span></p><p class="c14 c38 c35"><span class="c9">value=&quot;0&quot;/&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c9">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;138.06808&quot; residues=&quot;.&quot;&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c97"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002057&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein N-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;BS3_donor_n_term&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;XLMOD&quot; accession=&quot;XLMOD:02000&quot; name=&quot;BS3&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink donor&quot;</span></p><p class="c14 c38 c35"><span class="c9">value=&quot;0&quot; /&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c9">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;0.0&quot; residues=&quot;S T Y K&quot;&gt;</span></p><p class="c6 c35"><span class="c9">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;BS3_acceptor&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&nbsp; &lt;cvParam cvRef=&quot;XLMOD&quot; accession=&quot;XLMOD:02000&quot; name=&quot;BS3&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&nbsp; &lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink acceptor&quot; </span></p><p class="c14 c38 c35"><span class="c9">value=&quot;0&quot;/&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c8"><span class="c9"></span></p><p class="c14 c33"><span class="c9">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;0.0&quot; residues=&quot;.&quot;&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c38 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002058&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein N-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;BS3_acceptor_n_term&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;XLMOD&quot; accession=&quot;XLMOD:02000&quot; name=&quot;BS3&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink acceptor&quot;</span></p><p class="c14 c38 c35"><span class="c9">value=&quot;0&quot; /&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c8"><span class="c9"></span></p><p class="c14 c35"><span class="c9">&lt;/ModificationParams&gt;</span></p><p class="c14 c35"><span class="c9">...</span></p><p class="c14"><span class="c12">&lt;/SpectrumIdentificationProtocol&gt;</span><hr style="page-break-before:always;display:none;"></p><h2 class="c17" id="h.hfq5x99v2tmv"><span class="c88 c50">7.2 1-ethyl-3-(3-dimethylaminopropyl)carbodiimide hydrochloride (EDC)</span></h2><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c9">&lt;SpectrumIdentificationProtocol&gt;</span></p><p class="c14 c35"><span class="c9">...</span></p><p class="c14 c35"><span class="c9">&lt;ModificationParams&gt;</span></p><p class="c14 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification</span><span class="c12">&nbsp;fixedMod=&quot;false&quot; massDelta=&quot;-18.010565&quot; </span><span class="c12">residues=&quot;K&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=</span><span class="c12">&quot;EDC_donor&quot;</span><span class="c9">/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:2018&quot; name=&quot;Xlink:EDC&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c33"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002509&quot; name=&quot;crosslink donor&quot; </span></p><p class="c14 c38 c35"><span class="c9">value=&quot;1&quot; /&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;-18.010565&quot; </span><span class="c12">residues=&quot;</span><span class="c71 c50">.</span><span class="c12">&quot;</span><span class="c9">&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c97"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002057&quot; </span></p><p class="c14 c35 c97"><span class="c12">name=&quot;modification specificity protein</span><span class="c71 c50">&nbsp;</span><span class="c9">N-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;EDC_donor_n_term&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:2018&quot; name=&quot;Xlink:EDC&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink donor&quot;</span></p><p class="c14 c38 c35"><span class="c9">value=&quot;1&quot; /&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c33"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;0.0&quot;</span><span class="c12">&nbsp;residues=&quot;D E&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;EDC_acceptor&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:2018&quot; name=&quot;Xlink:EDC&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink acceptor&quot;</span></p><p class="c14 c38 c35"><span class="c9">value=&quot;1&quot; /&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c33"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;0.0&quot; </span><span class="c12">residues=&quot;</span><span class="c71 c50">.</span><span class="c12">&quot;</span><span class="c9">&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c38 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002058&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein C-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;EDC_acceptor_c_term&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:2018&quot; name=&quot;Xlink:EDC&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink acceptor&quot;</span></p><p class="c14 c38 c35"><span class="c12">value=&quot;1&quot; /&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c15"><span class="c9"></span></p><p class="c14 c35"><span class="c9">&lt;/ModificationParams&gt;</span></p><p class="c14 c35"><span class="c9">...</span></p><p class="c14"><span class="c9">&lt;/SpectrumIdentificationProtocol&gt;</span></p><p class="c14 c40 c15"><span class="c29 c53 c82"></span></p><hr style="page-break-before:always;display:none;"><h2 class="c17 c113" id="h.9jcrokbqta7q"><span class="c47 c29"></span></h2><h2 class="c17" id="h.utvdaw73ics"><span class="c88 c50">7.3 &nbsp;(NHS-Diazirine) succinimidyl 4,4&#39;-azipentanoate (SDA)</span></h2><p class="c14"><span class="c9">&lt;SpectrumIdentificationProtocol&gt;</span></p><p class="c14 c35"><span class="c9">...</span></p><p class="c14 c35"><span class="c9">&lt;ModificationParams&gt;</span></p><p class="c14 c35"><span class="c9">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;100.05243&quot; </span><span class="c12">residues=&quot;K S T Y&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">SDA_monolink_W</span><span class="c9">&quot;/&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:2000&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:SDA&quot;/&gt;</span></p><p class="c13"><span class="c12">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c9">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; </span></p><p class="c13 c38 c35"><span class="c12">value=&quot;W&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;100.05243&quot; </span><span class="c12">residues=&quot;.&quot;</span><span class="c9">&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c97"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002057&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein N-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">SDA_monolink_W_n_term</span><span class="c9">&quot;/&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:2000&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:SDA&quot;/&gt;</span></p><p class="c13"><span class="c12">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c9">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; </span></p><p class="c13 c38 c35"><span class="c12">value=&quot;W&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;82.041865&quot; </span><span class="c12">residues=&quot;K S T Y&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">SDA_crosslink_donor</span><span class="c9">&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:2000&quot; name=&quot;Xlink:DSSO&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c33"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002509&quot; name=&quot;crosslink donor&quot; </span></p><p class="c14 c38 c35"><span class="c12">value=&quot;</span><span class="c71 c50">2</span><span class="c9">&quot; /&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; value=&quot;S:82.041865:O&quot; </span></p><p class="c13 c38 c35"><span class="c12">cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;82.041865&quot; </span><span class="c12">residues=&quot;.&quot;</span><span class="c9">&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c97"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002057&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein N-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">SDA_crosslink_donor_n_term</span><span class="c9">&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; name=&quot;Xlink:DSSO&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink donor&quot;</span></p><p class="c14 c38 c35"><span class="c12">value=&quot;</span><span class="c71 c50">2</span><span class="c9">&quot; /&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; value=&quot;S:82.041865:O&quot; </span></p><p class="c13 c38 c35"><span class="c12">cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c33"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;0.0&quot;</span><span class="c12">&nbsp;residues=&quot;.&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">SDA_crosslink_acceptor</span><span class="c9">&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:2000&quot; name=&quot;Xlink:SDA&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink acceptor&quot;</span></p><p class="c14 c38 c35"><span class="c12">value=&quot;</span><span class="c71 c50">2</span><span class="c9">&quot; /&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; </span></p><p class="c13 c38 c35"><span class="c9">value=&quot;O:0:S&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c13 c49"><span class="c12">&nbsp;</span><span class="c9">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c33"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;0.0&quot; </span><span class="c12">residues=&quot;.&quot;</span><span class="c9">&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c97"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002057&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein N-term&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; </span></p><p class="c14 c38 c35"><span class="c12">value=&quot;</span><span class="c12">SDA_crosslink_acceptor_n_term</span><span class="c9">&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:2000&quot; name=&quot;Xlink:SDA&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink acceptor&quot;</span></p><p class="c14 c38 c35"><span class="c12">value=&quot;</span><span class="c71 c50">2</span><span class="c9">&quot; /&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; </span></p><p class="c13 c38 c35"><span class="c12">value=&quot;O:0:S&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c35"><span class="c9">&lt;/ModificationParams&gt;</span></p><p class="c14 c35"><span class="c9">...</span></p><p class="c14"><span class="c12">&lt;/SpectrumIdentificationProtocol&gt;</span></p><p class="c40 c15 c120"><span class="c29 c53 c44"></span></p><h2 class="c17" id="h.uhdz3v19usr5"><span class="c88 c50">7.4 Disuccinimidyl sulfoxide (DSSO)</span></h2><p class="c14"><span class="c9">&lt;SpectrumIdentificationProtocol&gt;</span></p><p class="c14 c35"><span class="c9">...</span></p><p class="c14 c35"><span class="c9">&lt;ModificationParams&gt;</span></p><p class="c14 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;175.030314&quot; </span><span class="c12">residues=&quot;K S T Y&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">DSSO_monolink_M</span><span class="c9">&quot;/&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:DSSO&quot;/&gt;</span></p><p class="c13"><span class="c12">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c9">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; </span></p><p class="c13 c38 c35"><span class="c12">value=&quot;M&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;176.01433&quot; </span><span class="c12">residues=&quot;K S T Y&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">DSSO_monolink_W</span><span class="c9">&quot;/&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:DSSO&quot;/&gt;</span></p><p class="c13"><span class="c12">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c9">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; </span></p><p class="c13 c38 c35"><span class="c12">value=&quot;W&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c15"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;175.030314&quot; </span><span class="c12">residues=&quot;.&quot;</span><span class="c9">&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c97"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002057&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein N-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">DSSO_monolink_M_n_term</span><span class="c9">&quot;/&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:DSSO&quot;/&gt;</span></p><p class="c13"><span class="c12">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c9">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; </span></p><p class="c13 c38 c35"><span class="c12">value=&quot;M&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;176.01433&quot; </span><span class="c12">residues=&quot;.&quot;</span><span class="c9">&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c97"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002057&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein N-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">DSSO_monolink_W_n_term</span><span class="c9">&quot;/&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; cvRef=&quot;UNIMOD&quot; name=&quot;Xlink:DSSO&quot;/&gt;</span></p><p class="c13"><span class="c12">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c9">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; </span></p><p class="c13 c38 c35"><span class="c12">value=&quot;W&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c6 c8"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;158.003765&quot; </span><span class="c12">residues=&quot;K S T Y&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">DSSO_crosslink_donor</span><span class="c9">&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; name=&quot;Xlink:DSSO&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c33"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; </span><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002509&quot; name=&quot;crosslink donor&quot; </span></p><p class="c14 c38 c35"><span class="c9">value=&quot;3&quot; /&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; </span></p><p class="c13 c38 c35"><span class="c9">value=&quot;A:54.0105647:ST&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c13 c33"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c9">&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot;</span></p><p class="c13 c38 c35"><span class="c9">value=&quot;S:103.9932001:A&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c13 c33"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp;</span><span class="c9">&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot;</span></p><p class="c13 c38 c35"><span class="c12">value=&quot;T:85.9826354:A&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c6 c35"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c6 c35"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;158.003765&quot; </span><span class="c12">residues=&quot;.&quot;</span><span class="c9">&gt;</span></p><p class="c13 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c97"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002057&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein N-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">DSSO_crosslink_donor_n_term</span><span class="c9">&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; name=&quot;Xlink:DSSO&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink donor&quot;</span></p><p class="c14 c38 c35"><span class="c9">value=&quot;3&quot; /&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; </span></p><p class="c13 c38 c35"><span class="c9">value=&quot;A:54.0105647:ST&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c13 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; </span></p><p class="c13 c38 c35"><span class="c9">value=&quot;S:103.9932001:A&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c13 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; </span></p><p class="c13 c38 c35"><span class="c12">value=&quot;T:85.9826354:A&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c15 c33"><span class="c9"></span></p><p class="c14 c33"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;0.0&quot;</span><span class="c12">&nbsp;residues=&quot;K S T Y&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">DSSO_crosslink_acceptor</span><span class="c9">&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; name=&quot;Xlink:DSSO&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink acceptor&quot;</span></p><p class="c14 c38 c35"><span class="c9">value=&quot;3&quot; /&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; </span></p><p class="c13 c38 c35"><span class="c9">value=&quot;A:54.0105647:ST&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c13 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; </span></p><p class="c13 c38 c35"><span class="c9">value=&quot;S:103.9932001:A&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c13 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; </span></p><p class="c13 c38 c35"><span class="c9">value=&quot;T:85.9826354:A&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c33"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;0.0&quot; </span><span class="c12">residues=&quot;.&quot;</span><span class="c9">&gt;</span></p><p class="c14 c38"><span class="c9">&lt;SpecificityRules&gt;</span></p><p class="c14 c97"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002057&quot; </span></p><p class="c14 c35 c97"><span class="c9">name=&quot;modification specificity protein N-term&quot;/&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;/SpecificityRules&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot;</span></p><p class="c14 c38 c35"><span class="c12">value=&quot;</span><span class="c12">DSSO_crosslink_acceptor_n_term</span><span class="c9">&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; name=&quot;Xlink:DSSO&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1002510&quot; name=&quot;crosslink acceptor&quot;</span></p><p class="c14 c38 c35"><span class="c9">value=&quot;3&quot; /&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot;</span></p><p class="c13 c38 c35"><span class="c9">value=&quot;A:54.0105647:ST&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c13 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; </span></p><p class="c13 c38 c35"><span class="c9">value=&quot;S:103.9932001:A&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c13 c33"><span class="c9">&nbsp; &nbsp; &nbsp; &nbsp;&lt;cvParam accession=&quot;MS:1003390&quot; name=&quot;crosslinker cleavage characteristics&quot; </span></p><p class="c13 c38 c35"><span class="c12">value=&quot;T:85.9826354:A&quot; cvRef=&quot;PSI-MS&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c33"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;54.010565&quot;</span><span class="c12">&nbsp;residues=&quot;K S T Y&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">DSSO_crosslink_stub_a</span><span class="c9">&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; name=&quot;Xlink:DSSO&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; </span></p><p class="c13 c38 c35"><span class="c9">value=&quot;A&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c13 c38"><span class="c12">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003346&quot; name=&quot;cleavable crosslinker stub&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c33"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;85.982636&quot;</span><span class="c12">&nbsp;residues=&quot;K S T Y&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">DSSO_crosslink_stub_t</span><span class="c9">&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; name=&quot;Xlink:DSSO&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; </span></p><p class="c13 c38 c35"><span class="c9">value=&quot;T&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c13 c38"><span class="c12">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003346&quot; name=&quot;cleavable crosslinker stub&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c14 c33 c15"><span class="c9"></span></p><p class="c14 c33"><span class="c12">&lt;SearchModification fixedMod=&quot;false&quot; massDelta=&quot;103.9932&quot;</span><span class="c12">&nbsp;residues=&quot;K S T Y&quot;</span><span class="c9">&gt;</span></p><p class="c14 c33 c35"><span class="c9">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003392&quot; </span></p><p class="c6 c35"><span class="c12">&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;name=&quot;search modification id&quot; value=&quot;</span><span class="c12">DSSO_crosslink_stub_s</span><span class="c9">&quot;/&gt;</span></p><p class="c14 c38"><span class="c9">&lt;cvParam accession=&quot;UNIMOD:1842&quot; name=&quot;Xlink:DSSO&quot; cvRef=&quot;UNIMOD&quot; /&gt;</span></p><p class="c1"><span class="c9">&lt;cvParam accession=&quot;MS:1003347&quot; name=&quot;UNIMOD derivative code&quot; </span></p><p class="c13 c38 c35"><span class="c9">value=&quot;S&quot; cvRef=&quot;PSI-MS&quot; /&gt;</span></p><p class="c13 c38"><span class="c12">&lt;cvParam cvRef=&quot;PSI-MS&quot; accession=&quot;MS:1003346&quot; name=&quot;cleavable crosslinker stub&quot;/&gt;</span></p><p class="c14 c33"><span class="c9">&lt;/SearchModification&gt;</span></p><p class="c6 c15"><span class="c9"></span></p><p class="c14 c35"><span class="c9">&lt;/ModificationParams&gt;</span></p><p class="c14 c35"><span class="c9">...</span></p><p class="c14"><span class="c12">&lt;/SpectrumIdentificationProtocol&gt;</span></p><p class="c14 c15"><span class="c0"></span></p><hr style="page-break-before:always;display:none;"><h1 class="c41 c81" id="h.2d7g9p2dc63j"><span class="c29 c43 c48"></span></h1><h1 class="c41" id="h.qlyprh427938"><span class="c43">8. </span><span class="c43">Authors</span><span class="c29 c43 c48">&nbsp;Information</span></h1><p class="c14"><span>Authors of this extension:</span></p><p class="c14"><span class="c0">Colin W. Combe, University of Edinburgh, colin.combe@ed.ac.uk</span></p><p class="c14"><span class="c0">Lars Kolbowski, Technische Universit&auml;t Berlin, lars.kolbowski@tu-berlin.de</span></p><p class="c14"><span class="c0">Lutz Fischer, Technische Universit&auml;t Berlin, lutz.fischer@tu-berlin.de</span></p><p class="c14"><span class="c0">Ville Koskinen, Matrix Science Ltd, villek@matrixscience.com</span></p><p class="c14"><span>Joshua Klein, University of Boston, </span><span class="c75">mobiusklein@gmail.com</span></p><p class="c14"><span class="c0">Alexander Leitner, ETH Zurich, leitner@imsb.biol.ethz.ch</span></p><p class="c14"><span class="c0">Juan Antonio Vizca&iacute;no, European Molecular Biology Laboratory, EMBL-EBI,</span></p><p class="c14"><span class="c0">juan@ebi.ac.uk</span></p><p class="c14"><span class="c0">Andy Jones, University of Liverpool, Andrew.Jones@liverpool.ac.uk</span></p><p class="c14"><span class="c0">Juri Rappsilber, Technische Universit&auml;t Berlin, Juri.Rappsilber@tu-Berlin.de</span></p><h1 class="c68" id="h.jk40ojgvmqu0"><span class="c43">9. </span><span class="c43">Glossary</span></h1><p class="c72 c15"><span class="c47 c29"></span></p><p class="c61 c49"><span>Cleavable Crosslinker - a crosslinker that can be broken in two to release the individual peptides (in a modified form), thus enabling their individual analysis. In crosslinking studies </span><span>this typically refers to </span><span class="c36">MS-cleavable crosslinkers</span><span>.</span></p><p class="c61 c49"><span>Controlled Vocabulary (CV) - A structured collection of terms describing a certain Crosslink - The covalent bond formed by a </span><span class="c36">crosslinker</span><span class="c0">.</span></p><p class="c61 c49"><span>Crosslink Acceptor - one end of a </span><span>crosslinking reaction</span><span>, the other is the </span><span class="c36">crosslink donor</span><span class="c0">. (Assumes the crosslinker has only two reactive groups.)</span></p><p class="c61 c49"><span>Crosslink Donor - one end of a crosslinking reaction, the other is the </span><span class="c36">crosslink acceptor</span><span class="c0">. (Assumes the crosslinker has only two reactive groups.)</span></p><p class="c61 c49"><span class="c0">Crosslinker - A chemical reagent. A molecule which creates a covalent bond either between proteins or within the same protein chain. This bond preserves proximity information which would otherwise be destroyed by the enzymatic digestion of the proteins. The proximity information is then recovered by identifying the peptides via mass spectrometry. </span></p><p class="c61 c49"><span>Crosslinker Modified Peptide - a peptide where one reactive group of a </span><span class="c36">crosslinker</span><span class="c0">&nbsp;has reacted with one of its amino acids, but the other reactive group has not reacted with any animo acid and no covalent bond is formed. </span></p><p class="c61 c49"><span>Crosslinker Specificity - The specificity of a crosslinker gives the amino acids it will react with. A </span><span class="c36">crosslinker</span><span class="c0">&nbsp;has reactive groups that react with amino acids or their side-chains, the reactive groups determine the specificity of the crosslinker.</span></p><p class="c61 c49"><span class="c0">Crosslinking Product - the result of a crosslinking reaction.</span></p><p class="c61 c49"><span class="c0">Crosslink Spectrum Match (CSM) - the subset of PSM level matches that contain a crosslink.</span></p><p class="c61 c49"><span>Decoy (decoy databases, decoy peptide) - decoys are a set of artificially generated sequences used to assess the performance of an identification algorithm. Decoys are typically created by randomising or reversing the sequences of the target proteins. These new sequences are then added to the database used for analysis.</span><span>&nbsp;</span><span class="c36">False Discovery Rates</span><span>&nbsp;may use the identifications of decoys to estimate the error rate in the data.</span></p><p class="c61 c49"><span class="c0">False Discovery Rate (FDR) - the fraction of identifications that are predicted to be incorrect (false positives) among the total number of identifications made.</span></p><p class="c61 c49"><span class="c0">Global FDR - the False Discovery Rate across the whole dataset, as opposed to &lsquo;local FDR&rsquo; which calculates the error rate within a given score window.</span></p><p class="c61 c49"><span>Heterobifunctional Crosslinker - a </span><span class="c36">crosslinker</span><span class="c0">&nbsp;with two reactive groups in which the reactive groups, and hence the specificity of each end, are different. </span></p><p class="c61 c49"><span>Higher Order Crosslink - A </span><span class="c36">crosslinking product</span><span class="c0">&nbsp;in which there are arbitrarily many peptides with many crosslinks between them.</span></p><p class="c61 c49"><span class="c0">Internally Linked Peptide - A crosslinking product in which both ends of the crosslinker have reacted within a single peptide (that is, within the same identical peptide, not between two copies of the same peptide). This type of crosslinking product is known to be intramolecular.</span></p><p class="c61 c49"><span>Looplink - colloquial name for an </span><span class="c36">internally linked peptide</span><span>.</span></p><p class="c61 c49"><span class="c0">Modification - a chemical change to an amino acid.</span></p><p class="c61 c49"><span>MS-Cleavable Crosslinker - </span><span>a type of </span><span class="c36">cleavable crosslinker</span><span>, MS-cleavable crosslinkers </span><span class="c0">can cleave upon activation in the mass spectrometer, releasing the individual peptides and thus enabling their individual analysis.</span></p><p class="c61 c49"><span class="c0">Noncovalently Associated Peptides - two different peptides which were not crosslinked but stayed associated with each other throughout the workflow, due to noncovalent interactions.</span></p><p class="c61 c49"><span>Monolink - colloquial name for a </span><span class="c36">crosslinker modified peptide</span><span>. Unimod uses this term to describe some derivatives of crosslinkers (e.g. see http://www.unimod.org/modifications_view.php?editid1=1842).</span><span class="c36">&nbsp;</span></p><p class="c61 c49"><span class="c0">Open Modification Search - a search strategy which allows for any type of mass shift to occur at any residue within a peptide sequence. In contrast to a &quot;closed&quot; search, which is limited to a defined set of modifications, an open modification search allows for the identification of novel or unexpected modifications</span></p><p class="c61 c49"><span class="c0">Posterior Error Probability - a statistical measure of the probability that an identification is incorrect.</span></p><p class="c61 c49"><span class="c0">Protein Heteromeric Crosslinks - crosslinks between distinct protein sequences. &nbsp;</span></p><p class="c61 c49"><span class="c0">Protein-Protein Interaction (PPI) - an interaction between proteins. In the context of crosslinking, it is a level of consolidation at which crosslinks may be analysed.</span></p><p class="c61 c49"><span class="c0">PSI-MS - the Human Proteome Organization (HUPO) Proteomics Standards Initiative&rsquo;s (PSI) controlled vocabulary for Mass Spectrometry (MS)</span></p><p class="c61 c49"><span>Peptide Spectrum Match (PSM) - a match to one </span><span>or more</span><span>&nbsp;peptides in a mass spectrum. The lowest, unconsolidated level at which analysis of identifications can occur. Because a single &ldquo;match&rdquo; may identify more than one peptide, the concept &lsquo;PSM&rsquo; does not correspond directly with &ldquo;spectrum identification item&rdquo; in mzIdentML, no element in mzIdentML does correspond directly to &ldquo;a </span><span>match</span><span class="c0">&rdquo;. &nbsp; &nbsp; </span></p><p class="c61 c49"><span>PSM-level - the level of analysis that looks at individual </span><span class="c36">peptide spectrum matches</span><span class="c0">.</span></p><p class="c49 c61"><span class="c0">Residue-pair - a unique pair of crosslinked residues, irrespective of the peptides identified. A level of consolidation higher than unique peptide but lower than PPI. </span></p><p class="c61 c49"><span class="c0">Self Crosslinks - crosslinks between peptides within one protein sequence. &nbsp; </span></p><p class="c61 c49"><span class="c0">Trimeric Crosslinker - a crosslinker with three reactive groups. </span></p><h1 class="c41" id="h.iqbq77xzfts5"><span class="c43">10. </span><span class="c43">References</span></h1><p class="c111"><span class="c25">1. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c25 c45"><a class="c18" href="https://www.google.com/url?q=http://paperpile.com/b/YrD2gH/vjf16&amp;sa=D&amp;source=editors&amp;ust=1719221372950472&amp;usg=AOvVaw0KXu5ObIqNQdgcjqrYVsCc">Vizca&iacute;no JA, Mayer G, Perkins S, Barsnes H, Vaudel M, Perez-Riverol Y, et al. The mzIdentML Data Standard Version 1.2, Supporting Advances in Proteome Informatics. Mol Cell Proteomics. 2017;16: 1275&ndash;1285.</a></span></p><p class="c100"><span class="c25">2. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c25 c45"><a class="c18" href="https://www.google.com/url?q=http://paperpile.com/b/YrD2gH/IanGW&amp;sa=D&amp;source=editors&amp;ust=1719221372950937&amp;usg=AOvVaw0hLKCJPP8e-59bP63KjSqT">Giese SH, Belsom A, Sinn L, Fischer L, Rappsilber J. Noncovalently Associated Peptides Observed during Liquid Chromatography-Mass Spectrometry and Their Effect on Cross-Link Analyses. Anal Chem. 2019;91: 2678&ndash;2685.</a></span></p><p class="c100"><span class="c25">3. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c25 c45"><a class="c18" href="https://www.google.com/url?q=http://paperpile.com/b/YrD2gH/9dZZN&amp;sa=D&amp;source=editors&amp;ust=1719221372951266&amp;usg=AOvVaw21dLwOOIw6MeC0g3KeoHjx">Mayer G, Jones AR, Binz P-A, Deutsch EW, Orchard S, Montecchi-Palazzi L, et al. Controlled vocabularies and ontologies in proteomics: overview, principles and practice. Biochim Biophys Acta. 2014;1844: 98&ndash;107.</a></span></p><p class="c100"><span class="c25">4. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c25 c45"><a class="c18" href="https://www.google.com/url?q=http://paperpile.com/b/YrD2gH/osIjk&amp;sa=D&amp;source=editors&amp;ust=1719221372951580&amp;usg=AOvVaw3x5VzCFSfjJOSGlgUYo-7k">Liu F, L&ouml;ssl P, Scheltema R, Viner R, Heck AJR. Optimized fragmentation schemes and data analysis strategies for proteome-wide cross-link identification. Nat Commun. 2017;8: 15473.</a></span></p><p class="c100"><span class="c25">5. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c25 c45"><a class="c18" href="https://www.google.com/url?q=http://paperpile.com/b/YrD2gH/kAbLF&amp;sa=D&amp;source=editors&amp;ust=1719221372951912&amp;usg=AOvVaw0tIUxCN7lrNaLffscW4lwt">Lenz S, Sinn LR, O&rsquo;Reilly FJ, Fischer L, Wegner F, Rappsilber J. Reliable identification of protein-protein interactions by crosslinking mass spectrometry. Nat Commun. 2021;12: 1&ndash;11.</a></span></p><p class="c100"><span class="c25">6. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c25 c45"><a class="c18" href="https://www.google.com/url?q=http://paperpile.com/b/YrD2gH/ghaA0&amp;sa=D&amp;source=editors&amp;ust=1719221372952226&amp;usg=AOvVaw23DB873xhrgyiNmpD_9UQ0">Fischer L, Rappsilber J. Quirks of Error Estimation in Cross-Linking/Mass Spectrometry. Anal Chem. 2017;89: 3829.</a></span></p><p class="c100"><span class="c25">7. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span><span class="c25 c45"><a class="c18" href="https://www.google.com/url?q=http://paperpile.com/b/YrD2gH/8J3I&amp;sa=D&amp;source=editors&amp;ust=1719221372952538&amp;usg=AOvVaw0vwbt19TYXjU9wP-7EygyH">Mohr JP, Caudal A, Tian R, Bruce JE. Multidimensional Cross-Linking and Real-Time Informatics for Multiprotein Interaction Studies. J Proteome Res. 2024;23. doi:</a></span><span class="c25 c45"><a class="c18" href="https://www.google.com/url?q=http://dx.doi.org/10.1021/acs.jproteome.3c00455&amp;sa=D&amp;source=editors&amp;ust=1719221372952747&amp;usg=AOvVaw1HODDhZ7fJ0MzbnwTauJcf">10.1021/acs.jproteome.3c00455</a></span></p><h1 class="c68" id="h.hftkqxpufrbu"><span class="c29 c43 c48">11. Intellectual Property Statement</span></h1><p class="c72"><span class="c47 c29">&nbsp;</span></p><p class="c72"><span class="c0">The PSI takes no position regarding the validity or scope of any intellectual property or other rights that might be claimed to pertain to the implementation or use of the technology described in this document or the extent to which any license under such rights might or might not be available; neither does it represent that it has made any effort to identify any such rights. Copies of claims of rights made available for publication and any assurances of licenses to be made available, or the result of an attempt made to obtain a general license or permission for the use of such proprietary rights by implementers or users of this specification can be obtained from the PSI Chair.</span></p><p class="c72"><span class="c0">&nbsp;</span></p><p class="c72"><span class="c0">The PSI invites any interested party to bring to its attention any copyrights, patents or patent applications, or other proprietary rights which may cover technology that may be required to practice this recommendation. Please address the information to the PSI Chair (see contacts information at PSI website).</span></p><p class="c72"><span class="c29 c47">&nbsp; </span></p><h1 class="c68" id="h.64x63egbmg6f"><span class="c29 c43 c48">Copyright Notice</span></h1><p class="c72"><span class="c47 c29">&nbsp;</span></p><p class="c72"><span class="c0">Copyright (C) Proteomics Standards Initiative (2023). All Rights Reserved.</span></p><p class="c72"><span class="c0">&nbsp;</span></p><p class="c72"><span class="c0">This document and translations of it may be copied and furnished to others, and derivative works that comment on or otherwise explain it or assist in its implementation may be prepared, copied, published and distributed, in whole or in part, without restriction of any kind, provided that the above copyright notice and this paragraph are included on all such copies and derivative works. However, this document itself may not be modified in any way, such as by removing the copyright notice or references to the PSI or other organizations, except as needed for the purpose of developing Proteomics Recommendations in which case the procedures for copyrights defined in the PSI Document process must be followed, or as required to translate it into languages other than English.</span></p><p class="c72"><span class="c0">&nbsp;</span></p><p class="c72"><span class="c0">The limited permissions granted above are perpetual and will not be revoked by the PSI or its successors or assigns.</span></p><p class="c72"><span class="c0">&nbsp;</span></p><p class="c72"><span class="c0">This document and the information contained herein is provided on an &quot;AS IS&quot; basis and THE PROTEOMICS STANDARDS INITIATIVE DISCLAIMS ALL WARRANTIES, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO ANY WARRANTY THAT THE USE OF THE INFORMATION HEREIN WILL NOT INFRINGE ANY RIGHTS OR ANY IMPLIED WARRANTIES OF MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE.&quot;</span></p><p class="c72"><span class="c0">&nbsp;</span></p><p class="c72"><span class="c10">&nbsp;</span></p><p class="c14 c15"><span class="c0"></span></p><p class="c14"><span class="c0">=======================================================================</span></p><div><p class="c14"><span class="c63 c94"><a class="c18" href="https://www.google.com/url?q=https://www.psidev.info/mzidentml&amp;sa=D&amp;source=editors&amp;ust=1719221372955246&amp;usg=AOvVaw2Dfh0IjpyfyyP7e1dhp51z">https://www.psidev.info/mzidentml</a></span><span>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span></p></div></body></html>